# Ribosome profiling

You can obtain results using [plotVisualRibo.R](https://github.com/bhattlab/bhattlab_workflows/tree/master/metariboseq) script that utilizes .bam files and results obtained from metagenomic analysis as input files. (Linux environment is recommended)

```R
#install RiboseqR
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("RiboseqR")
```
```bash
#ribo-seq running
#file download in https://github.com/bhattlab/bhattlab_workflows/tree/master/metariboseq
$ Rscript plotVisualRibo.R final.contigs.fa sample_metariboseq.bam
```

In summary, five types of files are generated as a result: "allFSshift.pdf," "histogram.pdf," and "plotCDSfilt.pdf" (with three variations for different footprints, i.e., 28, 31, and 35).

<figure align = "center">
  <img src="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaProteomic/img/P_7_1.png?raw=true" style="width:90%">
  <figcaption><b>Example of the allFSshift.pdf file generated by the plotVisualRibo.R results</b></figcaption>  
</figure>

Regarding the "allFSshift.pdf" results, this file provides frame information and the proportion of each frame for codon periodicity.
In the image above (an example from "allFSshift.pdf"), the x-axis represents the footprint (size), and the y-axis represents the number of reads.
This information is used as an indicator to predict the periodicity of gene nucleotide sequences in codons. It helps determine how ribosome-bound mRNA footprints are captured based on whether they align with a specific reading frame.
It's important to note that the results of this periodicity may exhibit different patterns among microbial species, reflecting variations in their translation mechanisms.

<figure align = "center">
  <img src="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaProteomic/img/P_7_2.png?raw=true" style="width:90%">
  <figcaption><b>Example of the histogram.pdf(left) and plotCDSfilt.pdf(right) file generated by the plotVisualRibo.R results</b></figcaption>  
</figure>  

The Histogram.pdf results in a graph that shows the length of reads and the alignment rate, allowing you to determine if there are a significant number of reads of specific lengths and the proportion of those reads in the dataset.
The plotCDSfilt.pdf files are generated for ribosome footprint sizes of 28, 31, and 35, resulting in three PDF files.
In contrast to RNA-seq results, ribo-seq results exhibit strong signals at the start codon and stop codon. This phenomenon is related to the characteristics of ribosomes during mRNA translation.
Ribosomes spend a longer time recognizing mRNA and preparing for translation during the start codon recognition phase, and the same happens at the stop codon when translation terminates.
This leads to the strong signals observed at the start and stop codons.

By using this information, you can obtain the count values for ribosome-bound mRNA. These count values provide information about the location of ribosomes and their proportions during translation.
Utilizing this ribosome information, you can indirectly gather insights related to protein synthesis.
However, analyzing ribosome-bound mRNA information alongside transcriptome data often yields more comprehensive and insightful results.


<figure align = "center">
  <img src="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaProteomic/img/P_7_3.png?raw=true" style="width:90%">
  <figcaption><b>Predicted translation efficiency; <a href="https://doi.org/10.1038/nrm4069">[ref]</a></b></figcaption>
</figure>

### References

Ingolia NT, Ghaemmaghami S, Newman JR, Weissman JS. Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. *Science*. 2009;324(5924):218-223. doi:10.1126/science.1168978

Ingolia NT. Ribosome profiling: new views of translation, from single codons to genome scale. *Nat Rev Genet*. 2014;15(3):205-213. doi:10.1038/nrg3645

Calviello L, Simm S, Wolff J, Bischoff V, Melemenidis A, Rappsilber J, KÃ¶nig J. RiboTaper: defining handler positions in ribosome profiling data. *Nat Methods*. 2016;13(2):165-170. doi:10.1038/nmeth.3688

Dunn JG, Weissman JS. Plastid: nucleotide-resolution analysis of next-generation sequencing and genomics data. *BMC Genomics*. 2016;17:958. doi:10.1186/s12864-016-3278-x

Ji Z, Song R, Regev A, Gsponer J. Many lncRNAs activate translation of upstream open reading frames. *Nat Struct Mol Biol*. 2016;23(12):1105-1112. doi:10.1038/nsmb.3331
