<!DOCTYPE html><html lang="en" class="" style="scroll-padding:60px"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>De novo assembly - META-Insight</title><meta property="og:title" content="De novo assembly - META-Insight"/><meta name="generator" content="mystmd"/><meta name="description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta property="og:description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta name="keywords" content="metagenomics, metatranscriptomics, metaproteomics, microbiome, bioinformatics"/><meta name="image" content="/build/030715b7a3310134807a1e7844273e01.png"/><meta property="og:image" content="/build/030715b7a3310134807a1e7844273e01.png"/><link rel="stylesheet" href="/build/_assets/app-MOQGDXHO.css"/><link rel="stylesheet" href="/build/_assets/thebe-core-VKVHG5VY.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/jupyter-matplotlib@0.11.3/css/mpl_widget.css"/><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.css" integrity="sha384-MlJdn/WNKDGXveldHDdyRP1R4CTHr3FeuDNfhsLPYrq2t0UBkUdK2jyTnXPEK1NQ" crossorigin="anonymous"/><link rel="icon" href="/favicon.ico"/><link rel="stylesheet" href="/myst-theme.css"/><script>
  const savedTheme = localStorage.getItem("myst:theme");
  const theme = window.matchMedia("(prefers-color-scheme: light)").matches ? 'light' : 'dark';
  const classes = document.documentElement.classList;
  const hasAnyTheme = classes.contains('light') || classes.contains('dark');
  if (!hasAnyTheme) classes.add(savedTheme ?? theme);
</script></head><body class="m-0 transition-colors duration-500 bg-white dark:bg-stone-900"><div class="myst-skip-to-article fixed top-1 left-1 h-[0px] w-[0px] focus-within:z-40 focus-within:h-auto focus-within:w-auto bg-white overflow-hidden focus-within:p-2 focus-within:ring-1" aria-label="skip to content options"><a href="#skip-to-frontmatter" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article frontmatter</a><a href="#skip-to-article" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article content</a></div><dialog id="myst-no-css" style="position:fixed;left:0px;top:0px;width:100vw;height:100vh;font-size:4rem;padding:1rem;color:black;background:white"><strong>Site not loading correctly?</strong><p>This may be due to an incorrect <code>BASE_URL</code> configuration. See<!-- --> <a href="https://mystmd.org/guide/deployment#deploy-base-url">the MyST Documentation</a> <!-- -->for reference.</p><script>
    (() => {
            // Test for has-styling variable set by the MyST stylesheet
            const node = document.currentScript.parentNode;
            const hasCSS = window.getComputedStyle(node).getPropertyValue("--has-styling");
            if (hasCSS === ""){
                    node.showModal();
            }

    })()
</script></dialog><div class="myst-top-nav bg-white/80 backdrop-blur dark:bg-stone-900/80 shadow dark:shadow-stone-700 p-3 md:px-8 sticky w-screen top-0 z-30 h-[60px]"><nav class="myst-top-nav-bar flex items-center justify-between flex-nowrap max-w-[1440px] mx-auto"><div class="flex flex-row xl:min-w-[19.5rem] mr-2 sm:mr-7 justify-start items-center shrink-0"><div class="block xl:hidden"><button class="myst-top-nav-menu-button flex items-center justify-center border-stone-400 text-stone-800 hover:text-stone-900 dark:text-stone-200 hover:dark:text-stone-100 w-10 h-10"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem"><path fill-rule="evenodd" d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75ZM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12Zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75Z" clip-rule="evenodd"></path></svg><span class="sr-only">Open Menu</span></button></div><a class="myst-home-link flex items-center ml-3 dark:text-white w-fit md:ml-5 xl:ml-7" href="/"><div class="myst-home-link-logo mr-3 flex items-center dark:bg-white dark:rounded px-1"><img src="/build/META-Insight_logo-dca4e62b5c9c37bfba1b2c18c24808e8.svg" class="h-9" height="2.25rem"/></div><span class="text-md sm:text-xl tracking-tight sm:mr-5 sr-only">Made with MyST</span></a></div><div class="flex items-center flex-grow w-auto"><div class="flex-grow hidden text-md lg:block"></div><div class="flex-grow block"></div><button type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R75cp:" data-state="closed" class="myst-search-bar flex items-center h-10 aspect-square sm:w-64 text-left text-gray-600 dark:text-gray-300 border border-gray-300 dark:border-gray-600 rounded-lg bg-gray-50 dark:bg-gray-700 myst-search-bar-disabled hover:ring-blue-500 dark:hover:ring-blue-500 hover:border-blue-500 dark:hover:border-blue-500"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="p-2.5 h-10 w-10 aspect-square"><path fill-rule="evenodd" d="M10.5 3.75a6.75 6.75 0 1 0 0 13.5 6.75 6.75 0 0 0 0-13.5ZM2.25 10.5a8.25 8.25 0 1 1 14.59 5.28l4.69 4.69a.75.75 0 1 1-1.06 1.06l-4.69-4.69A8.25 8.25 0 0 1 2.25 10.5Z" clip-rule="evenodd"></path></svg><span class="myst-search-text-placeholder hidden sm:block grow">Search</span><div aria-hidden="true" class="myst-search-shortcut items-center hidden mx-1 font-mono text-sm text-gray-600 dark:text-gray-300 sm:flex gap-x-1"><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none hide-mac">CTRL</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none show-mac">⌘</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none ">K</kbd><script>
;(() => {
const script = document.currentScript;
const root = script.parentElement;

const isMac = /mac/i.test(
      window.navigator.userAgentData?.platform ?? window.navigator.userAgent,
    );
root.querySelectorAll(".hide-mac").forEach(node => {node.classList.add(isMac ? "hidden" : "block")});
root.querySelectorAll(".show-mac").forEach(node => {node.classList.add(!isMac ? "hidden" : "block")});
})()</script></div></button><button class="myst-theme-button theme rounded-full aspect-square border border-stone-700 dark:border-white hover:bg-neutral-100 border-solid overflow-hidden text-stone-700 dark:text-white hover:text-stone-500 dark:hover:text-neutral-800 w-10 h-10 mx-3" title="Toggle theme between light and dark mode" aria-label="Toggle theme between light and dark mode"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-moon-icon h-full w-full p-0.5 hidden dark:block"><path fill-rule="evenodd" d="M9.528 1.718a.75.75 0 0 1 .162.819A8.97 8.97 0 0 0 9 6a9 9 0 0 0 9 9 8.97 8.97 0 0 0 3.463-.69.75.75 0 0 1 .981.98 10.503 10.503 0 0 1-9.694 6.46c-5.799 0-10.5-4.7-10.5-10.5 0-4.368 2.667-8.112 6.46-9.694a.75.75 0 0 1 .818.162Z" clip-rule="evenodd"></path></svg><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-sun-icon h-full w-full p-0.5 dark:hidden"><path stroke-linecap="round" stroke-linejoin="round" d="M12 3v2.25m6.364.386-1.591 1.591M21 12h-2.25m-.386 6.364-1.591-1.591M12 18.75V21m-4.773-4.227-1.591 1.591M5.25 12H3m4.227-4.773L5.636 5.636M15.75 12a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0Z"></path></svg></button><div class="block sm:hidden"></div><div class="hidden sm:block"></div></div></nav></div><div class="myst-primary-sidebar fixed xl:article-grid grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px] hidden z-10" style="top:60px"><div class="myst-primary-sidebar-pointer pointer-events-auto xl:col-margin-left flex-col overflow-hidden hidden xl:flex"><div class="myst-primary-sidebar-nav flex-grow py-6 overflow-y-auto primary-scrollbar"><nav aria-label="Navigation" class="myst-primary-sidebar-topnav overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px] lg:hidden"><div class="w-full px-1 dark:text-white font-medium"></div></nav><div class="my-3 border-b-2 lg:hidden"></div><nav aria-label="Table of Contents" class="myst-primary-sidebar-toc flex-grow overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px]"><div class="myst-toc w-full px-1 dark:text-white"><a title="META-Insight" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30 font-bold" href="/">META-Insight</a><div data-state="open" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metagenomics" class="block break-words rounded py-2 grow font-semibold text-blue-800 dark:text-blue-200 cursor-pointer">Metagenomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rmpsp:" aria-expanded="true" data-state="open"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="open" id="radix-:Rmpsp:" class="pl-3 pr-[2px] collapsible-content"><a title="Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/introduction">Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis</a><a title="Preparation for sequencing" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/sequencing">Preparation for sequencing</a><a title="Metagenomic analysis tools" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/installation">Metagenomic analysis tools</a><a title="Preprocessing of the sequencing reads" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/preprocessing">Preprocessing of the sequencing reads</a><a title="De novo assembly" aria-current="page" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg myst-toc-item-exact bg-blue-300/30 active" href="/denovo-assembly">De novo assembly</a><a title="Binning" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/binning">Binning</a><a title="Taxonomic annotation of contigs" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/taxonomic">Taxonomic annotation of contigs</a><a title="Phylogenetic" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/phylogenetic">Phylogenetic</a><a title="Functional annotation of contigs" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/functional">Functional annotation of contigs</a><a title="Read-based profiling" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30" href="/read-based">Read-based profiling</a></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metatranscriptomics" class="block break-words rounded py-2 grow cursor-pointer">Metatranscriptomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rupsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rupsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metaproteomics" class="block break-words rounded py-2 grow cursor-pointer">Metaproteomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:R16psp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:R16psp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div></div></nav></div><div class="myst-primary-sidebar-footer flex-none py-6 transition-all duration-700 translate-y-6 opacity-0"><a class="myst-made-with-myst flex mx-auto text-gray-700 w-fit hover:text-blue-700 dark:text-gray-200 dark:hover:text-blue-400" href="https://mystmd.org/made-with-myst" target="_blank" rel="noreferrer"><svg style="width:24px;height:24px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" stroke="none"><g id="icon"><path fill="currentColor" d="M23.8,54.8v-3.6l4.7-0.8V17.5l-4.7-0.8V13H36l13.4,31.7h0.2l13-31.7h12.6v3.6l-4.7,0.8v32.9l4.7,0.8v3.6h-15
          v-3.6l4.9-0.8V20.8H65L51.4,53.3h-3.8l-14-32.5h-0.1l0.2,17.4v12.1l5,0.8v3.6H23.8z"></path><path fill="#F37726" d="M47,86.9c0-5.9-3.4-8.8-10.1-8.8h-8.4c-5.2,0-9.4-1.3-12.5-3.8c-3.1-2.5-5.4-6.2-6.8-11l4.8-1.6
          c1.8,5.6,6.4,8.6,13.8,8.8h9.2c6.4,0,10.8,2.5,13.1,7.5c2.3-5,6.7-7.5,13.1-7.5h8.4c7.8,0,12.7-2.9,14.6-8.7l4.8,1.6
          c-1.4,4.9-3.6,8.6-6.8,11.1c-3.1,2.5-7.3,3.7-12.4,3.8H63c-6.7,0-10,2.9-10,8.8"></path></g></svg><span class="self-center ml-2 text-sm">Made with MyST</span></a></div></div></div><main class="article-grid grid-gap"><article class="article-grid subgrid-gap col-screen article content"><div class="hidden"></div><div id="skip-to-frontmatter" aria-label="article frontmatter" class="myst-fm-block mb-8 pt-9"><div class="myst-fm-block-header flex items-center mb-5 h-6 text-sm font-light"><div class="flex-grow"></div><div class="myst-fm-block-badges"><a href="https://github.com/your-username/metainsight-book" title="GitHub Repository: your-username/metainsight-book" target="_blank" rel="noopener noreferrer" class="myst-fm-github-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" width="1.25rem" height="1.25rem" class="myst-fm-github-icon inline-block mr-1 opacity-60 hover:opacity-100"><path d="M12 2.5c-5.4 0-9.8 4.4-9.8 9.7 0 4.3 2.8 8 6.7 9.2.5.1.7-.2.7-.5v-1.8c-2.4.5-3.1-.6-3.3-1.1-.1-.3-.6-1.1-1-1.4-.3-.2-.8-.6 0-.6s1.3.7 1.5 1c.9 1.5 2.3 1.1 2.8.8.1-.6.3-1.1.6-1.3-2.2-.2-4.4-1.1-4.4-4.8 0-1.1.4-1.9 1-2.6-.1-.2-.4-1.2.1-2.6 0 0 .8-.3 2.7 1 .8-.2 1.6-.3 2.4-.3.8 0 1.7.1 2.4.3 1.9-1.3 2.7-1 2.7-1 .5 1.3.2 2.3.1 2.6.6.7 1 1.5 1 2.6 0 3.7-2.3 4.6-4.4 4.8.4.3.7.9.7 1.8V21c0 .3.2.6.7.5 3.9-1.3 6.6-4.9 6.6-9.2 0-5.4-4.4-9.8-9.8-9.8z"></path></svg></a></div><a href="https://github.com/your-username/metainsight-book/edit/main/metagenomics/Denovo_assembly.md" title="Edit This Page" target="_blank" rel="noopener noreferrer" class="myst-fm-edit-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-edit-icon inline-block mr-1 opacity-60 hover:opacity-100"><path stroke-linecap="round" stroke-linejoin="round" d="m16.862 4.487 1.687-1.688a1.875 1.875 0 1 1 2.652 2.652L10.582 16.07a4.5 4.5 0 0 1-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 0 1 1.13-1.897l8.932-8.931Zm0 0L19.5 7.125M18 14v4.75A2.25 2.25 0 0 1 15.75 21H5.25A2.25 2.25 0 0 1 3 18.75V8.25A2.25 2.25 0 0 1 5.25 6H10"></path></svg></a><div class="myst-fm-downloads-dropdown relative flex inline-block mx-1 grow-0" data-headlessui-state=""><button class="myst-fm-downloads-button relative ml-2 -mr-1" id="headlessui-menu-button-:Rs8ucp:" type="button" aria-haspopup="menu" aria-expanded="false" data-headlessui-state=""><span class="sr-only">Downloads</span><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-downloads-icon"><title>Download</title><path stroke-linecap="round" stroke-linejoin="round" d="M3 16.5v2.25A2.25 2.25 0 0 0 5.25 21h13.5A2.25 2.25 0 0 0 21 18.75V16.5M16.5 12 12 16.5m0 0L7.5 12m4.5 4.5V3"></path></svg></button></div></div><h1 class="myst-fm-block-title mb-0">De novo assembly</h1><header class="myst-fm-authors-affiliations mt-4 not-prose"><div class="myst-fm-authors-list"><span class="myst-fm-author font-semibold text-sm myst-fm-author-item inline-block"><button class="myst-fm-author-popover focus:shadow-[0_0_0_2px] focus:shadow-black outline-none hover:underline" aria-label="Author Details" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R78ucp:" data-state="closed"><span class="myst-fm-author-name">META-Insight Team</span></button></span></div></header></div><div class="block my-10 lg:sticky lg:z-10 lg:h-0 lg:pt-0 lg:my-0 lg:ml-10 lg:col-margin-right" style="top:60px"><nav></nav></div><div id="skip-to-article"></div><figure id="SheGrq63n6" class=""><img id="uDB969cqcl" style="margin-left:auto;margin-right:auto" src="/build/030715b7a3310134807a1e7844273e01.png" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_1.png?raw=true" class=""/><figcaption class="group"><p><strong></strong></p></figcaption></figure><h2 id="short-reads" class="relative group"><span class="heading-text">Short reads</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#short-reads" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><p>Read-based profiling offers a rapid overview of the metagenome but is limited by the presence of a significant number of unmapped reads.
To overcome this limitation, de novo assembly is employed using assemblers to extend overlapping reads into contigs or scaffolds.
This approach enables in-depth profiling of genomic components and the discovery of previously unknown sequences since it is a database-independent process.</p><p>Currently, various assemblers are available. Some assemblers are specifically designed for the efficient assembly of short reads.
Among the most widely utilized assemblers, <a target="_blank" rel="noreferrer" href="https://github.com/voutcn/megahit" class="link">MEGAHIT<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a> and <a target="_blank" rel="noreferrer" href="https://cab.spbu.ru/software/meta-spades/" class="link">metaSPAdes<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a> stand out.
metaSPAdes is particularly renowned for its performance with high-depth samples. However, it demands a substantial amount of memory, even for small-sized metagenome data.
MEGAHIT shows competitive assembly quality and efficient memory usage compared to metaSPAdes. <cite class="" data-state="closed"><a href="https://doi.org/10.1038/s41596-020-00480-3" target="_blank" rel="noreferrer" class="hover-link">[ref]</a></cite>.</p><p>MEGAHIT employs k-mers, and the size of these k-mers can be adjusted based on the sample’s complexity using the <code>--k-min</code> and <code>--k-max</code> options.
When dealing with high-depth samples, it is advisable to select a relatively larger k-mer size (typically 25-31) to prevent excessive complexity in the de Bruijn graph.
Conversely, for low-depth samples where the assembly isn’t functioning optimally, you can enhance assembly efficiency by employing the <code>--no-mercy</code> option.
For paired-end reads, make sure to specify the forward and reverse read filenames separately using <code>-1</code> and <code>-2.</code> For single reads, use the <code>-r</code> option.
The maximum memory allocated for de Bruijn graph construction can be set using the <code>-m</code> option and the total system memory can be adjusted within the range of 0-1.
The default value is typically set to 0.9. The <code>-t</code> option allows you to define the number of CPU threads, with a minimum requirement of 2.
By default, the software automatically calculates the available CPU threads and utilizes them collectively.</p><div class="relative myst-code group not-prose my-5 text-sm shadow hover:shadow-md dark:shadow-2xl dark:shadow-neutral-900 bg-stone-200/10"><pre class="block overflow-auto p-3 myst-code-body hljs" style="background-color:unset"><code class="language-bash" style="white-space:pre"># run MEGAHIT 
$ conda activate MEGAHIT
$ MEGAHIT -1 kneaddata.trimmed.1.fastq  -2 kneaddata.trimmed.2.fastq  -m 0.5  -t 12 -o MEGAHIT_result
$ conda deactivate</code></pre><button title="Copy to Clipboard" class="inline-flex items-center opacity-0 group-hover:opacity-100 hover:opacity-100 focus:opacity-100 active:opacity-100 cursor-pointer ml-2 transition-color duration-200 ease-in-out text-blue-400 hover:text-blue-500 absolute right-1 myst-code-copy-icon top-1" aria-pressed="false" aria-label="Copy code to clipboard"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="24" height="24"><path stroke-linecap="round" stroke-linejoin="round" d="M15.75 17.25v3.375c0 .621-.504 1.125-1.125 1.125h-9.75a1.125 1.125 0 0 1-1.125-1.125V7.875c0-.621.504-1.125 1.125-1.125H6.75a9.06 9.06 0 0 1 1.5.124m7.5 10.376h3.375c.621 0 1.125-.504 1.125-1.125V11.25c0-4.46-3.243-8.161-7.5-8.876a9.06 9.06 0 0 0-1.5-.124H9.375c-.621 0-1.125.504-1.125 1.125v3.5m7.5 10.375H9.375a1.125 1.125 0 0 1-1.125-1.125v-9.25m12 6.625v-1.875a3.375 3.375 0 0 0-3.375-3.375h-1.5a1.125 1.125 0 0 1-1.125-1.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H9.75"></path></svg></button></div><h2 id="long-reads-assembly" class="relative group"><span class="heading-text">Long reads assembly</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#long-reads-assembly" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><p>PacBio HiFi long reads (typically 10–20 kb with &gt;99% accuracy) enable the resolution of repetitive genomic regions and the recovery of near-complete metagenome-assembled genomes that are inaccessible through short-read assembly alone. The assembler <a target="_blank" rel="noreferrer" href="https://github.com/lh3/hifiasm-meta" class="link">hifiasm-meta<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a> is specifically designed for metagenomic HiFi data, producing high-quality primary and alternate contigs by leveraging the length and accuracy of HiFi reads to resolve strain-level haplotypes within complex microbial communities. Unlike short-read assemblers based on de Bruijn graphs, hifiasm-meta uses an overlap-based approach that better preserves long-range genomic structure.</p><div class="relative myst-code group not-prose my-5 text-sm shadow hover:shadow-md dark:shadow-2xl dark:shadow-neutral-900 bg-stone-200/10"><pre class="block overflow-auto p-3 myst-code-body hljs" style="background-color:unset"><code class="language-bash" style="white-space:pre"># run hifiasm_meta 
$ conda activate hifiam_meta
$ hifiasm_meta -o N14.asm -t 60 N14.aln.unmapped.fq &gt; N14.log
$ conda deactivate
# .gfa 파일을 .fa 파일로 변환
$ awk &#x27;/^S/{print &quot;&gt;&quot;$2;print $3}&#x27; test.p_ctg.gfa &gt; test.p_ctg.fa</code></pre><button title="Copy to Clipboard" class="inline-flex items-center opacity-0 group-hover:opacity-100 hover:opacity-100 focus:opacity-100 active:opacity-100 cursor-pointer ml-2 transition-color duration-200 ease-in-out text-blue-400 hover:text-blue-500 absolute right-1 myst-code-copy-icon top-1" aria-pressed="false" aria-label="Copy code to clipboard"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="24" height="24"><path stroke-linecap="round" stroke-linejoin="round" d="M15.75 17.25v3.375c0 .621-.504 1.125-1.125 1.125h-9.75a1.125 1.125 0 0 1-1.125-1.125V7.875c0-.621.504-1.125 1.125-1.125H6.75a9.06 9.06 0 0 1 1.5.124m7.5 10.376h3.375c.621 0 1.125-.504 1.125-1.125V11.25c0-4.46-3.243-8.161-7.5-8.876a9.06 9.06 0 0 0-1.5-.124H9.375c-.621 0-1.125.504-1.125 1.125v3.5m7.5 10.375H9.375a1.125 1.125 0 0 1-1.125-1.125v-9.25m12 6.625v-1.875a3.375 3.375 0 0 0-3.375-3.375h-1.5a1.125 1.125 0 0 1-1.125-1.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H9.75"></path></svg></button></div><p>“.gfa” stands for graphical fragment assembly, a file format designed to store graph-based assembly information.
It contains data about the graph generated during the assembly process.
The structure of the graph file is akin to a de Bruijn graph, illustrating the connectivity of nodes and edges.
These graphs depict the interactions among overlapping DNA sequences, aiding in the assembly process.</p><p>When running <a target="_blank" rel="noreferrer" href="https://github.com/lh3/hifiasm-meta" class="link">Hifiasm-meta<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a>, the output files are as follows.</p><p><strong>1. Raw unitig graph: asm.r_utg*.gfa</strong><br/>This file contains the raw unitig graph generated during the assembly process. The raw unitig graph is the initial graph for assembly, which organizes nodes and edges based on overlapping sequence information.
<strong>2. Cleaned unitig graph: asm.p_utg*.gfa</strong><br/>This file contains preprocessed cleaned unitig graphs. Preprocessing refers to the process of removing errors from the raw unitig graph and improving its accuracy.
<strong>3. Contig graph: asm.p_ctg*.gfa, asm.a_ctg*.gfa</strong><br/>This file contains the contig graph. The file asm.p_ctg*.gfa means primary contig and asm.a_ctg*.gfa means alternate contig.</p><ul><li><p>Unitig<br/>A unitig, which stands for unique sequence, is a unit of overlapping DNA sequences that is the result of an intermediate step in assembly. Each unitig consists of one or more DNA sequences and contains overlapping parts. The assembly process generates a raw unitig graph. This raw unitig graph may contain some errors. The subsequent processing removes the errors from the raw unitig graph and generates a cleaned unitig graph.</p></li></ul><h2 id="contig-check" class="relative group"><span class="heading-text">Contig check</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#contig-check" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><p>Long read assembly results can be visualized using a bandage plot, which is a tool for representing complex assembly results, including De Bruijn graphs or overlap graphs.
Long read assemblies often yield intricate graphs, and the bandage plot is a good option for visualizing these graphs.</p><p>It provides the following information.</p><p>To generate bandage plots, you can use Bandage, a Bioinformatics Application for Navigating De novo Assembly Graphs Easily. Bandage is available for download at <a target="_blank" rel="noreferrer" href="http://rrwick.github.io/Bandage/" class="link">http://<wbr/>rrwick<wbr/>.github<wbr/>.io<wbr/>/Bandage/<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a>.</p><p>After running the Bandage program, load the *p_ctg.gfa file from File &gt; Load graph at the top. Once the file is loaded, click Draw graph in Graph drawing to display the graph.</p><figure id="HQXFe3V8y2" class=""><img id="m53H0PQtrH" style="margin-left:auto;margin-right:auto" src="/build/ce209dd902988924d189874ae25587b1.png" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_2.png?raw=true" class=""/><figcaption class="group"><p><strong></strong></p></figcaption></figure><p>After assembly, check the assembly result and quality using <a target="_blank" rel="noreferrer" href="https://github.com/ablab/quast" class="link">QUAST<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a>(Quality assessment tool) to know the information of the produced contigs.</p><div class="relative myst-code group not-prose my-5 text-sm shadow hover:shadow-md dark:shadow-2xl dark:shadow-neutral-900 bg-stone-200/10"><pre class="block overflow-auto p-3 myst-code-body hljs" style="background-color:unset"><code class="language-bash" style="white-space:pre"># calculate assembly statistics
$ conda activate quast
$ quast.py MEGAHIT_result/final.contigs.fa -o MEGAHIT_quast
$ conda deactivate</code></pre><button title="Copy to Clipboard" class="inline-flex items-center opacity-0 group-hover:opacity-100 hover:opacity-100 focus:opacity-100 active:opacity-100 cursor-pointer ml-2 transition-color duration-200 ease-in-out text-blue-400 hover:text-blue-500 absolute right-1 myst-code-copy-icon top-1" aria-pressed="false" aria-label="Copy code to clipboard"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="24" height="24"><path stroke-linecap="round" stroke-linejoin="round" d="M15.75 17.25v3.375c0 .621-.504 1.125-1.125 1.125h-9.75a1.125 1.125 0 0 1-1.125-1.125V7.875c0-.621.504-1.125 1.125-1.125H6.75a9.06 9.06 0 0 1 1.5.124m7.5 10.376h3.375c.621 0 1.125-.504 1.125-1.125V11.25c0-4.46-3.243-8.161-7.5-8.876a9.06 9.06 0 0 0-1.5-.124H9.375c-.621 0-1.125.504-1.125 1.125v3.5m7.5 10.375H9.375a1.125 1.125 0 0 1-1.125-1.125v-9.25m12 6.625v-1.875a3.375 3.375 0 0 0-3.375-3.375h-1.5a1.125 1.125 0 0 1-1.125-1.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H9.75"></path></svg></button></div><figure id="vywHCWl9K7" class=""><img id="aCinSCuBPl" style="margin-left:auto;margin-right:auto" src="/build/7b78d3d813c18d8e084f529f11c33e7f.png" alt="Example of report.html" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_3.png?raw=true" class=""/><figcaption class="group"><p><strong>Example of report.html</strong></p></figcaption></figure><p><strong>contigs: Number of contigs produced after assembly</strong></p><ul><li><p>Largest contig: The length of the longest contig among the contigs produced after assembly</p></li><li><p>Total length: Total number of bases of contigs generated after assembly</p></li><li><p>N50: Length of the contig with the top 50% length among the contigs generated after assembly</p></li><li><p>N75: Length of the contig with the top 75% length among the contigs generated after assembly</p></li><li><p>L50: Number of contigs with the length of N50 among the contigs generated after assembly</p></li><li><p>L75: Number of contigs with the length of N75 among the contigs generated after assembly</p></li><li><p>Mismatches # N’s: Number of uncalled bases that were not assembled</p></li><li><p>Mismatches # N’s per 100kbp: Number of uncalled bases per 100000 bases</p></li></ul><p>The left Graph shows cumulative length as the contig index increases and the right graph shows GC contents of contigs</p><figure id="nIwyahlxm9" class=""><img id="PKmmnhRZbr" style="margin-left:auto;margin-right:auto" src="/build/eaa6417f1aaec13589a70369854ace71.png" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_4.png?raw=true" class=""/><figcaption class="group"><p><strong></strong></p></figcaption></figure><div class="myst-backmatter-parts"></div><section id="references" class="myst-bibliography article-grid subgrid-gap col-screen"><div><header class="myst-bibliography-header text-lg font-semibold text-stone-900 dark:text-white group">References<a class="no-underline text-inherit hover:text-inherit ml-2 select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#references" title="Link to References" aria-label="Link to References">¶</a></header></div><div class="myst-bibliography-list pl-3 mb-8 text-xs text-stone-500 dark:text-stone-300"><ol><li class="myst-bibliography-item break-words" id="cite-Meyer_2021">Meyer, F., Lesker, T.-R., Koslicki, D., Fritz, A., Gurevich, A., Darling, A. E., Sczyrba, A., Bremges, A., & McHardy, A. C. (2021). Tutorial: assessing metagenomics software with the CAMI benchmarking toolkit. <i>Nature Protocols</i>, <i>16</i>(4), 1785–1801. <a target="_blank" rel="noreferrer" href="https://doi.org/10.1038/s41596-020-00480-3">10.1038/s41596-020-00480-3</a></li></ol></div></section><div class="myst-footer-links flex pt-10 mb-10 space-x-4"><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-prev" href="/preprocessing"><div class="flex h-full align-middle"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:-translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5 3 12m0 0 7.5-7.5M3 12h18"></path></svg><div class="flex-grow text-right"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metagenomics</div>Preprocessing of the sequencing reads</div></div></a><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-next" href="/binning"><div class="flex h-full align-middle"><div class="flex-grow"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metagenomics</div>Binning</div><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 4.5 21 12m0 0-7.5 7.5M21 12H3"></path></svg></div></a></div></article></main><script>((a,l)=>{if(!window.history.state||!window.history.state.key){let u=Math.random().toString(32).slice(2);window.history.replaceState({key:u},"")}try{let d=JSON.parse(sessionStorage.getItem(a)||"{}")[l||window.history.state.key];typeof d=="number"&&window.scrollTo(0,d)}catch(u){console.error(u),sessionStorage.removeItem(a)}})("positions", null)</script><link rel="modulepreload" href="/build/entry.client-PCJPW7TK.js"/><link rel="modulepreload" href="/build/_shared/chunk-AQ2CODAG.js"/><link rel="modulepreload" href="/build/_shared/chunk-JJXTQVMA.js"/><link rel="modulepreload" href="/build/_shared/chunk-OZE3FFNP.js"/><link rel="modulepreload" href="/build/_shared/chunk-CH4FVTDV.js"/><link rel="modulepreload" href="/build/_shared/chunk-C4DFGG5C.js"/><link rel="modulepreload" href="/build/_shared/chunk-J7TUH54J.js"/><link rel="modulepreload" href="/build/_shared/chunk-FZ2S7OYD.js"/><link rel="modulepreload" href="/build/_shared/chunk-JEM6JXYA.js"/><link rel="modulepreload" href="/build/_shared/chunk-34XIY2DH.js"/><link rel="modulepreload" href="/build/_shared/chunk-KQM5FBHR.js"/><link rel="modulepreload" href="/build/_shared/chunk-OCWQY3HK.js"/><link rel="modulepreload" href="/build/_shared/chunk-7HNKBP4B.js"/><link rel="modulepreload" href="/build/_shared/chunk-CUKUDK3R.js"/><link rel="modulepreload" href="/build/_shared/chunk-3EBOCCHJ.js"/><link rel="modulepreload" href="/build/_shared/chunk-O4VQNZ62.js"/><link rel="modulepreload" href="/build/_shared/chunk-4OEDG4JQ.js"/><link rel="modulepreload" href="/build/_shared/chunk-GUCIBHGO.js"/><link rel="modulepreload" href="/build/root-SIO6LUTY.js"/><link rel="modulepreload" href="/build/_shared/chunk-FAHZZXAC.js"/><link rel="modulepreload" href="/build/routes/$-PRP77N34.js"/><script>window.__remixContext = {"url":"/denovo-assembly","state":{"loaderData":{"root":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/build/META-Insight_logo-dca4e62b5c9c37bfba1b2c18c24808e8.svg","logo":"/build/META-Insight_logo-dca4e62b5c9c37bfba1b2c18c24808e8.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/your-username/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"CONTENT_CDN_PORT":"3100","MODE":"static"},"routes/$":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/build/META-Insight_logo-dca4e62b5c9c37bfba1b2c18c24808e8.svg","logo":"/build/META-Insight_logo-dca4e62b5c9c37bfba1b2c18c24808e8.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/your-username/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"page":{"version":3,"kind":"Article","sha256":"29c8c0888814261b80171a15bdee3929d019b350c37b7bdbe31a6b4c760fd804","slug":"denovo-assembly","location":"/metagenomics/Denovo_assembly.md","dependencies":[],"frontmatter":{"title":"De novo assembly","content_includes_title":false,"authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/your-username/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"numbering":{"title":{"offset":1}},"source_url":"https://github.com/your-username/metainsight-book/blob/main/metagenomics/Denovo_assembly.md","edit_url":"https://github.com/your-username/metainsight-book/edit/main/metagenomics/Denovo_assembly.md","thumbnail":"/build/030715b7a3310134807a1e7844273e01.png","exports":[{"format":"md","filename":"Denovo_assembly.md","url":"/build/Denovo_assembly-085a80efdbe6bd0621532bc96b31d181.md"}]},"mdast":{"type":"root","children":[{"type":"block","children":[{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/build/030715b7a3310134807a1e7844273e01.png","alt":"","data":{"altTextIsAutoGenerated":true},"key":"uDB969cqcl","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_1.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[],"key":"LOsPClzpDs"}],"key":"JbehkURyZv"}],"key":"EiqEJh0Kdi"}],"enumerator":"1","key":"SheGrq63n6"},{"type":"heading","depth":2,"position":{"start":{"line":7,"column":1},"end":{"line":7,"column":1}},"children":[{"type":"text","value":"Short reads","position":{"start":{"line":7,"column":1},"end":{"line":7,"column":1}},"key":"TqrEb2cE8s"}],"identifier":"short-reads","label":"Short reads","html_id":"short-reads","implicit":true,"key":"omRiHrrY4H"},{"type":"paragraph","position":{"start":{"line":8,"column":1},"end":{"line":10,"column":1}},"children":[{"type":"text","value":"Read-based profiling offers a rapid overview of the metagenome but is limited by the presence of a significant number of unmapped reads.\nTo overcome this limitation, de novo assembly is employed using assemblers to extend overlapping reads into contigs or scaffolds.\nThis approach enables in-depth profiling of genomic components and the discovery of previously unknown sequences since it is a database-independent process.","position":{"start":{"line":8,"column":1},"end":{"line":8,"column":1}},"key":"pyJYEbY9XW"}],"key":"WnhkGwI210"},{"type":"paragraph","position":{"start":{"line":12,"column":1},"end":{"line":15,"column":1}},"children":[{"type":"text","value":"Currently, various assemblers are available. Some assemblers are specifically designed for the efficient assembly of short reads.\nAmong the most widely utilized assemblers, ","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"lk48bChMtT"},{"type":"link","url":"https://github.com/voutcn/megahit","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"children":[{"type":"text","value":"MEGAHIT","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"g96JkL646C"}],"urlSource":"https://github.com/voutcn/megahit","error":true,"key":"vhwg6AOSLB"},{"type":"text","value":" and ","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"tM2ARsopMn"},{"type":"link","url":"https://cab.spbu.ru/software/meta-spades/","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"children":[{"type":"text","value":"metaSPAdes","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"Hv42zfQz5c"}],"urlSource":"https://cab.spbu.ru/software/meta-spades/","key":"WPcv0uLlsW"},{"type":"text","value":" stand out.\nmetaSPAdes is particularly renowned for its performance with high-depth samples. However, it demands a substantial amount of memory, even for small-sized metagenome data.\nMEGAHIT shows competitive assembly quality and efficient memory usage compared to metaSPAdes. ","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"l1PDmUUe5m"},{"type":"cite","url":"https://doi.org/10.1038/s41596-020-00480-3","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"children":[{"type":"text","value":"[ref]","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"IAxxQTGmbp"}],"kind":"narrative","label":"Meyer_2021","identifier":"https://doi.org/10.1038/s41596-020-00480-3","enumerator":"1","key":"OaFFqhbefl"},{"type":"text","value":".","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"UUsZT4Mp0T"}],"key":"fba4uiabwW"},{"type":"paragraph","position":{"start":{"line":17,"column":1},"end":{"line":23,"column":1}},"children":[{"type":"text","value":"MEGAHIT employs k-mers, and the size of these k-mers can be adjusted based on the sample’s complexity using the ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"EgRb8Sn1CL"},{"type":"inlineCode","value":"--k-min","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"ICzLQVPa4e"},{"type":"text","value":" and ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"sVPlvB6kuf"},{"type":"inlineCode","value":"--k-max","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"HeTjAsPY8a"},{"type":"text","value":" options.\nWhen dealing with high-depth samples, it is advisable to select a relatively larger k-mer size (typically 25-31) to prevent excessive complexity in the de Bruijn graph.\nConversely, for low-depth samples where the assembly isn’t functioning optimally, you can enhance assembly efficiency by employing the ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"pcaJQBrCWT"},{"type":"inlineCode","value":"--no-mercy","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"oS6cWdazfo"},{"type":"text","value":" option.\nFor paired-end reads, make sure to specify the forward and reverse read filenames separately using ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"JhRowXx7J9"},{"type":"inlineCode","value":"-1","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"wydK2AQvJV"},{"type":"text","value":" and ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"ZemJmsT3Wx"},{"type":"inlineCode","value":"-2.","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"rqI58nGRWG"},{"type":"text","value":" For single reads, use the ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"JNO9drPDTV"},{"type":"inlineCode","value":"-r","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"Kzyavi1dmR"},{"type":"text","value":" option.\nThe maximum memory allocated for de Bruijn graph construction can be set using the ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"RK7a44li9G"},{"type":"inlineCode","value":"-m","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"WQ7DR6jCwb"},{"type":"text","value":" option and the total system memory can be adjusted within the range of 0-1.\nThe default value is typically set to 0.9. The ","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"NEC1bYWmN0"},{"type":"inlineCode","value":"-t","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"T44UUpsKMn"},{"type":"text","value":" option allows you to define the number of CPU threads, with a minimum requirement of 2.\nBy default, the software automatically calculates the available CPU threads and utilizes them collectively.","position":{"start":{"line":17,"column":1},"end":{"line":17,"column":1}},"key":"jM40oI8Kig"}],"key":"ShH825MXfQ"},{"type":"code","lang":"bash","value":"# run MEGAHIT \n$ conda activate MEGAHIT\n$ MEGAHIT -1 kneaddata.trimmed.1.fastq  -2 kneaddata.trimmed.2.fastq  -m 0.5  -t 12 -o MEGAHIT_result\n$ conda deactivate","position":{"start":{"line":25,"column":1},"end":{"line":30,"column":1}},"key":"HTyNzLsLdN"},{"type":"heading","depth":2,"position":{"start":{"line":32,"column":1},"end":{"line":32,"column":1}},"children":[{"type":"text","value":"Long reads assembly","position":{"start":{"line":32,"column":1},"end":{"line":32,"column":1}},"key":"f3mKjc4vem"}],"identifier":"long-reads-assembly","label":"Long reads assembly","html_id":"long-reads-assembly","implicit":true,"key":"lElxjL7CVc"},{"type":"paragraph","position":{"start":{"line":34,"column":1},"end":{"line":34,"column":1}},"children":[{"type":"text","value":"PacBio HiFi long reads (typically 10–20 kb with \u003e99% accuracy) enable the resolution of repetitive genomic regions and the recovery of near-complete metagenome-assembled genomes that are inaccessible through short-read assembly alone. The assembler ","position":{"start":{"line":34,"column":1},"end":{"line":34,"column":1}},"key":"qZxLsPBJTE"},{"type":"link","url":"https://github.com/lh3/hifiasm-meta","position":{"start":{"line":34,"column":1},"end":{"line":34,"column":1}},"children":[{"type":"text","value":"hifiasm-meta","position":{"start":{"line":34,"column":1},"end":{"line":34,"column":1}},"key":"yX80JaO4l5"}],"urlSource":"https://github.com/lh3/hifiasm-meta","error":true,"key":"AGZaU2OFAG"},{"type":"text","value":" is specifically designed for metagenomic HiFi data, producing high-quality primary and alternate contigs by leveraging the length and accuracy of HiFi reads to resolve strain-level haplotypes within complex microbial communities. Unlike short-read assemblers based on de Bruijn graphs, hifiasm-meta uses an overlap-based approach that better preserves long-range genomic structure.","position":{"start":{"line":34,"column":1},"end":{"line":34,"column":1}},"key":"GWzKURT2zJ"}],"key":"ZgiHCNuSsX"},{"type":"code","lang":"bash","value":"# run hifiasm_meta \n$ conda activate hifiam_meta\n$ hifiasm_meta -o N14.asm -t 60 N14.aln.unmapped.fq \u003e N14.log\n$ conda deactivate\n# .gfa 파일을 .fa 파일로 변환\n$ awk '/^S/{print \"\u003e\"$2;print $3}' test.p_ctg.gfa \u003e test.p_ctg.fa","position":{"start":{"line":36,"column":1},"end":{"line":43,"column":1}},"key":"KjhePGBmqA"},{"type":"paragraph","position":{"start":{"line":45,"column":1},"end":{"line":48,"column":1}},"children":[{"type":"text","value":"“.gfa” stands for graphical fragment assembly, a file format designed to store graph-based assembly information.\nIt contains data about the graph generated during the assembly process.\nThe structure of the graph file is akin to a de Bruijn graph, illustrating the connectivity of nodes and edges.\nThese graphs depict the interactions among overlapping DNA sequences, aiding in the assembly process.","position":{"start":{"line":45,"column":1},"end":{"line":45,"column":1}},"key":"H9OqJA3upY"}],"key":"LyKmNnClWI"},{"type":"paragraph","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"text","value":"When running ","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"key":"UOmRKVdZWX"},{"type":"link","url":"https://github.com/lh3/hifiasm-meta","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"text","value":"Hifiasm-meta","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"key":"NFFct0h8Ry"}],"urlSource":"https://github.com/lh3/hifiasm-meta","error":true,"key":"SFvDWWDPdw"},{"type":"text","value":", the output files are as follows.","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"key":"aemlNeEI8y"}],"key":"miLobeOlch"},{"type":"paragraph","position":{"start":{"line":53,"column":1},"end":{"line":58,"column":1}},"children":[{"type":"strong","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"text","value":"1. Raw unitig graph: asm.r_utg*.gfa","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"C6ClMKKu81"}],"key":"yghkFQxKLL"},{"type":"break","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"oiZFR878Pe"},{"type":"text","value":"This file contains the raw unitig graph generated during the assembly process. The raw unitig graph is the initial graph for assembly, which organizes nodes and edges based on overlapping sequence information.\n","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"em7PIHdpWt"},{"type":"strong","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"text","value":"2. Cleaned unitig graph: asm.p_utg*.gfa","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"aSTdCuwwz2"}],"key":"x5A8CDpnBm"},{"type":"break","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"Kqqjij1dKd"},{"type":"text","value":"This file contains preprocessed cleaned unitig graphs. Preprocessing refers to the process of removing errors from the raw unitig graph and improving its accuracy.\n","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"WQHqDgKhgI"},{"type":"strong","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"text","value":"3. Contig graph: asm.p_ctg*.gfa, asm.a_ctg*.gfa","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"oc3tGnX8vN"}],"key":"Lye6iSwzCO"},{"type":"break","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"RQkU5sc3lG"},{"type":"text","value":"This file contains the contig graph. The file asm.p_ctg*.gfa means primary contig and asm.a_ctg*.gfa means alternate contig.","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"lCY1JHjjeC"}],"key":"BbvGNeBha9"},{"type":"list","ordered":false,"spread":false,"position":{"start":{"line":61,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"listItem","spread":true,"position":{"start":{"line":61,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Unitig","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"key":"V70E44h62Q"},{"type":"break","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"key":"kQIAnphhfg"},{"type":"text","value":"A unitig, which stands for unique sequence, is a unit of overlapping DNA sequences that is the result of an intermediate step in assembly. Each unitig consists of one or more DNA sequences and contains overlapping parts. The assembly process generates a raw unitig graph. This raw unitig graph may contain some errors. The subsequent processing removes the errors from the raw unitig graph and generates a cleaned unitig graph.","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"key":"ynG5PTqCSb"}],"key":"wvzThh21zJ"}],"key":"YqKO0hGv9F"}],"key":"Crx9750hua"},{"type":"heading","depth":2,"position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"children":[{"type":"text","value":"Contig check","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"key":"hhC5PGNcOn"}],"identifier":"contig-check","label":"Contig check","html_id":"contig-check","implicit":true,"key":"ZzKo0R4Gcy"},{"type":"paragraph","position":{"start":{"line":65,"column":1},"end":{"line":66,"column":1}},"children":[{"type":"text","value":"Long read assembly results can be visualized using a bandage plot, which is a tool for representing complex assembly results, including De Bruijn graphs or overlap graphs.\nLong read assemblies often yield intricate graphs, and the bandage plot is a good option for visualizing these graphs.","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"key":"D6eiLKUGJA"}],"key":"L9NNkgkoWq"},{"type":"paragraph","position":{"start":{"line":68,"column":1},"end":{"line":68,"column":1}},"children":[{"type":"text","value":"It provides the following information.","position":{"start":{"line":68,"column":1},"end":{"line":68,"column":1}},"key":"VenEKiFNqj"}],"key":"bnpgopRn83"},{"type":"paragraph","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"children":[{"type":"text","value":"To generate bandage plots, you can use Bandage, a Bioinformatics Application for Navigating De novo Assembly Graphs Easily. Bandage is available for download at ","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"key":"a29yw59rgD"},{"type":"link","url":"http://rrwick.github.io/Bandage/","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"children":[{"type":"text","value":"http://​rrwick​.github​.io​/Bandage/","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"key":"eWF4d84zd6"}],"urlSource":"http://rrwick.github.io/Bandage/","key":"uRULpIBac5"},{"type":"text","value":".","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"key":"yumArDEiCq"}],"key":"vv4GZvAILP"},{"type":"paragraph","position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"children":[{"type":"text","value":"After running the Bandage program, load the *p_ctg.gfa file from File \u003e Load graph at the top. Once the file is loaded, click Draw graph in Graph drawing to display the graph.","position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"key":"r139hLZowS"}],"key":"h1ddmkEBKW"},{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/build/ce209dd902988924d189874ae25587b1.png","alt":"","data":{"altTextIsAutoGenerated":true},"key":"m53H0PQtrH","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_2.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[],"key":"AtdxHodLUR"}],"key":"SSqonlGtgP"}],"key":"kqczm1fNDi"}],"enumerator":"2","key":"HQXFe3V8y2"},{"type":"paragraph","position":{"start":{"line":82,"column":1},"end":{"line":82,"column":1}},"children":[{"type":"text","value":"After assembly, check the assembly result and quality using ","position":{"start":{"line":82,"column":1},"end":{"line":82,"column":1}},"key":"CpLqqM3gH9"},{"type":"link","url":"https://github.com/ablab/quast","position":{"start":{"line":82,"column":1},"end":{"line":82,"column":1}},"children":[{"type":"text","value":"QUAST","position":{"start":{"line":82,"column":1},"end":{"line":82,"column":1}},"key":"siMrzvVbua"}],"urlSource":"https://github.com/ablab/quast","error":true,"key":"PwwxlUEO4I"},{"type":"text","value":"(Quality assessment tool) to know the information of the produced contigs.","position":{"start":{"line":82,"column":1},"end":{"line":82,"column":1}},"key":"Y9nENKzslM"}],"key":"vac8k7vn9z"},{"type":"code","lang":"bash","value":"# calculate assembly statistics\n$ conda activate quast\n$ quast.py MEGAHIT_result/final.contigs.fa -o MEGAHIT_quast\n$ conda deactivate","position":{"start":{"line":84,"column":1},"end":{"line":89,"column":1}},"key":"hmGxjK8LT3"},{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/build/7b78d3d813c18d8e084f529f11c33e7f.png","alt":"Example of report.html","data":{"altTextIsAutoGenerated":true},"key":"aCinSCuBPl","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_3.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Example of report.html","key":"mdkChdm4dW"}],"key":"khGereKwtm"}],"key":"mHQlxS5cmr"}],"key":"vfDlOpXiQz"}],"enumerator":"3","key":"vywHCWl9K7"},{"type":"paragraph","position":{"start":{"line":97,"column":1},"end":{"line":97,"column":1}},"children":[{"type":"strong","position":{"start":{"line":97,"column":1},"end":{"line":97,"column":1}},"children":[{"type":"text","value":"contigs: Number of contigs produced after assembly","position":{"start":{"line":97,"column":1},"end":{"line":97,"column":1}},"key":"TJyackhOzD"}],"key":"ZM5wnvSRGG"}],"key":"eII0nZ38R8"},{"type":"list","ordered":false,"spread":false,"position":{"start":{"line":98,"column":1},"end":{"line":106,"column":1}},"children":[{"type":"listItem","spread":true,"position":{"start":{"line":98,"column":1},"end":{"line":98,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Largest contig: The length of the longest contig among the contigs produced after assembly","position":{"start":{"line":98,"column":1},"end":{"line":98,"column":1}},"key":"uMeA0uqwG1"}],"key":"Rp6UqjOCBk"}],"key":"VgOrqQJBOp"},{"type":"listItem","spread":true,"position":{"start":{"line":99,"column":1},"end":{"line":99,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Total length: Total number of bases of contigs generated after assembly","position":{"start":{"line":99,"column":1},"end":{"line":99,"column":1}},"key":"QRLqN7poQp"}],"key":"C7tgnajR1f"}],"key":"xESGW0SnGh"},{"type":"listItem","spread":true,"position":{"start":{"line":100,"column":1},"end":{"line":100,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"N50: Length of the contig with the top 50% length among the contigs generated after assembly","position":{"start":{"line":100,"column":1},"end":{"line":100,"column":1}},"key":"rnA68qwxwO"}],"key":"OwH5BJgoXf"}],"key":"BztvB24Pax"},{"type":"listItem","spread":true,"position":{"start":{"line":101,"column":1},"end":{"line":101,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"N75: Length of the contig with the top 75% length among the contigs generated after assembly","position":{"start":{"line":101,"column":1},"end":{"line":101,"column":1}},"key":"xJCygrF1NL"}],"key":"iDwV82dIX0"}],"key":"DXzBJLDAb4"},{"type":"listItem","spread":true,"position":{"start":{"line":102,"column":1},"end":{"line":102,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"L50: Number of contigs with the length of N50 among the contigs generated after assembly","position":{"start":{"line":102,"column":1},"end":{"line":102,"column":1}},"key":"oQJfo1whCj"}],"key":"g1Lrh0IUEk"}],"key":"xW3hvEQNII"},{"type":"listItem","spread":true,"position":{"start":{"line":103,"column":1},"end":{"line":103,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"L75: Number of contigs with the length of N75 among the contigs generated after assembly","position":{"start":{"line":103,"column":1},"end":{"line":103,"column":1}},"key":"diS924AytZ"}],"key":"gvrxyXOf6b"}],"key":"zAa4hJk9Yj"},{"type":"listItem","spread":true,"position":{"start":{"line":104,"column":1},"end":{"line":104,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Mismatches # N’s: Number of uncalled bases that were not assembled","position":{"start":{"line":104,"column":1},"end":{"line":104,"column":1}},"key":"xsQOa32iP2"}],"key":"GfEqFfdvbz"}],"key":"cO91d9yvgM"},{"type":"listItem","spread":true,"position":{"start":{"line":105,"column":1},"end":{"line":106,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Mismatches # N’s per 100kbp: Number of uncalled bases per 100000 bases","position":{"start":{"line":105,"column":1},"end":{"line":105,"column":1}},"key":"nOqpwyxRur"}],"key":"RdD1ig9juS"}],"key":"olWBsjKx7Y"}],"key":"BdG38VzzUT"},{"type":"paragraph","position":{"start":{"line":107,"column":1},"end":{"line":107,"column":1}},"children":[{"type":"text","value":"The left Graph shows cumulative length as the contig index increases and the right graph shows GC contents of contigs","position":{"start":{"line":107,"column":1},"end":{"line":107,"column":1}},"key":"zpKAYkZZiu"}],"key":"sTff7XtVTg"},{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/build/eaa6417f1aaec13589a70369854ace71.png","alt":"","data":{"altTextIsAutoGenerated":true},"key":"PKmmnhRZbr","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_7_4.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[],"key":"LWBK6mLKq8"}],"key":"CDzxwkMS2Z"}],"key":"ReEu2OlZLj"}],"enumerator":"4","key":"nIwyahlxm9"}],"key":"ByNKJTa7Ra"}],"key":"GqUL5g4npi"},"references":{"cite":{"order":["Meyer_2021"],"data":{"Meyer_2021":{"label":"Meyer_2021","enumerator":"1","doi":"10.1038/s41596-020-00480-3","html":"Meyer, F., Lesker, T.-R., Koslicki, D., Fritz, A., Gurevich, A., Darling, A. E., Sczyrba, A., Bremges, A., \u0026 McHardy, A. C. (2021). Tutorial: assessing metagenomics software with the CAMI benchmarking toolkit. \u003ci\u003eNature Protocols\u003c/i\u003e, \u003ci\u003e16\u003c/i\u003e(4), 1785–1801. \u003ca target=\"_blank\" rel=\"noreferrer\" href=\"https://doi.org/10.1038/s41596-020-00480-3\"\u003e10.1038/s41596-020-00480-3\u003c/a\u003e","url":"https://doi.org/10.1038/s41596-020-00480-3"}}}},"footer":{"navigation":{"prev":{"title":"Preprocessing of the sequencing reads","url":"/preprocessing","group":"Metagenomics"},"next":{"title":"Binning","url":"/binning","group":"Metagenomics"}}},"domain":"http://localhost:3000"},"project":{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/your-username/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}}},"actionData":null,"errors":null},"future":{"unstable_dev":false,"unstable_postcss":false,"unstable_tailwind":false,"v2_errorBoundary":true,"v2_headers":true,"v2_meta":true,"v2_normalizeFormMethod":true,"v2_routeConvention":true}};</script><script type="module" async="">import "/build/manifest-3595279D.js";
import * as route0 from "/build/root-SIO6LUTY.js";
import * as route1 from "/build/routes/$-PRP77N34.js";
window.__remixRouteModules = {"root":route0,"routes/$":route1};

import("/build/entry.client-PCJPW7TK.js");</script></body></html>