<!DOCTYPE html><html lang="en" class="" style="scroll-padding:60px"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis - META-Insight</title><meta property="og:title" content="Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis - META-Insight"/><meta name="generator" content="mystmd"/><meta name="description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta property="og:description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta name="keywords" content="metagenomics, metatranscriptomics, metaproteomics, microbiome, bioinformatics"/><meta name="image" content="/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png"/><meta property="og:image" content="/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png"/><link rel="stylesheet" href="/metainsight-book/build/_assets/app-MOQGDXHO.css"/><link rel="stylesheet" href="/metainsight-book/build/_assets/thebe-core-VKVHG5VY.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/jupyter-matplotlib@0.11.3/css/mpl_widget.css"/><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.css" integrity="sha384-MlJdn/WNKDGXveldHDdyRP1R4CTHr3FeuDNfhsLPYrq2t0UBkUdK2jyTnXPEK1NQ" crossorigin="anonymous"/><link rel="icon" href="/metainsight-book/favicon.ico"/><link rel="stylesheet" href="/metainsight-book/myst-theme.css"/><script>
  const savedTheme = localStorage.getItem("myst:theme");
  const theme = window.matchMedia("(prefers-color-scheme: light)").matches ? 'light' : 'dark';
  const classes = document.documentElement.classList;
  const hasAnyTheme = classes.contains('light') || classes.contains('dark');
  if (!hasAnyTheme) classes.add(savedTheme ?? theme);
</script></head><body class="m-0 transition-colors duration-500 bg-white dark:bg-stone-900"><div class="myst-skip-to-article fixed top-1 left-1 h-[0px] w-[0px] focus-within:z-40 focus-within:h-auto focus-within:w-auto bg-white overflow-hidden focus-within:p-2 focus-within:ring-1" aria-label="skip to content options"><a href="#skip-to-frontmatter" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article frontmatter</a><a href="#skip-to-article" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article content</a></div><dialog id="myst-no-css" style="position:fixed;left:0px;top:0px;width:100vw;height:100vh;font-size:4rem;padding:1rem;color:black;background:white"><strong>Site not loading correctly?</strong><p>This may be due to an incorrect <code>BASE_URL</code> configuration. See<!-- --> <a href="https://mystmd.org/guide/deployment#deploy-base-url">the MyST Documentation</a> <!-- -->for reference.</p><script>
    (() => {
            // Test for has-styling variable set by the MyST stylesheet
            const node = document.currentScript.parentNode;
            const hasCSS = window.getComputedStyle(node).getPropertyValue("--has-styling");
            if (hasCSS === ""){
                    node.showModal();
            }

    })()
</script></dialog><div class="myst-top-nav bg-white/80 backdrop-blur dark:bg-stone-900/80 shadow dark:shadow-stone-700 p-3 md:px-8 sticky w-screen top-0 z-30 h-[60px]"><nav class="myst-top-nav-bar flex items-center justify-between flex-nowrap max-w-[1440px] mx-auto"><div class="flex flex-row xl:min-w-[19.5rem] mr-2 sm:mr-7 justify-start items-center shrink-0"><div class="block xl:hidden"><button class="myst-top-nav-menu-button flex items-center justify-center border-stone-400 text-stone-800 hover:text-stone-900 dark:text-stone-200 hover:dark:text-stone-100 w-10 h-10"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem"><path fill-rule="evenodd" d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75ZM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12Zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75Z" clip-rule="evenodd"></path></svg><span class="sr-only">Open Menu</span></button></div><a class="myst-home-link flex items-center ml-3 dark:text-white w-fit md:ml-5 xl:ml-7" href="/metainsight-book/"><div class="myst-home-link-logo mr-3 flex items-center dark:bg-white dark:rounded px-1"><img src="/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg" class="h-9" height="2.25rem"/></div><span class="text-md sm:text-xl tracking-tight sm:mr-5 sr-only">Made with MyST</span></a></div><div class="flex items-center flex-grow w-auto"><div class="flex-grow hidden text-md lg:block"></div><div class="flex-grow block"></div><button type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R75cp:" data-state="closed" class="myst-search-bar flex items-center h-10 aspect-square sm:w-64 text-left text-gray-600 dark:text-gray-300 border border-gray-300 dark:border-gray-600 rounded-lg bg-gray-50 dark:bg-gray-700 myst-search-bar-disabled hover:ring-blue-500 dark:hover:ring-blue-500 hover:border-blue-500 dark:hover:border-blue-500"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="p-2.5 h-10 w-10 aspect-square"><path fill-rule="evenodd" d="M10.5 3.75a6.75 6.75 0 1 0 0 13.5 6.75 6.75 0 0 0 0-13.5ZM2.25 10.5a8.25 8.25 0 1 1 14.59 5.28l4.69 4.69a.75.75 0 1 1-1.06 1.06l-4.69-4.69A8.25 8.25 0 0 1 2.25 10.5Z" clip-rule="evenodd"></path></svg><span class="myst-search-text-placeholder hidden sm:block grow">Search</span><div aria-hidden="true" class="myst-search-shortcut items-center hidden mx-1 font-mono text-sm text-gray-600 dark:text-gray-300 sm:flex gap-x-1"><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none hide-mac">CTRL</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none show-mac">⌘</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none ">K</kbd><script>
;(() => {
const script = document.currentScript;
const root = script.parentElement;

const isMac = /mac/i.test(
      window.navigator.userAgentData?.platform ?? window.navigator.userAgent,
    );
root.querySelectorAll(".hide-mac").forEach(node => {node.classList.add(isMac ? "hidden" : "block")});
root.querySelectorAll(".show-mac").forEach(node => {node.classList.add(!isMac ? "hidden" : "block")});
})()</script></div></button><button class="myst-theme-button theme rounded-full aspect-square border border-stone-700 dark:border-white hover:bg-neutral-100 border-solid overflow-hidden text-stone-700 dark:text-white hover:text-stone-500 dark:hover:text-neutral-800 w-10 h-10 mx-3" title="Toggle theme between light and dark mode" aria-label="Toggle theme between light and dark mode"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-moon-icon h-full w-full p-0.5 hidden dark:block"><path fill-rule="evenodd" d="M9.528 1.718a.75.75 0 0 1 .162.819A8.97 8.97 0 0 0 9 6a9 9 0 0 0 9 9 8.97 8.97 0 0 0 3.463-.69.75.75 0 0 1 .981.98 10.503 10.503 0 0 1-9.694 6.46c-5.799 0-10.5-4.7-10.5-10.5 0-4.368 2.667-8.112 6.46-9.694a.75.75 0 0 1 .818.162Z" clip-rule="evenodd"></path></svg><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-sun-icon h-full w-full p-0.5 dark:hidden"><path stroke-linecap="round" stroke-linejoin="round" d="M12 3v2.25m6.364.386-1.591 1.591M21 12h-2.25m-.386 6.364-1.591-1.591M12 18.75V21m-4.773-4.227-1.591 1.591M5.25 12H3m4.227-4.773L5.636 5.636M15.75 12a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0Z"></path></svg></button><div class="block sm:hidden"></div><div class="hidden sm:block"></div></div></nav></div><div class="myst-primary-sidebar fixed xl:article-grid grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px] hidden z-10" style="top:60px"><div class="myst-primary-sidebar-pointer pointer-events-auto xl:col-margin-left flex-col overflow-hidden hidden xl:flex"><div class="myst-primary-sidebar-nav flex-grow py-6 overflow-y-auto primary-scrollbar"><nav aria-label="Navigation" class="myst-primary-sidebar-topnav overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px] lg:hidden"><div class="w-full px-1 dark:text-white font-medium"></div></nav><div class="my-3 border-b-2 lg:hidden"></div><nav aria-label="Table of Contents" class="myst-primary-sidebar-toc flex-grow overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px]"><div class="myst-toc w-full px-1 dark:text-white"><a title="META-Insight" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30 font-bold" href="/metainsight-book/">META-Insight</a><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metagenomics" class="block break-words rounded py-2 grow cursor-pointer">Metagenomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rmpsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rmpsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metatranscriptomics" class="block break-words rounded py-2 grow cursor-pointer">Metatranscriptomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rupsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rupsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metaproteomics" class="block break-words rounded py-2 grow cursor-pointer">Metaproteomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:R16psp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:R16psp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div></div></nav></div><div class="myst-primary-sidebar-footer flex-none py-6 transition-all duration-700 translate-y-6 opacity-0"><a class="myst-made-with-myst flex mx-auto text-gray-700 w-fit hover:text-blue-700 dark:text-gray-200 dark:hover:text-blue-400" href="https://mystmd.org/made-with-myst" target="_blank" rel="noreferrer"><svg style="width:24px;height:24px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" stroke="none"><g id="icon"><path fill="currentColor" d="M23.8,54.8v-3.6l4.7-0.8V17.5l-4.7-0.8V13H36l13.4,31.7h0.2l13-31.7h12.6v3.6l-4.7,0.8v32.9l4.7,0.8v3.6h-15
          v-3.6l4.9-0.8V20.8H65L51.4,53.3h-3.8l-14-32.5h-0.1l0.2,17.4v12.1l5,0.8v3.6H23.8z"></path><path fill="#F37726" d="M47,86.9c0-5.9-3.4-8.8-10.1-8.8h-8.4c-5.2,0-9.4-1.3-12.5-3.8c-3.1-2.5-5.4-6.2-6.8-11l4.8-1.6
          c1.8,5.6,6.4,8.6,13.8,8.8h9.2c6.4,0,10.8,2.5,13.1,7.5c2.3-5,6.7-7.5,13.1-7.5h8.4c7.8,0,12.7-2.9,14.6-8.7l4.8,1.6
          c-1.4,4.9-3.6,8.6-6.8,11.1c-3.1,2.5-7.3,3.7-12.4,3.8H63c-6.7,0-10,2.9-10,8.8"></path></g></svg><span class="self-center ml-2 text-sm">Made with MyST</span></a></div></div></div><main class="article-grid grid-gap"><article class="article-grid subgrid-gap col-screen article content"><div class="hidden"></div><div id="skip-to-frontmatter" aria-label="article frontmatter" class="myst-fm-block mb-8 pt-9"><div class="myst-fm-block-header flex items-center mb-5 h-6 text-sm font-light"><div class="flex-grow"></div><div class="myst-fm-block-badges"><a href="https://github.com/aababc1/metainsight-book" title="GitHub Repository: aababc1/metainsight-book" target="_blank" rel="noopener noreferrer" class="myst-fm-github-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" width="1.25rem" height="1.25rem" class="myst-fm-github-icon inline-block mr-1 opacity-60 hover:opacity-100"><path d="M12 2.5c-5.4 0-9.8 4.4-9.8 9.7 0 4.3 2.8 8 6.7 9.2.5.1.7-.2.7-.5v-1.8c-2.4.5-3.1-.6-3.3-1.1-.1-.3-.6-1.1-1-1.4-.3-.2-.8-.6 0-.6s1.3.7 1.5 1c.9 1.5 2.3 1.1 2.8.8.1-.6.3-1.1.6-1.3-2.2-.2-4.4-1.1-4.4-4.8 0-1.1.4-1.9 1-2.6-.1-.2-.4-1.2.1-2.6 0 0 .8-.3 2.7 1 .8-.2 1.6-.3 2.4-.3.8 0 1.7.1 2.4.3 1.9-1.3 2.7-1 2.7-1 .5 1.3.2 2.3.1 2.6.6.7 1 1.5 1 2.6 0 3.7-2.3 4.6-4.4 4.8.4.3.7.9.7 1.8V21c0 .3.2.6.7.5 3.9-1.3 6.6-4.9 6.6-9.2 0-5.4-4.4-9.8-9.8-9.8z"></path></svg></a></div><a href="https://github.com/aababc1/metainsight-book/edit/main/metatranscriptomics/Introduction.md" title="Edit This Page" target="_blank" rel="noopener noreferrer" class="myst-fm-edit-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-edit-icon inline-block mr-1 opacity-60 hover:opacity-100"><path stroke-linecap="round" stroke-linejoin="round" d="m16.862 4.487 1.687-1.688a1.875 1.875 0 1 1 2.652 2.652L10.582 16.07a4.5 4.5 0 0 1-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 0 1 1.13-1.897l8.932-8.931Zm0 0L19.5 7.125M18 14v4.75A2.25 2.25 0 0 1 15.75 21H5.25A2.25 2.25 0 0 1 3 18.75V8.25A2.25 2.25 0 0 1 5.25 6H10"></path></svg></a><div class="myst-fm-downloads-dropdown relative flex inline-block mx-1 grow-0" data-headlessui-state=""><button class="myst-fm-downloads-button relative ml-2 -mr-1" id="headlessui-menu-button-:Rs8ucp:" type="button" aria-haspopup="menu" aria-expanded="false" data-headlessui-state=""><span class="sr-only">Downloads</span><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-downloads-icon"><title>Download</title><path stroke-linecap="round" stroke-linejoin="round" d="M3 16.5v2.25A2.25 2.25 0 0 0 5.25 21h13.5A2.25 2.25 0 0 0 21 18.75V16.5M16.5 12 12 16.5m0 0L7.5 12m4.5 4.5V3"></path></svg></button></div></div><h1 class="myst-fm-block-title mb-0">Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis</h1><header class="myst-fm-authors-affiliations mt-4 not-prose"><div class="myst-fm-authors-list"><span class="myst-fm-author font-semibold text-sm myst-fm-author-item inline-block"><button class="myst-fm-author-popover focus:shadow-[0_0_0_2px] focus:shadow-black outline-none hover:underline" aria-label="Author Details" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R78ucp:" data-state="closed"><span class="myst-fm-author-name">META-Insight Team</span></button></span></div></header></div><div class="block my-10 lg:sticky lg:z-10 lg:h-0 lg:pt-0 lg:my-0 lg:ml-10 lg:col-margin-right" style="top:60px"><nav></nav></div><div id="skip-to-article"></div><h2 id="introduction-to-metatranscriptome" class="relative group"><span class="heading-text">Introduction to Metatranscriptome</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#introduction-to-metatranscriptome" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><p>The term microbiome refers to the concept encompassing a community of microorganisms (microbiota) residing in a particular environment, along with their complete genetic information. Microbiome research traditionally involves the amplification and sequencing of conserved regions of marker genes, such as ribosomal genes, from environmental DNA using PCR, allowing for the identification of the types of microorganisms present in the environment. In contrast, microbiome research utilizes whole DNA sequence analysis, known as metagenome sequencing, which not only identifies the types of microorganisms present in the environment but also enables the exploration of the genetic diversity and functions possessed by these microorganisms. However, metagenomic (metagenome) data can inform about the presence or absence of specific species or genes but cannot determine whether they are active within the microbiome ecosystem, posing limitations.</p><figure id="n579JzXx7C" class=""><img id="mOwgMeaBPm" style="margin-left:auto;margin-right:auto" src="/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png" alt="Multi-omics information-based microbiome functional analysis" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaTranscriptomic/img/T_0_1.png?raw=true" class=""/><figcaption class="group"><p><strong>Multi-omics information-based microbiome functional analysis</strong></p></figcaption></figure><p>To investigate how microbial communities respond to environmental changes, metatranscriptome analysis has been initiated, and metatranscriptomic information will play a crucial role in understanding the ecology of the microbiome. Even before the advent of high-throughput DNA sequencing methods, research has been conducted to measure the expression levels of genes of interest using qPCR and to study the expression patterns of known transcripts from specific organisms using microarray technology. With the widespread adoption of Next-Generation Sequencing (NGS) technology, RNA sequencing (RNA-seq) has not only allowed for the measurement of the expression levels of known transcript targets but has also made it possible to obtain information on the entire transcriptome, including previously unknown transcripts and transcript variants, such as non-coding RNAs, in a specific environment. For these reasons, RNA-seq is increasingly prevalent in microbiome research and finds applications in various fields, including gene expression analysis, identification of highly active microbial community members, discovery of regulatory factors, investigation of microbial interactions, and studies on interactions with hosts.</p><p>Based on these requirements, RNA-seq is being increasingly utilized in microbiome research. In this Standard Operating Procedure (SOP), we aim to describe reference-guided metatranscriptome analysis and reference-independent metatranscriptome analysis methods based on mapping RNA-seq reads to de novo assembled contigs and metagenome-assembled genomes (MAGs) obtained from metagenomes.</p><figure id="z6QQaEJhwl" class=""><img id="SzsRdLrbim" style="margin-left:auto;margin-right:auto" src="/metainsight-book/build/150b6f28f8d688fa730cf83e0d36556c.png" alt="Reference-guided and reference-independent metatranscriptome analysis methods" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaTranscriptomic/img/T_0_2.png?raw=true" class=""/><figcaption class="group"><p><strong>Reference-guided and reference-independent metatranscriptome analysis methods</strong></p></figcaption></figure><p>Before diving into this comprehensive SOP, various tools that can be utilized for metatranscriptome analysis are introduced. In addition to the tools presented in this SOP, other tools listed in Table 1 or similar tools with similar purposes can be selectively applied at each step.</p><p><strong>Table 1. List of available RNA-seq analysis tools</strong></p><table class=""><tbody><tr class=""><th class="">Category</th><th class="">Tools</th><th class="">Description</th><th class="">Year</th></tr><tr class=""><td class="" rowspan="3">Trimming</td><td class=""><a target="_blank" rel="noreferrer" href="https://cutadapt.readthedocs.io/en/stable/" class="link">cutadapt<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">5&#x27; adaptor trimming</td><td class="">2011</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://github.com/FelixKrueger/TrimGalore" class="link">TrimGalore<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Wrapper around Cutadapt and FastQC<br/>paired-end trimming</td><td class="">2013</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="http://www.usadellab.org/cms/?page=trimmomatic" class="link">trimmomatic<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Paired-end trimming</td><td class="">2014</td></tr><tr class=""><td class="" rowspan="5">Alignment</td><td class=""><a target="_blank" rel="noreferrer" href="https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml" class="link">bowtie2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Short read mapping, local alignment, use FM-index algorithm</td><td class="">2012</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://github.com/lh3/bwa" class="link">BWA<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Low-divergent sequences mapping, need genome reference, use BWT and FM index algorithm</td><td class="">2009</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://daehwankimlab.github.io/hisat2/manual/" class="link">HiSat2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Highly useful for human data analysis. Utilized in tasks like Human Leukocyte Antigen (HLA) typing and DNA fingerprinting. Use the FM index.</td><td class="">2019</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://github.com/alexdobin/STAR" class="link">STAR<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Improve <em>de novo</em> RNA-seq analysis and allow for the mapping of long-read mRNA data.</td><td class="">2013</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://ccb.jhu.edu/software/tophat/manual.shtml" class="link">TopHat2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Bowtie-based short read mapping</td><td class="">2013</td></tr><tr class=""><td class="" rowspan="6">DE analysis</td><td class=""><a target="_blank" rel="noreferrer" href="https://cole-trapnell-lab.github.io/cufflinks/manual/" class="link">Cuffdiiff2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">cufflink-associated tool. utilize sam(bam)file to compare gene and isoform expression</td><td class="">2013</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" class="link">DESeq2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">R package, employ shrinkage estimation techniques and utilizes RLE (Relative Log Expression) normalization.</td><td class="">2014</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://bioconductor.org/packages/release/bioc/html/edgeR.html" class="link">EdgeR<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">R package, Use raw count value as input file. TMM(Trimmed Mean M-values) normalization ���</td><td class="">2010</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://bioconductor.org/packages/release/bioc/html/limma.html" class="link">limma<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">R package. Effective for a small number of samples, using TMM normalization</td><td class="">2015</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html" class="link">NOISeq<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">R package. Using a nonparametric approach and an RPKM/TMM/uppartile normalization approach</td><td class="">2011</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://github.com/jefferys/SamSeq" class="link">SAMSeq<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">R package. Gene-level DE analysis using sam file as input file. Normalization based on the mean value of the total count</td><td class="">2013</td></tr><tr class=""><td class="" rowspan="3"><em>de novo</em> assembly</td><td class=""><a target="_blank" rel="noreferrer" href="https://github.com/voutcn/megahit" class="link">megahit<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Single genome assembly, Multiple library analysis is possible</td><td class="">2015</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://cab.spbu.ru/software/spades/" class="link">SPAdes<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Various types of data, such as RNA, virus RNA, plasmid, etc., can be assembled. Not only fastq, fasta files but also bam files can be inputted</td><td class="">2012</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://github.com/trinityrnaseq/trinityrnaseq/wiki" class="link">trinity<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Transcriptome assembler</td><td class="">2011</td></tr><tr class=""><td class="" rowspan="4">Taxonomy assignment</td><td class=""><a target="_blank" rel="noreferrer" href="https://bioinformatics-centre.github.io/kaiju/" class="link">kaiju<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Analyze the entire sequence of the meta-genome or the sequence of the meta-transcript using the GenBank protein-redundant database</td><td class="">2016</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://ccb.jhu.edu/software/kraken/MANUAL.html" class="link">kraken<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Tazonomy analysis using GenBank nucleotide non-redundant database</td><td class="">2014</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://huttenhower.sph.harvard.edu/metaphlan3/" class="link">MetaPhlAn<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Analysis of microbial composition from meta-genetic information at species level. Use your own marker genes</td><td class="">2012</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://www.onecodex.com/" class="link">onecodex<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Web-based analysis tool using Genome database and targeted loci database</td><td class="">2015</td></tr><tr class=""><td class="" rowspan="4">Format transition</td><td class=""><a target="_blank" rel="noreferrer" href="https://bedtools.readthedocs.io/en/latest/" class="link">BEDTools<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Software using bam or bed files. Raw count value can be obtained, but a candidate is required</td><td class="">2010</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://cole-trapnell-lab.github.io/cufflinks/manual/" class="link">Cufflink<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Tool that assembles RNA-seq mapping results (sam file) and calculates an unbalance haved FPKM value as result</td><td class="">2010</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://htseq.readthedocs.io/en/release_0.11.1/index.html" class="link">HTSeq<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Python package, calculation of read mapping using gff information</td><td class="">2015</td></tr><tr class=""><td class=""><a target="_blank" rel="noreferrer" href="https://www.htslib.org/" class="link">SAMTools<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td><td class="">Software that converts or sorts sam files</td><td class="">2009</td></tr></tbody></table><div class="myst-backmatter-parts"></div><div class="myst-footer-links flex pt-10 mb-10 space-x-4"><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-prev" href="/metainsight-book/read-based"><div class="flex h-full align-middle"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:-translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5 3 12m0 0 7.5-7.5M3 12h18"></path></svg><div class="flex-grow text-right"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metagenomics</div>Read-based profiling</div></div></a><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-next" href="/metainsight-book/sequencing-1"><div class="flex h-full align-middle"><div class="flex-grow"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metatranscriptomics</div>Sequencing</div><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 4.5 21 12m0 0-7.5 7.5M21 12H3"></path></svg></div></a></div></article></main><script>((a,l)=>{if(!window.history.state||!window.history.state.key){let u=Math.random().toString(32).slice(2);window.history.replaceState({key:u},"")}try{let d=JSON.parse(sessionStorage.getItem(a)||"{}")[l||window.history.state.key];typeof d=="number"&&window.scrollTo(0,d)}catch(u){console.error(u),sessionStorage.removeItem(a)}})("positions", null)</script><link rel="modulepreload" href="/metainsight-book/build/entry.client-PCJPW7TK.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-AQ2CODAG.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-JJXTQVMA.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-OZE3FFNP.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-CH4FVTDV.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-C4DFGG5C.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-J7TUH54J.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-FZ2S7OYD.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-JEM6JXYA.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-34XIY2DH.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-KQM5FBHR.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-OCWQY3HK.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-7HNKBP4B.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-CUKUDK3R.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-3EBOCCHJ.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-O4VQNZ62.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-4OEDG4JQ.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-GUCIBHGO.js"/><link rel="modulepreload" href="/metainsight-book/build/root-SIO6LUTY.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-FAHZZXAC.js"/><link rel="modulepreload" href="/metainsight-book/build/routes/$-PRP77N34.js"/><script>window.__remixContext = {"url":"/introduction-1","state":{"loaderData":{"root":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg","logo":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"CONTENT_CDN_PORT":"3100","MODE":"static","BASE_URL":"/metainsight-book"},"routes/$":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg","logo":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"page":{"version":3,"kind":"Article","sha256":"37955a095490bb151e82564de6d2ec933c81af9488e3ca3820466a9453030414","slug":"introduction-1","location":"/metatranscriptomics/Introduction.md","dependencies":[],"frontmatter":{"title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","content_includes_title":false,"authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"numbering":{"title":{"offset":1}},"source_url":"https://github.com/aababc1/metainsight-book/blob/main/metatranscriptomics/Introduction.md","edit_url":"https://github.com/aababc1/metainsight-book/edit/main/metatranscriptomics/Introduction.md","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","exports":[{"format":"md","filename":"Introduction.md","url":"/metainsight-book/build/Introduction-899bca5e73ff46d65af4ad081d57d00b.md"}]},"mdast":{"type":"root","children":[{"type":"block","children":[{"type":"heading","depth":2,"position":{"start":{"line":3,"column":1},"end":{"line":3,"column":1}},"children":[{"type":"text","value":"Introduction to Metatranscriptome","position":{"start":{"line":3,"column":1},"end":{"line":3,"column":1}},"key":"GE5ABNpaOd"}],"identifier":"introduction-to-metatranscriptome","label":"Introduction to Metatranscriptome","html_id":"introduction-to-metatranscriptome","implicit":true,"key":"VRagVOf7BF"},{"type":"paragraph","position":{"start":{"line":5,"column":1},"end":{"line":5,"column":1}},"children":[{"type":"text","value":"The term microbiome refers to the concept encompassing a community of microorganisms (microbiota) residing in a particular environment, along with their complete genetic information. Microbiome research traditionally involves the amplification and sequencing of conserved regions of marker genes, such as ribosomal genes, from environmental DNA using PCR, allowing for the identification of the types of microorganisms present in the environment. In contrast, microbiome research utilizes whole DNA sequence analysis, known as metagenome sequencing, which not only identifies the types of microorganisms present in the environment but also enables the exploration of the genetic diversity and functions possessed by these microorganisms. However, metagenomic (metagenome) data can inform about the presence or absence of specific species or genes but cannot determine whether they are active within the microbiome ecosystem, posing limitations.","position":{"start":{"line":5,"column":1},"end":{"line":5,"column":1}},"key":"Z0KpEwfLxP"}],"key":"QOHlDI3ViA"},{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","alt":"Multi-omics information-based microbiome functional analysis","data":{"altTextIsAutoGenerated":true},"key":"mOwgMeaBPm","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaTranscriptomic/img/T_0_1.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Multi-omics information-based microbiome functional analysis","key":"hAajHBdsW6"}],"key":"JfAORRv2w4"}],"key":"OPCXbotzo2"}],"key":"DJeApZUfqV"}],"enumerator":"1","key":"n579JzXx7C"},{"type":"paragraph","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"children":[{"type":"text","value":"To investigate how microbial communities respond to environmental changes, metatranscriptome analysis has been initiated, and metatranscriptomic information will play a crucial role in understanding the ecology of the microbiome. Even before the advent of high-throughput DNA sequencing methods, research has been conducted to measure the expression levels of genes of interest using qPCR and to study the expression patterns of known transcripts from specific organisms using microarray technology. With the widespread adoption of Next-Generation Sequencing (NGS) technology, RNA sequencing (RNA-seq) has not only allowed for the measurement of the expression levels of known transcript targets but has also made it possible to obtain information on the entire transcriptome, including previously unknown transcripts and transcript variants, such as non-coding RNAs, in a specific environment. For these reasons, RNA-seq is increasingly prevalent in microbiome research and finds applications in various fields, including gene expression analysis, identification of highly active microbial community members, discovery of regulatory factors, investigation of microbial interactions, and studies on interactions with hosts.","position":{"start":{"line":12,"column":1},"end":{"line":12,"column":1}},"key":"hEhAA7PSrL"}],"key":"WQMIPFnS6Y"},{"type":"paragraph","position":{"start":{"line":14,"column":1},"end":{"line":14,"column":1}},"children":[{"type":"text","value":"Based on these requirements, RNA-seq is being increasingly utilized in microbiome research. In this Standard Operating Procedure (SOP), we aim to describe reference-guided metatranscriptome analysis and reference-independent metatranscriptome analysis methods based on mapping RNA-seq reads to de novo assembled contigs and metagenome-assembled genomes (MAGs) obtained from metagenomes.","position":{"start":{"line":14,"column":1},"end":{"line":14,"column":1}},"key":"E6ROLC4p5I"}],"key":"ei4D3rx64T"},{"type":"container","children":[{"type":"image","style":{"width":"65%"},"url":"/metainsight-book/build/150b6f28f8d688fa730cf83e0d36556c.png","alt":"Reference-guided and reference-independent metatranscriptome analysis methods","data":{"altTextIsAutoGenerated":true},"key":"SzsRdLrbim","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaTranscriptomic/img/T_0_2.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Reference-guided and reference-independent metatranscriptome analysis methods","key":"DOeNYFqVqq"}],"key":"yiRBUwTkmE"}],"key":"B718nDa2Rp"}],"key":"aEdceY6RCm"}],"enumerator":"2","key":"z6QQaEJhwl"},{"type":"paragraph","position":{"start":{"line":21,"column":1},"end":{"line":21,"column":1}},"children":[{"type":"text","value":"Before diving into this comprehensive SOP, various tools that can be utilized for metatranscriptome analysis are introduced. In addition to the tools presented in this SOP, other tools listed in Table 1 or similar tools with similar purposes can be selectively applied at each step.","position":{"start":{"line":21,"column":1},"end":{"line":21,"column":1}},"key":"coprHjltGM"}],"key":"IPzzpI5WNa"},{"type":"paragraph","position":{"start":{"line":23,"column":1},"end":{"line":23,"column":1}},"children":[{"type":"strong","position":{"start":{"line":23,"column":1},"end":{"line":23,"column":1}},"children":[{"type":"text","value":"Table 1. List of available RNA-seq analysis tools","position":{"start":{"line":23,"column":1},"end":{"line":23,"column":1}},"key":"N59KaV7Mzt"}],"key":"PsHpCXaafM"}],"key":"t4AV3GdDIv"},{"type":"table","children":[{"type":"tableRow","children":[{"type":"tableCell","header":true,"children":[{"type":"text","value":"Category","key":"cWD9IW13F2"}],"key":"jBApR3noqM"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"Tools","key":"ikyLdZUuGH"}],"key":"UATzbvG9p3"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"Description","key":"mOIiCCDopq"}],"key":"gZ49USaEc2"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"Year","key":"j5m2riILyB"}],"key":"NlGCk3H6iL"}],"key":"HUU5qArocC"},{"type":"tableRow","children":[{"type":"tableCell","rowspan":3,"children":[{"type":"text","value":"Trimming","key":"ADQlzOE6kX"}],"key":"RzHez8PFVE"},{"type":"tableCell","children":[{"type":"link","url":"https://cutadapt.readthedocs.io/en/stable/","children":[{"type":"text","value":"cutadapt","key":"NYT1CnqnCy"}],"urlSource":"https://cutadapt.readthedocs.io/en/stable/","key":"qNhexMy7ET"}],"key":"eTobsrU00Q"},{"type":"tableCell","children":[{"type":"text","value":"5' adaptor trimming","key":"MymXeIIHgH"}],"key":"nchuOTlnXu"},{"type":"tableCell","children":[{"type":"text","value":"2011","key":"hKtdc1bY18"}],"key":"FFh0eIPfIt"}],"key":"Q9faoolYZL"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://github.com/FelixKrueger/TrimGalore","children":[{"type":"text","value":"TrimGalore","key":"xjykqtYgyq"}],"urlSource":"https://github.com/FelixKrueger/TrimGalore","error":true,"key":"zbEsC8IkNf"}],"key":"oM8ZVWiRkL"},{"type":"tableCell","children":[{"type":"text","value":"Wrapper around Cutadapt and FastQC","key":"kkSJHpkV41"},{"type":"break","key":"IMgBpT4050"},{"type":"text","value":"paired-end trimming","key":"HYVRjIjKXp"}],"key":"gZc4nBRu1d"},{"type":"tableCell","children":[{"type":"text","value":"2013","key":"RNKqZi4qiJ"}],"key":"hEG1tsYcOb"}],"key":"G5GbxBJcki"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"http://www.usadellab.org/cms/?page=trimmomatic","children":[{"type":"text","value":"trimmomatic","key":"oLKCUO7TkD"}],"urlSource":"http://www.usadellab.org/cms/?page=trimmomatic","key":"ETQmAtAS9H"}],"key":"slVty1vVFQ"},{"type":"tableCell","children":[{"type":"text","value":"Paired-end trimming","key":"OIxRiM8M15"}],"key":"W0xwuuDnOj"},{"type":"tableCell","children":[{"type":"text","value":"2014","key":"WoXnb1wwzO"}],"key":"Vhi1eJHcxX"}],"key":"CckmTogF53"},{"type":"tableRow","children":[{"type":"tableCell","rowspan":5,"children":[{"type":"text","value":"Alignment","key":"Av5Za0tTri"}],"key":"DnB59MX6FV"},{"type":"tableCell","children":[{"type":"link","url":"https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml","children":[{"type":"text","value":"bowtie2","key":"jG99p39IZL"}],"urlSource":"https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml","key":"SkLgi3cSLj"}],"key":"idOwLV5i3s"},{"type":"tableCell","children":[{"type":"text","value":"Short read mapping, local alignment, use FM-index algorithm","key":"zywPrikgmT"}],"key":"pkbkEYMMeC"},{"type":"tableCell","children":[{"type":"text","value":"2012","key":"iOkxzQN5pH"}],"key":"JwHRcQKG1P"}],"key":"jjBTEdjaVf"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://github.com/lh3/bwa","children":[{"type":"text","value":"BWA","key":"I49ZsZ4Ygd"}],"urlSource":"https://github.com/lh3/bwa","error":true,"key":"DVdJ5cFqS4"}],"key":"m1WqnbnJ2y"},{"type":"tableCell","children":[{"type":"text","value":"Low-divergent sequences mapping, need genome reference, use BWT and FM index algorithm","key":"kpjyyLzqtF"}],"key":"mzvMDqusQU"},{"type":"tableCell","children":[{"type":"text","value":"2009","key":"IECtwW05x1"}],"key":"kQMKAPMgmp"}],"key":"abxBJMg75z"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://daehwankimlab.github.io/hisat2/manual/","children":[{"type":"text","value":"HiSat2","key":"hFe8Q0Zs36"}],"urlSource":"https://daehwankimlab.github.io/hisat2/manual/","key":"mnWCslM98O"}],"key":"GjavbmaxuK"},{"type":"tableCell","children":[{"type":"text","value":"Highly useful for human data analysis. Utilized in tasks like Human Leukocyte Antigen (HLA) typing and DNA fingerprinting. Use the FM index.","key":"dnVaUspXVl"}],"key":"exWUEMrnAJ"},{"type":"tableCell","children":[{"type":"text","value":"2019","key":"YbFZyAQXDL"}],"key":"bz67v2OexQ"}],"key":"PbJtvKn1KI"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://github.com/alexdobin/STAR","children":[{"type":"text","value":"STAR","key":"iQWNsknIHX"}],"urlSource":"https://github.com/alexdobin/STAR","error":true,"key":"Y3i8M15fDC"}],"key":"k2n56eA0La"},{"type":"tableCell","children":[{"type":"text","value":"Improve ","key":"koxmUaIDSt"},{"type":"emphasis","children":[{"type":"text","value":"de novo","key":"RpuFD9Pp2u"}],"key":"c7QmWA177z"},{"type":"text","value":" RNA-seq analysis and allow for the mapping of long-read mRNA data.","key":"Qc449EOTpw"}],"key":"MwyVOsh8aD"},{"type":"tableCell","children":[{"type":"text","value":"2013","key":"gfdDqLbZKz"}],"key":"c2ZJlLwZ5o"}],"key":"fHvOa50l0N"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://ccb.jhu.edu/software/tophat/manual.shtml","children":[{"type":"text","value":"TopHat2","key":"WT3GBoR6d0"}],"urlSource":"https://ccb.jhu.edu/software/tophat/manual.shtml","key":"y6Xh4GRY18"}],"key":"a63dkdfr32"},{"type":"tableCell","children":[{"type":"text","value":"Bowtie-based short read mapping","key":"iGPpVNlKZT"}],"key":"vQcIJsA9WQ"},{"type":"tableCell","children":[{"type":"text","value":"2013","key":"Auc438B1yB"}],"key":"tMi2jWE1ik"}],"key":"k8M1lTrkqq"},{"type":"tableRow","children":[{"type":"tableCell","rowspan":6,"children":[{"type":"text","value":"DE analysis","key":"kjJfgoE0h0"}],"key":"K9y0V1F2rZ"},{"type":"tableCell","children":[{"type":"link","url":"https://cole-trapnell-lab.github.io/cufflinks/manual/","children":[{"type":"text","value":"Cuffdiiff2","key":"pwyUOrnxZa"}],"urlSource":"https://cole-trapnell-lab.github.io/cufflinks/manual/","key":"BJXrRropYX"}],"key":"kbq1TFIUbe"},{"type":"tableCell","children":[{"type":"text","value":"cufflink-associated tool. utilize sam(bam)file to compare gene and isoform expression","key":"ND7b1tZIM0"}],"key":"gv73wMD4u4"},{"type":"tableCell","children":[{"type":"text","value":"2013","key":"og58R9wIqh"}],"key":"zqMkcrDoIt"}],"key":"d0xkQuIClQ"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://bioconductor.org/packages/release/bioc/html/DESeq2.html","children":[{"type":"text","value":"DESeq2","key":"Rkb5PNWuNh"}],"urlSource":"https://bioconductor.org/packages/release/bioc/html/DESeq2.html","key":"NuNzrTrdeO"}],"key":"bQUMf15mRf"},{"type":"tableCell","children":[{"type":"text","value":"R package, employ shrinkage estimation techniques and utilizes RLE (Relative Log Expression) normalization.","key":"AsgkLabRzK"}],"key":"TbaO08q6wc"},{"type":"tableCell","children":[{"type":"text","value":"2014","key":"LNrEncecxB"}],"key":"OsZX31xAF3"}],"key":"WyoSSeN11x"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://bioconductor.org/packages/release/bioc/html/edgeR.html","children":[{"type":"text","value":"EdgeR","key":"E19GUc4km9"}],"urlSource":"https://bioconductor.org/packages/release/bioc/html/edgeR.html","key":"HdHzBhv2mo"}],"key":"B3YBNzis4W"},{"type":"tableCell","children":[{"type":"text","value":"R package, Use raw count value as input file. TMM(Trimmed Mean M-values) normalization ���","key":"zoSXrle9xF"}],"key":"QXbzjM4lXN"},{"type":"tableCell","children":[{"type":"text","value":"2010","key":"HPGgwTMAAu"}],"key":"WLYnIrLbTs"}],"key":"AmjS3Rpn3f"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://bioconductor.org/packages/release/bioc/html/limma.html","children":[{"type":"text","value":"limma","key":"SHhYXMF9VS"}],"urlSource":"https://bioconductor.org/packages/release/bioc/html/limma.html","key":"lp3ShYEl6e"}],"key":"XgtXSUPRb3"},{"type":"tableCell","children":[{"type":"text","value":"R package. Effective for a small number of samples, using TMM normalization","key":"eghXsTchDb"}],"key":"aHuJFe1bTR"},{"type":"tableCell","children":[{"type":"text","value":"2015","key":"Q9gUZyM4V8"}],"key":"DuJ4odRVGd"}],"key":"mxURulIUkM"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html","children":[{"type":"text","value":"NOISeq","key":"xwvRrpHmCQ"}],"urlSource":"https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html","key":"L6wXo1MLDp"}],"key":"L484enxaFr"},{"type":"tableCell","children":[{"type":"text","value":"R package. Using a nonparametric approach and an RPKM/TMM/uppartile normalization approach","key":"znOpv68ujf"}],"key":"NPICw0GjVJ"},{"type":"tableCell","children":[{"type":"text","value":"2011","key":"DyycG43549"}],"key":"uqynfCYB6U"}],"key":"ITxNv0b4dA"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://github.com/jefferys/SamSeq","children":[{"type":"text","value":"SAMSeq","key":"thpq89hxoX"}],"urlSource":"https://github.com/jefferys/SamSeq","error":true,"key":"ejxekQ4dDg"}],"key":"ZScPKwNwTf"},{"type":"tableCell","children":[{"type":"text","value":"R package. Gene-level DE analysis using sam file as input file. Normalization based on the mean value of the total count","key":"r6wrKhyJp9"}],"key":"mCqg5qjTpF"},{"type":"tableCell","children":[{"type":"text","value":"2013","key":"xGIov5Qxc0"}],"key":"ma2dCkoRiG"}],"key":"okiMXMsn60"},{"type":"tableRow","children":[{"type":"tableCell","rowspan":3,"children":[{"type":"emphasis","children":[{"type":"text","value":"de novo","key":"FFAer2H13q"}],"key":"Xz9AZnkWEH"},{"type":"text","value":" assembly","key":"qnTZu6aMJT"}],"key":"OUWMI5rQhE"},{"type":"tableCell","children":[{"type":"link","url":"https://github.com/voutcn/megahit","children":[{"type":"text","value":"megahit","key":"iyUYaF11RD"}],"urlSource":"https://github.com/voutcn/megahit","error":true,"key":"BsEmlkRr9Q"}],"key":"ZNfthnQXVs"},{"type":"tableCell","children":[{"type":"text","value":"Single genome assembly, Multiple library analysis is possible","key":"htYAzwRGdC"}],"key":"IYlUWSOrrm"},{"type":"tableCell","children":[{"type":"text","value":"2015","key":"fthCTNe1Km"}],"key":"uE12l6kClI"}],"key":"xMRd8d7k56"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://cab.spbu.ru/software/spades/","children":[{"type":"text","value":"SPAdes","key":"syPJQwh7GM"}],"urlSource":"https://cab.spbu.ru/software/spades/","key":"a49OO2l3Vl"}],"key":"slj1E07iFK"},{"type":"tableCell","children":[{"type":"text","value":"Various types of data, such as RNA, virus RNA, plasmid, etc., can be assembled. Not only fastq, fasta files but also bam files can be inputted","key":"eBCT13nhDO"}],"key":"nhnUqe6xFy"},{"type":"tableCell","children":[{"type":"text","value":"2012","key":"ZOCN67DlIB"}],"key":"hIOcJvCgtZ"}],"key":"GpQsPsrM10"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://github.com/trinityrnaseq/trinityrnaseq/wiki","children":[{"type":"text","value":"trinity","key":"XdLAEFHApu"}],"urlSource":"https://github.com/trinityrnaseq/trinityrnaseq/wiki","error":true,"key":"PoXDjaRruM"}],"key":"LnkHqI1jZ0"},{"type":"tableCell","children":[{"type":"text","value":"Transcriptome assembler","key":"qB6cChqH9o"}],"key":"RGrEdSbawG"},{"type":"tableCell","children":[{"type":"text","value":"2011","key":"Ml7AOBtqfy"}],"key":"X5WfxVBQeX"}],"key":"sOfN7e6rwT"},{"type":"tableRow","children":[{"type":"tableCell","rowspan":4,"children":[{"type":"text","value":"Taxonomy assignment","key":"gye2FwCnOm"}],"key":"tbUjxcwkhA"},{"type":"tableCell","children":[{"type":"link","url":"https://bioinformatics-centre.github.io/kaiju/","children":[{"type":"text","value":"kaiju","key":"Je75WyEl1A"}],"urlSource":"https://bioinformatics-centre.github.io/kaiju/","key":"vVwQLA8gOk"}],"key":"owxaICNLN5"},{"type":"tableCell","children":[{"type":"text","value":"Analyze the entire sequence of the meta-genome or the sequence of the meta-transcript using the GenBank protein-redundant database","key":"UTDqblM7FN"}],"key":"EVjt7HhFNJ"},{"type":"tableCell","children":[{"type":"text","value":"2016","key":"tnjv8x0mqQ"}],"key":"BPPu7rFRZv"}],"key":"jDXnpEWV2P"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://ccb.jhu.edu/software/kraken/MANUAL.html","children":[{"type":"text","value":"kraken","key":"GXX5XTDkuX"}],"urlSource":"https://ccb.jhu.edu/software/kraken/MANUAL.html","key":"GbSYGOt6MC"}],"key":"NsziNzkASj"},{"type":"tableCell","children":[{"type":"text","value":"Tazonomy analysis using GenBank nucleotide non-redundant database","key":"cHa1XHq2Eo"}],"key":"xBvoRO9LeJ"},{"type":"tableCell","children":[{"type":"text","value":"2014","key":"ia7xFEISQc"}],"key":"qBZ7DcoHqI"}],"key":"m67JhckjI5"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://huttenhower.sph.harvard.edu/metaphlan3/","children":[{"type":"text","value":"MetaPhlAn","key":"h5uQ6iIBea"}],"urlSource":"https://huttenhower.sph.harvard.edu/metaphlan3/","key":"sN3ZTuRWQc"}],"key":"WAPaM8F20M"},{"type":"tableCell","children":[{"type":"text","value":"Analysis of microbial composition from meta-genetic information at species level. Use your own marker genes","key":"q0oRCGmImA"}],"key":"mklb6P1MW6"},{"type":"tableCell","children":[{"type":"text","value":"2012","key":"ITSSQ3wayW"}],"key":"UA4KOIPjQw"}],"key":"Pj4JHQtsBQ"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://www.onecodex.com/","children":[{"type":"text","value":"onecodex","key":"WtAR9RKbZ9"}],"urlSource":"https://www.onecodex.com/","key":"w3oazAbx7V"}],"key":"vnjW9Dwnjh"},{"type":"tableCell","children":[{"type":"text","value":"Web-based analysis tool using Genome database and targeted loci database","key":"HCvOxs7pWR"}],"key":"nueZhtKx0G"},{"type":"tableCell","children":[{"type":"text","value":"2015","key":"g466LuqfGn"}],"key":"C6fLzXxdqG"}],"key":"DPWBWoWpzX"},{"type":"tableRow","children":[{"type":"tableCell","rowspan":4,"children":[{"type":"text","value":"Format transition","key":"lZd6iWf7Mp"}],"key":"BS6useVtcg"},{"type":"tableCell","children":[{"type":"link","url":"https://bedtools.readthedocs.io/en/latest/","children":[{"type":"text","value":"BEDTools","key":"wUdw4jRbyL"}],"urlSource":"https://bedtools.readthedocs.io/en/latest/","key":"oXy3xft3mS"}],"key":"ThEYAx6ZpR"},{"type":"tableCell","children":[{"type":"text","value":"Software using bam or bed files. Raw count value can be obtained, but a candidate is required","key":"Pic4CCqUxA"}],"key":"aBqnMg4Q2f"},{"type":"tableCell","children":[{"type":"text","value":"2010","key":"btpFmSbxLf"}],"key":"DL2vsEw6ID"}],"key":"p24GWQSjvL"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://cole-trapnell-lab.github.io/cufflinks/manual/","children":[{"type":"text","value":"Cufflink","key":"WkU2C2gyEe"}],"urlSource":"https://cole-trapnell-lab.github.io/cufflinks/manual/","key":"Q1zP65iW83"}],"key":"QoGPgUcMcF"},{"type":"tableCell","children":[{"type":"text","value":"Tool that assembles RNA-seq mapping results (sam file) and calculates an unbalance haved FPKM value as result","key":"Guss7OZ56L"}],"key":"FtcPJpwz02"},{"type":"tableCell","children":[{"type":"text","value":"2010","key":"qVRdegz77k"}],"key":"pRM06jXeIW"}],"key":"pJ53yAubgE"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://htseq.readthedocs.io/en/release_0.11.1/index.html","children":[{"type":"text","value":"HTSeq","key":"NXmfSJOUSU"}],"urlSource":"https://htseq.readthedocs.io/en/release_0.11.1/index.html","key":"gIJjOWO7rw"}],"key":"EVzfBC3nLG"},{"type":"tableCell","children":[{"type":"text","value":"Python package, calculation of read mapping using gff information","key":"US4V1W1wPq"}],"key":"xc1OYIdNuy"},{"type":"tableCell","children":[{"type":"text","value":"2015","key":"nGU5aCVKFp"}],"key":"Dcj5YNDSJ7"}],"key":"qYWGQ5faO3"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"link","url":"https://www.htslib.org/","children":[{"type":"text","value":"SAMTools","key":"C2B3b2kLWz"}],"urlSource":"https://www.htslib.org/","key":"zOeCQi4UHk"}],"key":"yWfkdHiUua"},{"type":"tableCell","children":[{"type":"text","value":"Software that converts or sorts sam files","key":"NmaSnV1wYP"}],"key":"CAZCeWNE69"},{"type":"tableCell","children":[{"type":"text","value":"2009","key":"dMrMIBTbFT"}],"key":"dSfTbjouk9"}],"key":"NEWaPiorSB"}],"key":"s79WahLaVX"}],"key":"PGS79vi39d"}],"key":"yQl3Mk6wL9"},"references":{"cite":{"order":[],"data":{}}},"footer":{"navigation":{"prev":{"title":"Read-based profiling","url":"/read-based","group":"Metagenomics"},"next":{"title":"Sequencing","url":"/sequencing-1","group":"Metatranscriptomics"}}},"domain":"http://localhost:3000"},"project":{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}}},"actionData":null,"errors":null},"future":{"unstable_dev":false,"unstable_postcss":false,"unstable_tailwind":false,"v2_errorBoundary":true,"v2_headers":true,"v2_meta":true,"v2_normalizeFormMethod":true,"v2_routeConvention":true}};</script><script type="module" async="">import "/metainsight-book/build/manifest-3595279D.js";
import * as route0 from "/metainsight-book/build/root-SIO6LUTY.js";
import * as route1 from "/metainsight-book/build/routes/$-PRP77N34.js";
window.__remixRouteModules = {"root":route0,"routes/$":route1};

import("/metainsight-book/build/entry.client-PCJPW7TK.js");</script></body></html>