<!DOCTYPE html><html lang="en" class="" style="scroll-padding:60px"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis - META-Insight</title><meta property="og:title" content="Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis - META-Insight"/><meta name="generator" content="mystmd"/><meta name="description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta property="og:description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta name="keywords" content="metagenomics, metatranscriptomics, metaproteomics, microbiome, bioinformatics"/><meta name="image" content="/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png"/><meta property="og:image" content="/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png"/><link rel="stylesheet" href="/metainsight-book/build/_assets/app-MOQGDXHO.css"/><link rel="stylesheet" href="/metainsight-book/build/_assets/thebe-core-VKVHG5VY.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/jupyter-matplotlib@0.11.3/css/mpl_widget.css"/><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.css" integrity="sha384-MlJdn/WNKDGXveldHDdyRP1R4CTHr3FeuDNfhsLPYrq2t0UBkUdK2jyTnXPEK1NQ" crossorigin="anonymous"/><link rel="icon" href="/metainsight-book/favicon.ico"/><link rel="stylesheet" href="/metainsight-book/myst-theme.css"/><script>
  const savedTheme = localStorage.getItem("myst:theme");
  const theme = window.matchMedia("(prefers-color-scheme: light)").matches ? 'light' : 'dark';
  const classes = document.documentElement.classList;
  const hasAnyTheme = classes.contains('light') || classes.contains('dark');
  if (!hasAnyTheme) classes.add(savedTheme ?? theme);
</script></head><body class="m-0 transition-colors duration-500 bg-white dark:bg-stone-900"><div class="myst-skip-to-article fixed top-1 left-1 h-[0px] w-[0px] focus-within:z-40 focus-within:h-auto focus-within:w-auto bg-white overflow-hidden focus-within:p-2 focus-within:ring-1" aria-label="skip to content options"><a href="#skip-to-frontmatter" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article frontmatter</a><a href="#skip-to-article" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article content</a></div><dialog id="myst-no-css" style="position:fixed;left:0px;top:0px;width:100vw;height:100vh;font-size:4rem;padding:1rem;color:black;background:white"><strong>Site not loading correctly?</strong><p>This may be due to an incorrect <code>BASE_URL</code> configuration. See<!-- --> <a href="https://mystmd.org/guide/deployment#deploy-base-url">the MyST Documentation</a> <!-- -->for reference.</p><script>
    (() => {
            // Test for has-styling variable set by the MyST stylesheet
            const node = document.currentScript.parentNode;
            const hasCSS = window.getComputedStyle(node).getPropertyValue("--has-styling");
            if (hasCSS === ""){
                    node.showModal();
            }

    })()
</script></dialog><div class="myst-top-nav bg-white/80 backdrop-blur dark:bg-stone-900/80 shadow dark:shadow-stone-700 p-3 md:px-8 sticky w-screen top-0 z-30 h-[60px]"><nav class="myst-top-nav-bar flex items-center justify-between flex-nowrap max-w-[1440px] mx-auto"><div class="flex flex-row xl:min-w-[19.5rem] mr-2 sm:mr-7 justify-start items-center shrink-0"><div class="block xl:hidden"><button class="myst-top-nav-menu-button flex items-center justify-center border-stone-400 text-stone-800 hover:text-stone-900 dark:text-stone-200 hover:dark:text-stone-100 w-10 h-10"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem"><path fill-rule="evenodd" d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75ZM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12Zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75Z" clip-rule="evenodd"></path></svg><span class="sr-only">Open Menu</span></button></div><a class="myst-home-link flex items-center ml-3 dark:text-white w-fit md:ml-5 xl:ml-7" href="/metainsight-book/"><div class="myst-home-link-logo mr-3 flex items-center dark:bg-white dark:rounded px-1"><img src="/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg" class="h-9" height="2.25rem"/></div><span class="text-md sm:text-xl tracking-tight sm:mr-5 sr-only">Made with MyST</span></a></div><div class="flex items-center flex-grow w-auto"><div class="flex-grow hidden text-md lg:block"></div><div class="flex-grow block"></div><button type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R75cp:" data-state="closed" class="myst-search-bar flex items-center h-10 aspect-square sm:w-64 text-left text-gray-600 dark:text-gray-300 border border-gray-300 dark:border-gray-600 rounded-lg bg-gray-50 dark:bg-gray-700 myst-search-bar-disabled hover:ring-blue-500 dark:hover:ring-blue-500 hover:border-blue-500 dark:hover:border-blue-500"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="p-2.5 h-10 w-10 aspect-square"><path fill-rule="evenodd" d="M10.5 3.75a6.75 6.75 0 1 0 0 13.5 6.75 6.75 0 0 0 0-13.5ZM2.25 10.5a8.25 8.25 0 1 1 14.59 5.28l4.69 4.69a.75.75 0 1 1-1.06 1.06l-4.69-4.69A8.25 8.25 0 0 1 2.25 10.5Z" clip-rule="evenodd"></path></svg><span class="myst-search-text-placeholder hidden sm:block grow">Search</span><div aria-hidden="true" class="myst-search-shortcut items-center hidden mx-1 font-mono text-sm text-gray-600 dark:text-gray-300 sm:flex gap-x-1"><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none hide-mac">CTRL</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none show-mac">⌘</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none ">K</kbd><script>
;(() => {
const script = document.currentScript;
const root = script.parentElement;

const isMac = /mac/i.test(
      window.navigator.userAgentData?.platform ?? window.navigator.userAgent,
    );
root.querySelectorAll(".hide-mac").forEach(node => {node.classList.add(isMac ? "hidden" : "block")});
root.querySelectorAll(".show-mac").forEach(node => {node.classList.add(!isMac ? "hidden" : "block")});
})()</script></div></button><button class="myst-theme-button theme rounded-full aspect-square border border-stone-700 dark:border-white hover:bg-neutral-100 border-solid overflow-hidden text-stone-700 dark:text-white hover:text-stone-500 dark:hover:text-neutral-800 w-10 h-10 mx-3" title="Toggle theme between light and dark mode" aria-label="Toggle theme between light and dark mode"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-moon-icon h-full w-full p-0.5 hidden dark:block"><path fill-rule="evenodd" d="M9.528 1.718a.75.75 0 0 1 .162.819A8.97 8.97 0 0 0 9 6a9 9 0 0 0 9 9 8.97 8.97 0 0 0 3.463-.69.75.75 0 0 1 .981.98 10.503 10.503 0 0 1-9.694 6.46c-5.799 0-10.5-4.7-10.5-10.5 0-4.368 2.667-8.112 6.46-9.694a.75.75 0 0 1 .818.162Z" clip-rule="evenodd"></path></svg><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-sun-icon h-full w-full p-0.5 dark:hidden"><path stroke-linecap="round" stroke-linejoin="round" d="M12 3v2.25m6.364.386-1.591 1.591M21 12h-2.25m-.386 6.364-1.591-1.591M12 18.75V21m-4.773-4.227-1.591 1.591M5.25 12H3m4.227-4.773L5.636 5.636M15.75 12a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0Z"></path></svg></button><div class="block sm:hidden"></div><div class="hidden sm:block"></div></div></nav></div><div class="myst-primary-sidebar fixed xl:article-grid grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px] hidden z-10" style="top:60px"><div class="myst-primary-sidebar-pointer pointer-events-auto xl:col-margin-left flex-col overflow-hidden hidden xl:flex"><div class="myst-primary-sidebar-nav flex-grow py-6 overflow-y-auto primary-scrollbar"><nav aria-label="Navigation" class="myst-primary-sidebar-topnav overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px] lg:hidden"><div class="w-full px-1 dark:text-white font-medium"></div></nav><div class="my-3 border-b-2 lg:hidden"></div><nav aria-label="Table of Contents" class="myst-primary-sidebar-toc flex-grow overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px]"><div class="myst-toc w-full px-1 dark:text-white"><a title="META-Insight" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30 font-bold" href="/metainsight-book/">META-Insight</a><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metagenomics" class="block break-words rounded py-2 grow cursor-pointer">Metagenomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rmpsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rmpsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metatranscriptomics" class="block break-words rounded py-2 grow cursor-pointer">Metatranscriptomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rupsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rupsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metaproteomics" class="block break-words rounded py-2 grow cursor-pointer">Metaproteomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:R16psp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:R16psp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div></div></nav></div><div class="myst-primary-sidebar-footer flex-none py-6 transition-all duration-700 translate-y-6 opacity-0"><a class="myst-made-with-myst flex mx-auto text-gray-700 w-fit hover:text-blue-700 dark:text-gray-200 dark:hover:text-blue-400" href="https://mystmd.org/made-with-myst" target="_blank" rel="noreferrer"><svg style="width:24px;height:24px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" stroke="none"><g id="icon"><path fill="currentColor" d="M23.8,54.8v-3.6l4.7-0.8V17.5l-4.7-0.8V13H36l13.4,31.7h0.2l13-31.7h12.6v3.6l-4.7,0.8v32.9l4.7,0.8v3.6h-15
          v-3.6l4.9-0.8V20.8H65L51.4,53.3h-3.8l-14-32.5h-0.1l0.2,17.4v12.1l5,0.8v3.6H23.8z"></path><path fill="#F37726" d="M47,86.9c0-5.9-3.4-8.8-10.1-8.8h-8.4c-5.2,0-9.4-1.3-12.5-3.8c-3.1-2.5-5.4-6.2-6.8-11l4.8-1.6
          c1.8,5.6,6.4,8.6,13.8,8.8h9.2c6.4,0,10.8,2.5,13.1,7.5c2.3-5,6.7-7.5,13.1-7.5h8.4c7.8,0,12.7-2.9,14.6-8.7l4.8,1.6
          c-1.4,4.9-3.6,8.6-6.8,11.1c-3.1,2.5-7.3,3.7-12.4,3.8H63c-6.7,0-10,2.9-10,8.8"></path></g></svg><span class="self-center ml-2 text-sm">Made with MyST</span></a></div></div></div><main class="article-grid grid-gap"><article class="article-grid subgrid-gap col-screen article content"><div class="hidden"></div><div id="skip-to-frontmatter" aria-label="article frontmatter" class="myst-fm-block mb-8 pt-9"><div class="myst-fm-block-header flex items-center mb-5 h-6 text-sm font-light"><div class="flex-grow"></div><div class="myst-fm-block-badges"><a href="https://github.com/aababc1/metainsight-book" title="GitHub Repository: aababc1/metainsight-book" target="_blank" rel="noopener noreferrer" class="myst-fm-github-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" width="1.25rem" height="1.25rem" class="myst-fm-github-icon inline-block mr-1 opacity-60 hover:opacity-100"><path d="M12 2.5c-5.4 0-9.8 4.4-9.8 9.7 0 4.3 2.8 8 6.7 9.2.5.1.7-.2.7-.5v-1.8c-2.4.5-3.1-.6-3.3-1.1-.1-.3-.6-1.1-1-1.4-.3-.2-.8-.6 0-.6s1.3.7 1.5 1c.9 1.5 2.3 1.1 2.8.8.1-.6.3-1.1.6-1.3-2.2-.2-4.4-1.1-4.4-4.8 0-1.1.4-1.9 1-2.6-.1-.2-.4-1.2.1-2.6 0 0 .8-.3 2.7 1 .8-.2 1.6-.3 2.4-.3.8 0 1.7.1 2.4.3 1.9-1.3 2.7-1 2.7-1 .5 1.3.2 2.3.1 2.6.6.7 1 1.5 1 2.6 0 3.7-2.3 4.6-4.4 4.8.4.3.7.9.7 1.8V21c0 .3.2.6.7.5 3.9-1.3 6.6-4.9 6.6-9.2 0-5.4-4.4-9.8-9.8-9.8z"></path></svg></a></div><a href="https://github.com/aababc1/metainsight-book/edit/main/metagenomics/Introduction.md" title="Edit This Page" target="_blank" rel="noopener noreferrer" class="myst-fm-edit-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-edit-icon inline-block mr-1 opacity-60 hover:opacity-100"><path stroke-linecap="round" stroke-linejoin="round" d="m16.862 4.487 1.687-1.688a1.875 1.875 0 1 1 2.652 2.652L10.582 16.07a4.5 4.5 0 0 1-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 0 1 1.13-1.897l8.932-8.931Zm0 0L19.5 7.125M18 14v4.75A2.25 2.25 0 0 1 15.75 21H5.25A2.25 2.25 0 0 1 3 18.75V8.25A2.25 2.25 0 0 1 5.25 6H10"></path></svg></a><div class="myst-fm-downloads-dropdown relative flex inline-block mx-1 grow-0" data-headlessui-state=""><button class="myst-fm-downloads-button relative ml-2 -mr-1" id="headlessui-menu-button-:Rs8ucp:" type="button" aria-haspopup="menu" aria-expanded="false" data-headlessui-state=""><span class="sr-only">Downloads</span><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-downloads-icon"><title>Download</title><path stroke-linecap="round" stroke-linejoin="round" d="M3 16.5v2.25A2.25 2.25 0 0 0 5.25 21h13.5A2.25 2.25 0 0 0 21 18.75V16.5M16.5 12 12 16.5m0 0L7.5 12m4.5 4.5V3"></path></svg></button></div></div><h1 class="myst-fm-block-title mb-0">Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis</h1><header class="myst-fm-authors-affiliations mt-4 not-prose"><div class="myst-fm-authors-list"><span class="myst-fm-author font-semibold text-sm myst-fm-author-item inline-block"><button class="myst-fm-author-popover focus:shadow-[0_0_0_2px] focus:shadow-black outline-none hover:underline" aria-label="Author Details" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R78ucp:" data-state="closed"><span class="myst-fm-author-name">META-Insight Team</span></button></span></div></header></div><div class="block my-10 lg:sticky lg:z-10 lg:h-0 lg:pt-0 lg:my-0 lg:ml-10 lg:col-margin-right" style="top:60px"><nav></nav></div><div id="skip-to-article"></div><h2 id="introduction-to-metagenome" class="relative group"><span class="heading-text">Introduction to metagenome</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#introduction-to-metagenome" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><figure id="MvpzK9wbt7" class=""><img id="QmoEdlXShU" style="margin-left:auto;margin-right:auto" src="/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png" alt="Metagenomics overview" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_0_1.png?raw=true" class=""/><figcaption class="group"><p><strong>Metagenomics overview</strong></p></figcaption></figure><p>A metagenome is the pool of genetic material present in various environmental samples (animal, plant, marine, etc.).
Metagenomic analysis extracts and analyzes the genetic information of all organisms present in a given environmental ecosystem.
Before the rise of metagenomics, microbial genomes could only be analyzed from culturable microorganisms. However, since only a small fraction of the microbes in a given environment are culturable, this approach results in biased information centered on culturable microbes.
Metagenomic analyses, on the other hand, analyze the entire genome of an environment, providing a more accurate picture of a given ecosystem.</p><p>Advancements in sequencing technology have made it easier for researchers to access whole metagenome shotgun sequencing, which sequences billions of nucleic acid fragments at once, beyond amplicon sequencing, which involves targeted amplification and sequencing of specific parts of the genome such as 16S ribosomal DNA, and the demand for research has increased.
For amplicon sequencing data, there are standard analysis protocols and platforms such as QIIME2, but for WMS analysis, there is no standard gold-standard analysis SOP.
In this SOP, we will describe the methods and flow for analyzing whole genome sequencing (WGS) data, focusing on the most commonly used tools among various analysis programs.</p><div class="" style="color:#1565C0"><h2 id="sequencing-technology-selection" class="relative group"><span class="heading-text">Sequencing Technology Selection</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#sequencing-technology-selection" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><p>Choosing the appropriate sequencing platform is one of the most critical decisions in metagenomic study design, as it directly determines the achievable resolution of taxonomic classification, assembly completeness, and overall analytical scope. Currently, three major platforms dominate the field: Illumina (short-read), Pacific Biosciences (PacBio, long-read), and Oxford Nanopore Technologies (ONT, long-read). Each platform has distinct strengths and trade-offs that must be carefully considered based on the research objectives, available budget, and sample characteristics.</p><h3 id="platform-comparison" class="relative group"><span class="heading-text">Platform Comparison</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#platform-comparison" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><table class=""><tbody><tr class=""><th class="">Feature</th><th class="">Illumina NovaSeq 6000</th><th class="">PacBio Revio (HiFi)</th><th class="">ONT PromethION</th></tr><tr class=""><td class=""><strong>Read length</strong></td><td class="">Up to 2 × 250 bp</td><td class="">15–18 kb</td><td class="">up to 4 Mb (avg ~20 kb)</td></tr><tr class=""><td class=""><strong>Read accuracy</strong></td><td class="">~99.75% (Q30+)</td><td class="">&gt;99.5% (Q30+)</td><td class="">97–99% (Q20+)</td></tr><tr class=""><td class=""><strong>Yield per run</strong></td><td class="">~350 Gb</td><td class="">~90 Gb per SMRT Cell (×4)</td><td class="">~120 Gb</td></tr><tr class=""><td class=""><strong>Cost per Gb</strong></td><td class="">~US$4</td><td class="">~US$8–11</td><td class="">~US$6–12</td></tr><tr class=""><td class=""><strong>DNA input</strong></td><td class="">1–500 ng</td><td class="">150 ng – 1 µg</td><td class="">150 ng – 1 µg</td></tr><tr class=""><td class=""><strong>Runtime</strong></td><td class="">13–44 h</td><td class="">~24 h</td><td class="">~72 h</td></tr><tr class=""><td class=""><strong>Direct RNA-seq</strong></td><td class="">No</td><td class="">No</td><td class="">Yes</td></tr><tr class=""><td class=""><strong>Portability</strong></td><td class="">Low</td><td class="">Low</td><td class="">High (MinION)</td></tr></tbody></table><p><em>Note: Sequencing costs are estimates and subject to rapid changes</em></p><div class=""><div class=""></div></div><em>.</em><h3 id="when-to-choose-short-reads-illumina" class="relative group"><span class="heading-text">When to Choose Short Reads (Illumina)</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#when-to-choose-short-reads-illumina" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><p>Short-read sequencing remains the most widely adopted approach in metagenomics due to its low cost per gigabase, high base-level accuracy, and lower DNA input requirements. It is particularly well-suited for:</p><ul><li><p><strong>Community profiling and abundance estimation</strong> — when the primary goal is to determine which organisms are present and at what relative abundance, short reads provide cost-effective, high-throughput results.</p></li><li><p><strong>Functional gene catalog construction</strong> — read-based profiling tools such as HUMAnN and MetaPhlAn are optimized for short-read data.</p></li><li><p><strong>Differential abundance studies</strong> — when comparing microbial communities across many samples (e.g., case-control studies), the lower per-sample cost of short reads enables larger sample sizes.</p></li><li><p><strong>Low-biomass samples</strong> — Illumina library preparation with PCR amplification can work with as little as 1 ng of input DNA, an important advantage when sample material is limited.</p></li></ul><p>However, short reads have well-documented limitations. Repetitive genomic regions longer than the read length (~250 bp) cannot be resolved, resulting in fragmented assemblies. This makes it difficult to recover complete genomes (MAGs), characterize mobile genetic elements (plasmids, transposons), and distinguish closely related strains that differ in only a few genomic loci.</p><h3 id="when-to-choose-long-reads-pacbio-hifi-or-ont" class="relative group"><span class="heading-text">When to Choose Long Reads (PacBio HiFi or ONT)</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#when-to-choose-long-reads-pacbio-hifi-or-ont" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><p>Long-read sequencing has matured considerably in recent years, with significant improvements in accuracy, throughput, and cost. Long reads are recommended when the study requires:</p><ul><li><p><strong>Complete or near-complete MAGs</strong> — long reads can span repetitive regions and resolve strain-level haplotypes, enabling the assembly of circularized genomes directly from metagenomic samples. For example, PacBio HiFi reads with hifiasm-meta have been shown to circularize 56 genomes from human gut microbiome samples.</p></li><li><p><strong>Strain-level resolution</strong> — intergenomic repeats (sequences shared between related organisms) are difficult to resolve with short reads alone. Long reads increase the likelihood that a read includes an organism-specific region, improving taxonomic classification at species and sub-species levels.</p></li><li><p><strong>Structural variant detection</strong> — insertions, deletions, inversions, and other large-scale genomic rearrangements (&gt;50 bp) are far more reliably detected with long reads.</p></li><li><p><strong>Methylation analysis</strong> — both PacBio and ONT can detect DNA methylation patterns (5mC, 6mA, 4mC) directly during sequencing without bisulfite conversion, enabling epigenomic characterization of microbial communities.</p></li><li><p><strong>Plasmid and mobile element recovery</strong> — short-read assemblies disproportionately fail for multi-copy DNA elements such as 16S genes, transposons, and plasmids. Long reads overcome this by spanning entire elements in single reads.</p></li></ul><h4 id="pacbio-hifi-vs-ont" class="relative group"><span class="heading-text">PacBio HiFi vs. ONT</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#pacbio-hifi-vs-ont" title="Link to this Section" aria-label="Link to this Section">¶</a></h4><table class=""><tbody><tr class=""><th class="">Consideration</th><th class="">PacBio HiFi</th><th class="">ONT</th></tr><tr class=""><td class=""><strong>Best for</strong></td><td class="">High-accuracy MAG assembly, variant calling</td><td class="">Rapid/field sequencing, ultra-long reads, direct RNA-seq</td></tr><tr class=""><td class=""><strong>Accuracy</strong></td><td class="">&gt;99.5% (comparable to Illumina)</td><td class="">97–99% (improving with R10 chemistry)</td></tr><tr class=""><td class=""><strong>Read length</strong></td><td class="">15–18 kb (consistent)</td><td class="">Up to 4 Mb (variable, avg ~20 kb)</td></tr><tr class=""><td class=""><strong>Methylation</strong></td><td class="">5mC, 6mA</td><td class="">5mC, 6mA, 4mC + RNA modifications</td></tr><tr class=""><td class=""><strong>Portability</strong></td><td class="">Lab-based only</td><td class="">MinION/Flongle can be used in field settings</td></tr><tr class=""><td class=""><strong>Error correction needed</strong></td><td class="">Rarely (HiFi reads are self-corrected)</td><td class="">Often recommended before assembly</td></tr></tbody></table><h3 id="hybrid-approach-short-long-reads" class="relative group"><span class="heading-text">Hybrid Approach (Short + Long Reads)</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#hybrid-approach-short-long-reads" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><p>A hybrid sequencing strategy combines the strengths of both technologies: the high base-level accuracy of short reads with the long-range contiguity information from long reads. Hybrid assembly tools such as <strong>OPERA-MS</strong>, <strong>Unicycler</strong>, and <strong>DBG2OLC</strong> can leverage both data types to produce assemblies with improved contiguity, gene completeness, and binning accuracy compared to short-read-only approaches.</p><p>However, hybrid approaches involve higher costs (two library preparations and sequencing runs) and can introduce biases from combining different library preparation chemistries. With the increasing accuracy of PacBio HiFi reads (&gt;Q30), long-read-only assemblies are now achieving comparable or superior quality to hybrid assemblies for many applications, making the hybrid approach less necessary than in previous years.</p><h3 id="decision-framework" class="relative group"><span class="heading-text">Decision Framework</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#decision-framework" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><p>The choice of sequencing strategy depends on the study’s objectives:</p><table class=""><tbody><tr class=""><th class="">Research Goal</th><th class="">Recommended Strategy</th></tr><tr class=""><td class="">Large-scale community profiling (many samples)</td><td class=""><strong>Illumina short reads</strong></td></tr><tr class=""><td class="">Complete genome recovery (MAGs)</td><td class=""><strong>PacBio HiFi</strong> or <strong>ONT + polishing</strong></td></tr><tr class=""><td class="">Strain-level resolution</td><td class=""><strong>Long reads</strong> (PacBio or ONT)</td></tr><tr class=""><td class="">Plasmid / mobile element tracking</td><td class=""><strong>Long reads</strong></td></tr><tr class=""><td class="">Antibiotic resistance gene surveillance</td><td class=""><strong>ONT</strong> (rapid, portable)</td></tr><tr class=""><td class="">Low-biomass / clinical samples</td><td class=""><strong>Illumina</strong> (low DNA input)</td></tr><tr class=""><td class="">Multi-omics integration (DNA + RNA)</td><td class=""><strong>ONT</strong> (direct RNA-seq capability)</td></tr><tr class=""><td class="">Budget-constrained study</td><td class=""><strong>Illumina</strong> (lowest cost/Gb)</td></tr></tbody></table><p>In this SOP, we provide analysis pipelines for both <strong>Illumina short-read</strong> and <strong>PacBio HiFi long-read</strong> metagenomics, covering sample preparation through computational analysis for each approach.</p></div><div class="myst-backmatter-parts"></div><div class="myst-footer-links flex pt-10 mb-10 space-x-4"><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-prev" href="/metainsight-book/"><div class="flex h-full align-middle"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:-translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5 3 12m0 0 7.5-7.5M3 12h18"></path></svg><div class="flex-grow text-right"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">META-Insight</div>META-Insight</div></div></a><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-next" href="/metainsight-book/sequencing"><div class="flex h-full align-middle"><div class="flex-grow"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metagenomics</div>Preparation for sequencing</div><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 4.5 21 12m0 0-7.5 7.5M21 12H3"></path></svg></div></a></div></article></main><script>((a,l)=>{if(!window.history.state||!window.history.state.key){let u=Math.random().toString(32).slice(2);window.history.replaceState({key:u},"")}try{let d=JSON.parse(sessionStorage.getItem(a)||"{}")[l||window.history.state.key];typeof d=="number"&&window.scrollTo(0,d)}catch(u){console.error(u),sessionStorage.removeItem(a)}})("positions", null)</script><link rel="modulepreload" href="/metainsight-book/build/entry.client-PCJPW7TK.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-AQ2CODAG.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-JJXTQVMA.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-OZE3FFNP.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-CH4FVTDV.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-C4DFGG5C.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-J7TUH54J.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-FZ2S7OYD.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-JEM6JXYA.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-34XIY2DH.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-KQM5FBHR.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-OCWQY3HK.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-7HNKBP4B.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-CUKUDK3R.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-3EBOCCHJ.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-O4VQNZ62.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-4OEDG4JQ.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-GUCIBHGO.js"/><link rel="modulepreload" href="/metainsight-book/build/root-SIO6LUTY.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-FAHZZXAC.js"/><link rel="modulepreload" href="/metainsight-book/build/routes/$-PRP77N34.js"/><script>window.__remixContext = {"url":"/introduction","state":{"loaderData":{"root":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg","logo":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"CONTENT_CDN_PORT":"3100","MODE":"static","BASE_URL":"/metainsight-book"},"routes/$":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg","logo":"/metainsight-book/build/META-Insight_logo-b2aece8d8ce45f6cfda96b8bc0d86cfd.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"page":{"version":3,"kind":"Article","sha256":"3de00bc5c6da8cfaf258a666caff6841007b51a20f909384272298600bd65e94","slug":"introduction","location":"/metagenomics/Introduction.md","dependencies":[],"frontmatter":{"title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","content_includes_title":false,"authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"numbering":{"title":{"offset":1}},"source_url":"https://github.com/aababc1/metainsight-book/blob/main/metagenomics/Introduction.md","edit_url":"https://github.com/aababc1/metainsight-book/edit/main/metagenomics/Introduction.md","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","exports":[{"format":"md","filename":"Introduction.md","url":"/metainsight-book/build/Introduction-9a983c0e38d9c042cf754c2e60a809b2.md"}]},"mdast":{"type":"root","children":[{"type":"block","children":[{"type":"heading","depth":2,"position":{"start":{"line":3,"column":1},"end":{"line":3,"column":1}},"children":[{"type":"text","value":"Introduction to metagenome","position":{"start":{"line":3,"column":1},"end":{"line":3,"column":1}},"key":"wiI6d2iiro"}],"identifier":"introduction-to-metagenome","label":"Introduction to metagenome","html_id":"introduction-to-metagenome","implicit":true,"key":"CYJ927lde9"},{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","alt":"Metagenomics overview","data":{"altTextIsAutoGenerated":true},"key":"QmoEdlXShU","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_0_1.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Metagenomics overview","key":"cjgcLgdJ55"}],"key":"ymRZmfUFOd"}],"key":"AdBROSXtJ4"}],"key":"vG7bKkrSTA"}],"enumerator":"1","key":"MvpzK9wbt7"},{"type":"paragraph","position":{"start":{"line":10,"column":1},"end":{"line":13,"column":1}},"children":[{"type":"text","value":"A metagenome is the pool of genetic material present in various environmental samples (animal, plant, marine, etc.).\nMetagenomic analysis extracts and analyzes the genetic information of all organisms present in a given environmental ecosystem.\nBefore the rise of metagenomics, microbial genomes could only be analyzed from culturable microorganisms. However, since only a small fraction of the microbes in a given environment are culturable, this approach results in biased information centered on culturable microbes.\nMetagenomic analyses, on the other hand, analyze the entire genome of an environment, providing a more accurate picture of a given ecosystem.","position":{"start":{"line":10,"column":1},"end":{"line":10,"column":1}},"key":"kMfe072UQw"}],"key":"vJpRLrmSNa"},{"type":"paragraph","position":{"start":{"line":15,"column":1},"end":{"line":17,"column":1}},"children":[{"type":"text","value":"Advancements in sequencing technology have made it easier for researchers to access whole metagenome shotgun sequencing, which sequences billions of nucleic acid fragments at once, beyond amplicon sequencing, which involves targeted amplification and sequencing of specific parts of the genome such as 16S ribosomal DNA, and the demand for research has increased.\nFor amplicon sequencing data, there are standard analysis protocols and platforms such as QIIME2, but for WMS analysis, there is no standard gold-standard analysis SOP.\nIn this SOP, we will describe the methods and flow for analyzing whole genome sequencing (WGS) data, focusing on the most commonly used tools among various analysis programs.","position":{"start":{"line":15,"column":1},"end":{"line":15,"column":1}},"key":"c71m3Yao7x"}],"key":"Q0GkF6Vrgk"},{"type":"div","style":{"color":"#1565C0"},"children":[{"type":"heading","depth":2,"children":[{"type":"text","value":"Sequencing Technology Selection","key":"B4CNWbTPrT"}],"identifier":"sequencing-technology-selection","label":"Sequencing Technology Selection","html_id":"sequencing-technology-selection","implicit":true,"key":"jdYmqVoYuA"},{"type":"paragraph","children":[{"type":"text","value":"Choosing the appropriate sequencing platform is one of the most critical decisions in metagenomic study design, as it directly determines the achievable resolution of taxonomic classification, assembly completeness, and overall analytical scope. Currently, three major platforms dominate the field: Illumina (short-read), Pacific Biosciences (PacBio, long-read), and Oxford Nanopore Technologies (ONT, long-read). Each platform has distinct strengths and trade-offs that must be carefully considered based on the research objectives, available budget, and sample characteristics.","key":"xjfNUHlNuF"}],"key":"CvZJYvPIaN"},{"type":"heading","depth":3,"children":[{"type":"text","value":"Platform Comparison","key":"E43GPxZCnk"}],"identifier":"platform-comparison","label":"Platform Comparison","html_id":"platform-comparison","implicit":true,"key":"ZUExKP1TMb"},{"type":"table","children":[{"type":"tableRow","children":[{"type":"tableCell","header":true,"children":[{"type":"text","value":"Feature","key":"UKTiVXKj8j"}],"key":"kUoMk1NfTQ"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"Illumina NovaSeq 6000","key":"dfQKKicKOz"}],"key":"jeoIiNo1oR"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"PacBio Revio (HiFi)","key":"BEXaoTDicj"}],"key":"ivz3AoAb8x"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"ONT PromethION","key":"heI8BWKx3L"}],"key":"hgDdb9xTEV"}],"key":"WKio0CZlA2"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Read length","key":"eZP9BeqEwe"}],"key":"i89ZKu2MXx"}],"key":"ypNNeevGNQ"},{"type":"tableCell","children":[{"type":"text","value":"Up to 2 × 250 bp","key":"STStCv7pWb"}],"key":"gAXl4ej2Iu"},{"type":"tableCell","children":[{"type":"text","value":"15–18 kb","key":"isuEqDFDtb"}],"key":"Su3aXauSm7"},{"type":"tableCell","children":[{"type":"text","value":"up to 4 Mb (avg ~20 kb)","key":"hRlETA8p7s"}],"key":"EJfNR2QFiX"}],"key":"sfWiRJI5ai"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Read accuracy","key":"QriuEfuJlC"}],"key":"SpzH1uY8KT"}],"key":"GcExtyKa35"},{"type":"tableCell","children":[{"type":"text","value":"~99.75% (Q30+)","key":"NXO5ssnrPT"}],"key":"arOP2coc5h"},{"type":"tableCell","children":[{"type":"text","value":"\u003e99.5% (Q30+)","key":"qXalQkexoP"}],"key":"Gg5YTd5BxQ"},{"type":"tableCell","children":[{"type":"text","value":"97–99% (Q20+)","key":"Lwjf8LEznK"}],"key":"edxBlzzvLk"}],"key":"Ps0LM0l6FZ"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Yield per run","key":"pyTViTKmND"}],"key":"qDfBBCoj6q"}],"key":"pxQ8EWaWya"},{"type":"tableCell","children":[{"type":"text","value":"~350 Gb","key":"xM6Fy8WqtA"}],"key":"FId97mISzu"},{"type":"tableCell","children":[{"type":"text","value":"~90 Gb per SMRT Cell (×4)","key":"G3u6jPDEgC"}],"key":"HCzINKzR7b"},{"type":"tableCell","children":[{"type":"text","value":"~120 Gb","key":"asuIhEdr3q"}],"key":"hvnTvtUyJt"}],"key":"lYfa5sjYQb"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Cost per Gb","key":"lhZTmzkKBQ"}],"key":"rCCeCWfr0L"}],"key":"ToQ4pOGSmZ"},{"type":"tableCell","children":[{"type":"text","value":"~US$4","key":"qPEWNHfyc4"}],"key":"AaZDAMZBAa"},{"type":"tableCell","children":[{"type":"text","value":"~US$8–11","key":"AGcTv4yI1m"}],"key":"cfZlYpKF8q"},{"type":"tableCell","children":[{"type":"text","value":"~US$6–12","key":"OkhCFR2aAI"}],"key":"iGqFMk32Xe"}],"key":"aX0Xru1Qkd"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"DNA input","key":"heKDdbayV5"}],"key":"RtKkkvuw7A"}],"key":"TMh1jz085t"},{"type":"tableCell","children":[{"type":"text","value":"1–500 ng","key":"SVExYjoEEO"}],"key":"I8l66T0MvX"},{"type":"tableCell","children":[{"type":"text","value":"150 ng – 1 µg","key":"pYAB6GlmVT"}],"key":"NFvInyJCx9"},{"type":"tableCell","children":[{"type":"text","value":"150 ng – 1 µg","key":"EmGzlcudri"}],"key":"ga2nYsosep"}],"key":"yjdwwRzbYY"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Runtime","key":"mD0DvnwW0Z"}],"key":"qzbt7JGob0"}],"key":"MpVb5NDITZ"},{"type":"tableCell","children":[{"type":"text","value":"13–44 h","key":"KNK8SjKgyJ"}],"key":"dad79RZIE5"},{"type":"tableCell","children":[{"type":"text","value":"~24 h","key":"qSYxxtQM6a"}],"key":"UTfvafapLl"},{"type":"tableCell","children":[{"type":"text","value":"~72 h","key":"fXRxYLrBap"}],"key":"PxnF27oAA0"}],"key":"Ce3Z1cujdq"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Direct RNA-seq","key":"pudlhkHrA8"}],"key":"Q4hnDPxDpJ"}],"key":"wNaslEhvPf"},{"type":"tableCell","children":[{"type":"text","value":"No","key":"qbq0zYvEiD"}],"key":"ToJsvKAh1f"},{"type":"tableCell","children":[{"type":"text","value":"No","key":"aHFhU2pUZb"}],"key":"DWL1zvlhMV"},{"type":"tableCell","children":[{"type":"text","value":"Yes","key":"dXVKGwYril"}],"key":"WiTx0bSQ47"}],"key":"PpQoeBTsCp"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Portability","key":"Z7duZd6GJC"}],"key":"RCcsgOWeY0"}],"key":"O9MVFimn5S"},{"type":"tableCell","children":[{"type":"text","value":"Low","key":"PpXvJyDnSa"}],"key":"sP23pZcqyI"},{"type":"tableCell","children":[{"type":"text","value":"Low","key":"sMshlqjbM0"}],"key":"KyDrBl1X0b"},{"type":"tableCell","children":[{"type":"text","value":"High (MinION)","key":"g7ATWxvjto"}],"key":"VCsyoMcC6Q"}],"key":"YkYk3JDtgs"}],"key":"MG8eg0zUyj"},{"type":"paragraph","children":[{"type":"emphasis","children":[{"type":"text","value":"Note: Sequencing costs are estimates and subject to rapid changes","key":"FZH36WwGMd"}],"key":"qbcSyEqopp"}],"key":"DlEwSA3GTF"},{"type":"div","children":[{"type":"div","children":[],"key":"TwVTbeWaPN"}],"key":"CKNnkY8rOg"},{"type":"emphasis","children":[{"type":"text","value":".","key":"ZKOcY2PqFa"}],"key":"wWterx9IaB"},{"type":"heading","depth":3,"children":[{"type":"text","value":"When to Choose Short Reads (Illumina)","key":"qQlm49xWUQ"}],"identifier":"when-to-choose-short-reads-illumina","label":"When to Choose Short Reads (Illumina)","html_id":"when-to-choose-short-reads-illumina","implicit":true,"key":"OwXrB7t4na"},{"type":"paragraph","children":[{"type":"text","value":"Short-read sequencing remains the most widely adopted approach in metagenomics due to its low cost per gigabase, high base-level accuracy, and lower DNA input requirements. It is particularly well-suited for:","key":"IFM2NIHNqM"}],"key":"QqbY5goral"},{"type":"list","ordered":false,"start":null,"spread":false,"children":[{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Community profiling and abundance estimation","key":"eHF9uBF8Oe"}],"key":"Mus31zqVrX"},{"type":"text","value":" — when the primary goal is to determine which organisms are present and at what relative abundance, short reads provide cost-effective, high-throughput results.","key":"lMkI0wTxcA"}],"key":"vPL4C8WlmQ"}],"key":"n1u89mSAsU"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Functional gene catalog construction","key":"qb01SPTwhZ"}],"key":"LSXZBt5ki8"},{"type":"text","value":" — read-based profiling tools such as HUMAnN and MetaPhlAn are optimized for short-read data.","key":"XEMwV9ZaDm"}],"key":"ftpZMEsCxw"}],"key":"yQSP0xQ5Aj"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Differential abundance studies","key":"vUfRVtTNMq"}],"key":"xLmYcNyuSg"},{"type":"text","value":" — when comparing microbial communities across many samples (e.g., case-control studies), the lower per-sample cost of short reads enables larger sample sizes.","key":"Q0ecxASpwK"}],"key":"VEemPtQcvJ"}],"key":"pcoHwl02JT"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Low-biomass samples","key":"SLp700L1mO"}],"key":"vsPBL8QOpj"},{"type":"text","value":" — Illumina library preparation with PCR amplification can work with as little as 1 ng of input DNA, an important advantage when sample material is limited.","key":"JUIxDD2qE4"}],"key":"ZS1aVyReQm"}],"key":"A2MsWbfBPV"}],"key":"beCFwGrc4w"},{"type":"paragraph","children":[{"type":"text","value":"However, short reads have well-documented limitations. Repetitive genomic regions longer than the read length (~250 bp) cannot be resolved, resulting in fragmented assemblies. This makes it difficult to recover complete genomes (MAGs), characterize mobile genetic elements (plasmids, transposons), and distinguish closely related strains that differ in only a few genomic loci.","key":"tJFHbykGwi"}],"key":"ncx8X8U2Nt"},{"type":"heading","depth":3,"children":[{"type":"text","value":"When to Choose Long Reads (PacBio HiFi or ONT)","key":"iV5BJBvxsZ"}],"identifier":"when-to-choose-long-reads-pacbio-hifi-or-ont","label":"When to Choose Long Reads (PacBio HiFi or ONT)","html_id":"when-to-choose-long-reads-pacbio-hifi-or-ont","implicit":true,"key":"jimCPyml1w"},{"type":"paragraph","children":[{"type":"text","value":"Long-read sequencing has matured considerably in recent years, with significant improvements in accuracy, throughput, and cost. Long reads are recommended when the study requires:","key":"atRbEDvwkV"}],"key":"Na9IPQbTzK"},{"type":"list","ordered":false,"start":null,"spread":false,"children":[{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Complete or near-complete MAGs","key":"L89d4P5Djb"}],"key":"RFlVxkjKi7"},{"type":"text","value":" — long reads can span repetitive regions and resolve strain-level haplotypes, enabling the assembly of circularized genomes directly from metagenomic samples. For example, PacBio HiFi reads with hifiasm-meta have been shown to circularize 56 genomes from human gut microbiome samples.","key":"hR95aLShH6"}],"key":"ifAj4yTEzx"}],"key":"iMQml6mhI8"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Strain-level resolution","key":"DN4Zy4YswS"}],"key":"l7WAq73Nhp"},{"type":"text","value":" — intergenomic repeats (sequences shared between related organisms) are difficult to resolve with short reads alone. Long reads increase the likelihood that a read includes an organism-specific region, improving taxonomic classification at species and sub-species levels.","key":"BrdGxvDipv"}],"key":"COl3LZMbmQ"}],"key":"zT1ZUWgfip"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Structural variant detection","key":"DbovokHiJ9"}],"key":"SVuRD5gB2V"},{"type":"text","value":" — insertions, deletions, inversions, and other large-scale genomic rearrangements (\u003e50 bp) are far more reliably detected with long reads.","key":"GTXzLJ78DS"}],"key":"RrVOC2HQA9"}],"key":"nbt0vY7UBi"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Methylation analysis","key":"u83QRWmhG9"}],"key":"oDYAgbzjXZ"},{"type":"text","value":" — both PacBio and ONT can detect DNA methylation patterns (5mC, 6mA, 4mC) directly during sequencing without bisulfite conversion, enabling epigenomic characterization of microbial communities.","key":"wOv3RbKYY1"}],"key":"pn4V4UsLFQ"}],"key":"qVJEjKnTRt"},{"type":"listItem","spread":false,"checked":null,"children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Plasmid and mobile element recovery","key":"JJXXcgYPMm"}],"key":"m36y7gl6rU"},{"type":"text","value":" — short-read assemblies disproportionately fail for multi-copy DNA elements such as 16S genes, transposons, and plasmids. Long reads overcome this by spanning entire elements in single reads.","key":"mhzU9eF5I9"}],"key":"CCEGVORoCy"}],"key":"NWWQyoBojU"}],"key":"eh4fxSVc62"},{"type":"heading","depth":4,"children":[{"type":"text","value":"PacBio HiFi vs. ONT","key":"IO5G5Auvtx"}],"identifier":"pacbio-hifi-vs-ont","label":"PacBio HiFi vs. ONT","html_id":"pacbio-hifi-vs-ont","implicit":true,"key":"acAiSNnpFK"},{"type":"table","children":[{"type":"tableRow","children":[{"type":"tableCell","header":true,"children":[{"type":"text","value":"Consideration","key":"R2TeVoCO3F"}],"key":"sEyPQB0Lrk"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"PacBio HiFi","key":"iG0UroNtAa"}],"key":"ocAvqQ662l"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"ONT","key":"Y580kjqi7k"}],"key":"riRQxreM67"}],"key":"WXH6REchF4"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Best for","key":"d0EwClds8K"}],"key":"RFrxhyzoTS"}],"key":"NJEDtUOGDI"},{"type":"tableCell","children":[{"type":"text","value":"High-accuracy MAG assembly, variant calling","key":"mUnopGkUeW"}],"key":"BcwoytE47I"},{"type":"tableCell","children":[{"type":"text","value":"Rapid/field sequencing, ultra-long reads, direct RNA-seq","key":"NMBtiABlj2"}],"key":"hnYWtputmJ"}],"key":"H1w1koClbn"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Accuracy","key":"rypQPiqjui"}],"key":"gAKuAWtrI1"}],"key":"qijKMkUIDz"},{"type":"tableCell","children":[{"type":"text","value":"\u003e99.5% (comparable to Illumina)","key":"RncoRB9VhR"}],"key":"oA1mEIxVAi"},{"type":"tableCell","children":[{"type":"text","value":"97–99% (improving with R10 chemistry)","key":"mf4NOepJZy"}],"key":"t2WlyHRJ7I"}],"key":"uDisYJ9Bww"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Read length","key":"Yp97WGHcgv"}],"key":"nKP4l5u9Qf"}],"key":"HtjiKiFaWp"},{"type":"tableCell","children":[{"type":"text","value":"15–18 kb (consistent)","key":"OIWfD8pUGs"}],"key":"T3C0jywlA6"},{"type":"tableCell","children":[{"type":"text","value":"Up to 4 Mb (variable, avg ~20 kb)","key":"lyc565s3Ds"}],"key":"kwe8QX39bd"}],"key":"pLuoDz6PHG"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Methylation","key":"UP7jvHf0BF"}],"key":"D3zUslMAIT"}],"key":"m2M8ym6i9C"},{"type":"tableCell","children":[{"type":"text","value":"5mC, 6mA","key":"eiv4rd6E8t"}],"key":"nIb7IgoK5M"},{"type":"tableCell","children":[{"type":"text","value":"5mC, 6mA, 4mC + RNA modifications","key":"fpxUgNpyIk"}],"key":"u2qBqxrDdX"}],"key":"HThLOcrfYH"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Portability","key":"X9yzdaJEKF"}],"key":"UzKLXfMkvq"}],"key":"PkqZCZh1yT"},{"type":"tableCell","children":[{"type":"text","value":"Lab-based only","key":"l4pG8x9W4R"}],"key":"wLynIbR43Z"},{"type":"tableCell","children":[{"type":"text","value":"MinION/Flongle can be used in field settings","key":"M4u0eCY2oL"}],"key":"ZtUleJcIiy"}],"key":"cZy7ieqUBV"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Error correction needed","key":"JdijO2lBOX"}],"key":"eQkDaM1ZgZ"}],"key":"bXhupNN3Os"},{"type":"tableCell","children":[{"type":"text","value":"Rarely (HiFi reads are self-corrected)","key":"WdkmNDpawC"}],"key":"InUsRdHXil"},{"type":"tableCell","children":[{"type":"text","value":"Often recommended before assembly","key":"ckkUlSavAW"}],"key":"lOsaiGxfkw"}],"key":"McPL52pFjV"}],"key":"b9wikkunZ7"},{"type":"heading","depth":3,"children":[{"type":"text","value":"Hybrid Approach (Short + Long Reads)","key":"Y8h39jEzsN"}],"identifier":"hybrid-approach-short-long-reads","label":"Hybrid Approach (Short + Long Reads)","html_id":"hybrid-approach-short-long-reads","implicit":true,"key":"AbyNS7irQp"},{"type":"paragraph","children":[{"type":"text","value":"A hybrid sequencing strategy combines the strengths of both technologies: the high base-level accuracy of short reads with the long-range contiguity information from long reads. Hybrid assembly tools such as ","key":"gpHpDQQnQj"},{"type":"strong","children":[{"type":"text","value":"OPERA-MS","key":"j26Na53xXZ"}],"key":"Baas088TYW"},{"type":"text","value":", ","key":"EGy8coywr0"},{"type":"strong","children":[{"type":"text","value":"Unicycler","key":"MJAZ7J3jgS"}],"key":"DxUdC4l6hT"},{"type":"text","value":", and ","key":"cbPWCCS5v3"},{"type":"strong","children":[{"type":"text","value":"DBG2OLC","key":"MHsk0paWsq"}],"key":"PFyHS2CCtw"},{"type":"text","value":" can leverage both data types to produce assemblies with improved contiguity, gene completeness, and binning accuracy compared to short-read-only approaches.","key":"Oj8v2HJpGa"}],"key":"yAFqiBTkSk"},{"type":"paragraph","children":[{"type":"text","value":"However, hybrid approaches involve higher costs (two library preparations and sequencing runs) and can introduce biases from combining different library preparation chemistries. With the increasing accuracy of PacBio HiFi reads (\u003eQ30), long-read-only assemblies are now achieving comparable or superior quality to hybrid assemblies for many applications, making the hybrid approach less necessary than in previous years.","key":"E2xHmLzG7S"}],"key":"vqBRCWkrRE"},{"type":"heading","depth":3,"children":[{"type":"text","value":"Decision Framework","key":"L7rT5PCtUR"}],"identifier":"decision-framework","label":"Decision Framework","html_id":"decision-framework","implicit":true,"key":"HEFLRYZIbr"},{"type":"paragraph","children":[{"type":"text","value":"The choice of sequencing strategy depends on the study’s objectives:","key":"DCE5guuPU1"}],"key":"lkTTl5dLAR"},{"type":"table","children":[{"type":"tableRow","children":[{"type":"tableCell","header":true,"children":[{"type":"text","value":"Research Goal","key":"l6BQ8ZTcSF"}],"key":"PFVTCiX9Mf"},{"type":"tableCell","header":true,"children":[{"type":"text","value":"Recommended Strategy","key":"X1UMIXtUyb"}],"key":"veT3q58ZfG"}],"key":"kjk78RTOn5"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Large-scale community profiling (many samples)","key":"ldxvpKH923"}],"key":"zWv4BcXJXj"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Illumina short reads","key":"VjcQi9meKv"}],"key":"GrHz3TuYaw"}],"key":"jX6SHYu1Cl"}],"key":"q29ClTHUpq"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Complete genome recovery (MAGs)","key":"Zl5rEtL2ul"}],"key":"ecenslLB5R"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"PacBio HiFi","key":"WBvLrJzXUt"}],"key":"tLvrh2J0GS"},{"type":"text","value":" or ","key":"yYY0ZwuQRq"},{"type":"strong","children":[{"type":"text","value":"ONT + polishing","key":"Qk6EyzhvTV"}],"key":"tn1AFMGyfi"}],"key":"ZEyqcT8CLJ"}],"key":"atIN43Lu01"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Strain-level resolution","key":"uYLyk3zqlH"}],"key":"uYwGkMY85y"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Long reads","key":"JIv1wpsnd3"}],"key":"QIc1oGqhUz"},{"type":"text","value":" (PacBio or ONT)","key":"ZJrnS20E77"}],"key":"OmvYXGjOdQ"}],"key":"QUmkgSBFuF"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Plasmid / mobile element tracking","key":"rwE8942R3e"}],"key":"RuCeqKYfGO"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Long reads","key":"IYTUqurJdx"}],"key":"bHKlstgpsi"}],"key":"rYC5MIWL84"}],"key":"eDGR9khfln"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Antibiotic resistance gene surveillance","key":"IUi6WeyNFP"}],"key":"wO1BtT7U7c"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"ONT","key":"B2FqsS57tf"}],"key":"soqiO8kEym"},{"type":"text","value":" (rapid, portable)","key":"ETHKj9i6uN"}],"key":"r8eMUJ4cDo"}],"key":"TFYSIeXEhI"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Low-biomass / clinical samples","key":"WO6XXZM4AP"}],"key":"wd8XCglG1x"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Illumina","key":"gBAnV7elRx"}],"key":"pfiNCx5qkR"},{"type":"text","value":" (low DNA input)","key":"Zm2CNM9RqO"}],"key":"Gy5hnA7zLc"}],"key":"NYcjLVzpDV"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Multi-omics integration (DNA + RNA)","key":"mB19rqWrR9"}],"key":"uaqu0sT0Ij"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"ONT","key":"zt4EtpKadX"}],"key":"YZ39fCA9ge"},{"type":"text","value":" (direct RNA-seq capability)","key":"nRJZoaFYkh"}],"key":"J1uwzVVHgk"}],"key":"wf0O6CQ73E"},{"type":"tableRow","children":[{"type":"tableCell","children":[{"type":"text","value":"Budget-constrained study","key":"vcJaRWaaTc"}],"key":"wpuqfNuKJP"},{"type":"tableCell","children":[{"type":"strong","children":[{"type":"text","value":"Illumina","key":"CM6lTxcW9M"}],"key":"RjgE2MQxE6"},{"type":"text","value":" (lowest cost/Gb)","key":"aG0xAC1XR0"}],"key":"CNCmZvb3sI"}],"key":"zlOgeeFO03"}],"key":"JkTi0sFcZz"},{"type":"paragraph","children":[{"type":"text","value":"In this SOP, we provide analysis pipelines for both ","key":"PrHcieDXwT"},{"type":"strong","children":[{"type":"text","value":"Illumina short-read","key":"zUVuWp7bnT"}],"key":"UmQhKMFu8t"},{"type":"text","value":" and ","key":"xRMzJq0veK"},{"type":"strong","children":[{"type":"text","value":"PacBio HiFi long-read","key":"ImQxGfcCXe"}],"key":"ml8fbH7LhK"},{"type":"text","value":" metagenomics, covering sample preparation through computational analysis for each approach.","key":"ojafxEHGqU"}],"key":"A1yVszrHRt"}],"key":"AELRelpoIs"}],"key":"g9dbkLyfHB"}],"key":"BHD88nvfEk"},"references":{"cite":{"order":[],"data":{}}},"footer":{"navigation":{"prev":{"title":"META-Insight","url":"/","group":"META-Insight"},"next":{"title":"Preparation for sequencing","url":"/sequencing","group":"Metagenomics"}}},"domain":"http://localhost:3000"},"project":{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Metaproteomics"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"Phylogenetic","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metaproteomics"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metaproteome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Metaproteomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}}},"actionData":null,"errors":null},"future":{"unstable_dev":false,"unstable_postcss":false,"unstable_tailwind":false,"v2_errorBoundary":true,"v2_headers":true,"v2_meta":true,"v2_normalizeFormMethod":true,"v2_routeConvention":true}};</script><script type="module" async="">import "/metainsight-book/build/manifest-3595279D.js";
import * as route0 from "/metainsight-book/build/root-SIO6LUTY.js";
import * as route1 from "/metainsight-book/build/routes/$-PRP77N34.js";
window.__remixRouteModules = {"root":route0,"routes/$":route1};

import("/metainsight-book/build/entry.client-PCJPW7TK.js");</script></body></html>