<!DOCTYPE html><html lang="en" class="" style="scroll-padding:60px"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>Metagenomic analysis tools - META-Insight</title><meta property="og:title" content="Metagenomic analysis tools - META-Insight"/><meta name="generator" content="mystmd"/><meta name="description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta property="og:description" content="Meta-omics Analysis Protocols for Microbiome Research"/><meta name="keywords" content="metagenomics, metatranscriptomics, metaproteomics, microbiome, bioinformatics"/><meta name="image" content="/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png"/><meta property="og:image" content="/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png"/><link rel="stylesheet" href="/metainsight-book/build/_assets/app-MOQGDXHO.css"/><link rel="stylesheet" href="/metainsight-book/build/_assets/thebe-core-VKVHG5VY.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/jupyter-matplotlib@0.11.3/css/mpl_widget.css"/><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.css" integrity="sha384-MlJdn/WNKDGXveldHDdyRP1R4CTHr3FeuDNfhsLPYrq2t0UBkUdK2jyTnXPEK1NQ" crossorigin="anonymous"/><link rel="icon" href="/metainsight-book/favicon.ico"/><link rel="stylesheet" href="/metainsight-book/myst-theme.css"/><script>
  const savedTheme = localStorage.getItem("myst:theme");
  const theme = window.matchMedia("(prefers-color-scheme: light)").matches ? 'light' : 'dark';
  const classes = document.documentElement.classList;
  const hasAnyTheme = classes.contains('light') || classes.contains('dark');
  if (!hasAnyTheme) classes.add(savedTheme ?? theme);
</script></head><body class="m-0 transition-colors duration-500 bg-white dark:bg-stone-900"><div class="myst-skip-to-article fixed top-1 left-1 h-[0px] w-[0px] focus-within:z-40 focus-within:h-auto focus-within:w-auto bg-white overflow-hidden focus-within:p-2 focus-within:ring-1" aria-label="skip to content options"><a href="#skip-to-frontmatter" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article frontmatter</a><a href="#skip-to-article" class="myst-skip-to-link block px-2 py-1 text-black underline">Skip to article content</a></div><dialog id="myst-no-css" style="position:fixed;left:0px;top:0px;width:100vw;height:100vh;font-size:4rem;padding:1rem;color:black;background:white"><strong>Site not loading correctly?</strong><p>This may be due to an incorrect <code>BASE_URL</code> configuration. See<!-- --> <a href="https://mystmd.org/guide/deployment#deploy-base-url">the MyST Documentation</a> <!-- -->for reference.</p><script>
    (() => {
            // Test for has-styling variable set by the MyST stylesheet
            const node = document.currentScript.parentNode;
            const hasCSS = window.getComputedStyle(node).getPropertyValue("--has-styling");
            if (hasCSS === ""){
                    node.showModal();
            }

    })()
</script></dialog><div class="myst-top-nav bg-white/80 backdrop-blur dark:bg-stone-900/80 shadow dark:shadow-stone-700 p-3 md:px-8 sticky w-screen top-0 z-30 h-[60px]"><nav class="myst-top-nav-bar flex items-center justify-between flex-nowrap max-w-[1440px] mx-auto"><div class="flex flex-row xl:min-w-[19.5rem] mr-2 sm:mr-7 justify-start items-center shrink-0"><div class="block xl:hidden"><button class="myst-top-nav-menu-button flex items-center justify-center border-stone-400 text-stone-800 hover:text-stone-900 dark:text-stone-200 hover:dark:text-stone-100 w-10 h-10"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem"><path fill-rule="evenodd" d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75ZM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12Zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75Z" clip-rule="evenodd"></path></svg><span class="sr-only">Open Menu</span></button></div><a class="myst-home-link flex items-center ml-3 dark:text-white w-fit md:ml-5 xl:ml-7" href="/metainsight-book/"><div class="myst-home-link-logo mr-3 flex items-center dark:bg-white dark:rounded px-1"><img src="/metainsight-book/build/META-Insight_logo-f832acbac4efa00dfb06f54c7d2985ee.svg" class="h-9" height="2.25rem"/></div><span class="text-md sm:text-xl tracking-tight sm:mr-5 sr-only">Made with MyST</span></a></div><div class="flex items-center flex-grow w-auto"><div class="flex-grow hidden text-md lg:block"></div><div class="flex-grow block"></div><button type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R75cp:" data-state="closed" class="myst-search-bar flex items-center h-10 aspect-square sm:w-64 text-left text-gray-600 dark:text-gray-300 border border-gray-300 dark:border-gray-600 rounded-lg bg-gray-50 dark:bg-gray-700 myst-search-bar-disabled hover:ring-blue-500 dark:hover:ring-blue-500 hover:border-blue-500 dark:hover:border-blue-500"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="p-2.5 h-10 w-10 aspect-square"><path fill-rule="evenodd" d="M10.5 3.75a6.75 6.75 0 1 0 0 13.5 6.75 6.75 0 0 0 0-13.5ZM2.25 10.5a8.25 8.25 0 1 1 14.59 5.28l4.69 4.69a.75.75 0 1 1-1.06 1.06l-4.69-4.69A8.25 8.25 0 0 1 2.25 10.5Z" clip-rule="evenodd"></path></svg><span class="myst-search-text-placeholder hidden sm:block grow">Search</span><div aria-hidden="true" class="myst-search-shortcut items-center hidden mx-1 font-mono text-sm text-gray-600 dark:text-gray-300 sm:flex gap-x-1"><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none hide-mac">CTRL</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none show-mac">⌘</kbd><kbd class="px-2 py-1 border border-gray-300 dark:border-gray-600 rounded-md shadow-[0px_2px_0px_0px_rgba(0,0,0,0.08)] dark:shadow-none ">K</kbd><script>
;(() => {
const script = document.currentScript;
const root = script.parentElement;

const isMac = /mac/i.test(
      window.navigator.userAgentData?.platform ?? window.navigator.userAgent,
    );
root.querySelectorAll(".hide-mac").forEach(node => {node.classList.add(isMac ? "hidden" : "block")});
root.querySelectorAll(".show-mac").forEach(node => {node.classList.add(!isMac ? "hidden" : "block")});
})()</script></div></button><button class="myst-theme-button theme rounded-full aspect-square border border-stone-700 dark:border-white hover:bg-neutral-100 border-solid overflow-hidden text-stone-700 dark:text-white hover:text-stone-500 dark:hover:text-neutral-800 w-10 h-10 mx-3" title="Toggle theme between light and dark mode" aria-label="Toggle theme between light and dark mode"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-moon-icon h-full w-full p-0.5 hidden dark:block"><path fill-rule="evenodd" d="M9.528 1.718a.75.75 0 0 1 .162.819A8.97 8.97 0 0 0 9 6a9 9 0 0 0 9 9 8.97 8.97 0 0 0 3.463-.69.75.75 0 0 1 .981.98 10.503 10.503 0 0 1-9.694 6.46c-5.799 0-10.5-4.7-10.5-10.5 0-4.368 2.667-8.112 6.46-9.694a.75.75 0 0 1 .818.162Z" clip-rule="evenodd"></path></svg><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="myst-theme-sun-icon h-full w-full p-0.5 dark:hidden"><path stroke-linecap="round" stroke-linejoin="round" d="M12 3v2.25m6.364.386-1.591 1.591M21 12h-2.25m-.386 6.364-1.591-1.591M12 18.75V21m-4.773-4.227-1.591 1.591M5.25 12H3m4.227-4.773L5.636 5.636M15.75 12a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0Z"></path></svg></button><div class="block sm:hidden"></div><div class="hidden sm:block"></div></div></nav></div><div class="myst-primary-sidebar fixed xl:article-grid grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px] hidden z-10" style="top:60px"><div class="myst-primary-sidebar-pointer pointer-events-auto xl:col-margin-left flex-col overflow-hidden hidden xl:flex"><div class="myst-primary-sidebar-nav flex-grow py-6 overflow-y-auto primary-scrollbar"><nav aria-label="Navigation" class="myst-primary-sidebar-topnav overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px] lg:hidden"><div class="w-full px-1 dark:text-white font-medium"></div></nav><div class="my-3 border-b-2 lg:hidden"></div><nav aria-label="Table of Contents" class="myst-primary-sidebar-toc flex-grow overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px]"><div class="myst-toc w-full px-1 dark:text-white"><a title="META-Insight" class="block break-words focus:outline outline-blue-200 outline-2 rounded myst-toc-item p-2 my-1 rounded-lg hover:bg-slate-300/30 font-bold" href="/metainsight-book/">META-Insight</a><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metagenomics" class="block break-words rounded py-2 grow cursor-pointer">Metagenomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rmpsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rmpsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Metatranscriptomics" class="block break-words rounded py-2 grow cursor-pointer">Metatranscriptomics</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:Rupsp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:Rupsp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div><div data-state="closed" class="w-full"><div class="myst-toc-item flex flex-row w-full gap-2 pl-2 my-1 text-left rounded-lg outline-none hover:bg-slate-300/30"><div title="Translatome" class="block break-words rounded py-2 grow cursor-pointer">Translatome</div><button class="self-stretch flex items-center flex-none px-1 rounded-l-md group hover:bg-slate-300/30 focus-visible:outline outline-blue-200 outline-2" aria-label="Open Folder" type="button" aria-controls="radix-:R16psp:" aria-expanded="false" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" data-slot="icon" class="transition-transform duration-300 group-data-[state=open]:rotate-90 text-text-slate-700 dark:text-slate-100" height="1.5rem" width="1.5rem"><path fill-rule="evenodd" d="M16.28 11.47a.75.75 0 0 1 0 1.06l-7.5 7.5a.75.75 0 0 1-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 0 1 1.06-1.06l7.5 7.5Z" clip-rule="evenodd"></path></svg></button></div><div data-state="closed" id="radix-:R16psp:" hidden="" class="pl-3 pr-[2px] collapsible-content"></div></div></div></nav></div><div class="myst-primary-sidebar-footer flex-none py-6 transition-all duration-700 translate-y-6 opacity-0"><a class="myst-made-with-myst flex mx-auto text-gray-700 w-fit hover:text-blue-700 dark:text-gray-200 dark:hover:text-blue-400" href="https://mystmd.org/made-with-myst" target="_blank" rel="noreferrer"><svg style="width:24px;height:24px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" stroke="none"><g id="icon"><path fill="currentColor" d="M23.8,54.8v-3.6l4.7-0.8V17.5l-4.7-0.8V13H36l13.4,31.7h0.2l13-31.7h12.6v3.6l-4.7,0.8v32.9l4.7,0.8v3.6h-15
          v-3.6l4.9-0.8V20.8H65L51.4,53.3h-3.8l-14-32.5h-0.1l0.2,17.4v12.1l5,0.8v3.6H23.8z"></path><path fill="#F37726" d="M47,86.9c0-5.9-3.4-8.8-10.1-8.8h-8.4c-5.2,0-9.4-1.3-12.5-3.8c-3.1-2.5-5.4-6.2-6.8-11l4.8-1.6
          c1.8,5.6,6.4,8.6,13.8,8.8h9.2c6.4,0,10.8,2.5,13.1,7.5c2.3-5,6.7-7.5,13.1-7.5h8.4c7.8,0,12.7-2.9,14.6-8.7l4.8,1.6
          c-1.4,4.9-3.6,8.6-6.8,11.1c-3.1,2.5-7.3,3.7-12.4,3.8H63c-6.7,0-10,2.9-10,8.8"></path></g></svg><span class="self-center ml-2 text-sm">Made with MyST</span></a></div></div></div><main class="article-grid grid-gap"><article class="article-grid subgrid-gap col-screen article content"><div class="hidden"></div><div id="skip-to-frontmatter" aria-label="article frontmatter" class="myst-fm-block mb-8 pt-9"><div class="myst-fm-block-header flex items-center mb-5 h-6 text-sm font-light"><div class="flex-grow"></div><div class="myst-fm-block-badges"><a href="https://github.com/aababc1/metainsight-book" title="GitHub Repository: aababc1/metainsight-book" target="_blank" rel="noopener noreferrer" class="myst-fm-github-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" width="1.25rem" height="1.25rem" class="myst-fm-github-icon inline-block mr-1 opacity-60 hover:opacity-100"><path d="M12 2.5c-5.4 0-9.8 4.4-9.8 9.7 0 4.3 2.8 8 6.7 9.2.5.1.7-.2.7-.5v-1.8c-2.4.5-3.1-.6-3.3-1.1-.1-.3-.6-1.1-1-1.4-.3-.2-.8-.6 0-.6s1.3.7 1.5 1c.9 1.5 2.3 1.1 2.8.8.1-.6.3-1.1.6-1.3-2.2-.2-4.4-1.1-4.4-4.8 0-1.1.4-1.9 1-2.6-.1-.2-.4-1.2.1-2.6 0 0 .8-.3 2.7 1 .8-.2 1.6-.3 2.4-.3.8 0 1.7.1 2.4.3 1.9-1.3 2.7-1 2.7-1 .5 1.3.2 2.3.1 2.6.6.7 1 1.5 1 2.6 0 3.7-2.3 4.6-4.4 4.8.4.3.7.9.7 1.8V21c0 .3.2.6.7.5 3.9-1.3 6.6-4.9 6.6-9.2 0-5.4-4.4-9.8-9.8-9.8z"></path></svg></a></div><a href="https://github.com/aababc1/metainsight-book/edit/main/metagenomics/Installation.md" title="Edit This Page" target="_blank" rel="noopener noreferrer" class="myst-fm-edit-link text-inherit hover:text-inherit"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-edit-icon inline-block mr-1 opacity-60 hover:opacity-100"><path stroke-linecap="round" stroke-linejoin="round" d="m16.862 4.487 1.687-1.688a1.875 1.875 0 1 1 2.652 2.652L10.582 16.07a4.5 4.5 0 0 1-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 0 1 1.13-1.897l8.932-8.931Zm0 0L19.5 7.125M18 14v4.75A2.25 2.25 0 0 1 15.75 21H5.25A2.25 2.25 0 0 1 3 18.75V8.25A2.25 2.25 0 0 1 5.25 6H10"></path></svg></a><div class="myst-fm-downloads-dropdown relative flex inline-block mx-1 grow-0" data-headlessui-state=""><button class="myst-fm-downloads-button relative ml-2 -mr-1" id="headlessui-menu-button-:Rs8ucp:" type="button" aria-haspopup="menu" aria-expanded="false" data-headlessui-state=""><span class="sr-only">Downloads</span><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.25rem" height="1.25rem" class="myst-fm-downloads-icon"><title>Download</title><path stroke-linecap="round" stroke-linejoin="round" d="M3 16.5v2.25A2.25 2.25 0 0 0 5.25 21h13.5A2.25 2.25 0 0 0 21 18.75V16.5M16.5 12 12 16.5m0 0L7.5 12m4.5 4.5V3"></path></svg></button></div></div><h1 class="myst-fm-block-title mb-0">Metagenomic analysis tools</h1><header class="myst-fm-authors-affiliations mt-4 not-prose"><div class="myst-fm-authors-list"><span class="myst-fm-author font-semibold text-sm myst-fm-author-item inline-block"><button class="myst-fm-author-popover focus:shadow-[0_0_0_2px] focus:shadow-black outline-none hover:underline" aria-label="Author Details" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:R78ucp:" data-state="closed"><span class="myst-fm-author-name">META-Insight Team</span></button></span></div></header></div><div class="block my-10 lg:sticky lg:z-10 lg:h-0 lg:pt-0 lg:my-0 lg:ml-10 lg:col-margin-right" style="top:60px"><nav></nav></div><div id="skip-to-article"></div><p>This chapter provides a comprehensive guide to installing the bioinformatics software required for the metagenomic analysis pipeline described in this protocol. Metagenomic analysis relies on a diverse set of specialized tools spanning quality control, assembly, binning, taxonomic and functional annotation, and phylogenetic reconstruction. Managing these tools and their dependencies can be challenging, particularly when different programs require incompatible software versions. We recommend using Conda, a cross-platform package and environment manager, which allows each tool to be installed in an isolated environment, thereby avoiding version conflicts and ensuring reproducibility across different computing systems.</p><ul><li><p>Preprocessing of the sequencing reads : Kneaddata, minimap2, samtools</p></li><li><p>Read-based profiling : HUMAnN 3.0, MetaPhlAn</p></li><li><p>De novo assembly : MEGAHIT, Quast, hifiasm_meta, Bandage</p></li><li><p>Taxonomic annotation : Kaiju</p></li><li><p>Functional annotation : Prodigal, EggNOG</p></li><li><p>Binning : MetaBAT2, BBmap, minimap2, samtools, CheckM</p></li><li><p>Phylogenetic tree construction : GTDB-tk, PhyloPhlAn3</p></li></ul><figure id="wQAgfLt76H" class=""><img id="lUNDXSjxFt" style="margin-left:auto;margin-right:auto" src="/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png" alt="Metagenomics analysis pipeline" data-canonical-url="https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_4_1.png?raw=true" class=""/><figcaption class="group"><p><strong>Metagenomics analysis pipeline</strong></p></figcaption></figure><h2 id="installation" class="relative group"><span class="heading-text">Installation</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#installation" title="Link to this Section" aria-label="Link to this Section">¶</a></h2><h3 id="an-introduction-to-conda" class="relative group"><span class="heading-text">An introduction to Conda</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#an-introduction-to-conda" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><p>Metagenomic analysis necessitates the installation of various analysis programs, a process that can sometimes result in conflicts or version dependencies between these programs.
For instance, when program A relies on a specific version of program B, and program C depends on a different version of program B, it can be challenging and time-consuming for beginners to set up program C.
In such scenarios, conda, a Python package and environment management tool, offers an effective solution for resolving these version conflicts.</p><p>For this Standard Operating Procedure (SOP), we recommend installing the analysis tools using conda.
You can find the official documentation for conda at the following <a target="_blank" rel="noreferrer" href="https://docs.conda.io/projects/conda/en/latest/user-guide/index.html" class="link">website<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a>.
This approach ensures a smoother and more efficient installation process, making metagenomic analysis more accessible and less prone to compatibility issues.</p><h3 id="making-a-new-environment" class="relative group"><span class="heading-text">Making a new environment</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#making-a-new-environment" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><p>It is highly recommended to create a new conda environment every time you install a program. Doing so helps to avoid conflicts between programs and ensures a clean, isolated environment for each analysis tool.
An example of how to create a new conda environment is provided below.</p><div class="relative myst-code group not-prose my-5 text-sm shadow hover:shadow-md dark:shadow-2xl dark:shadow-neutral-900 bg-stone-200/10"><pre class="block overflow-auto p-3 myst-code-body hljs" style="background-color:unset"><code class="language-bash" style="white-space:pre"># Create a new conda environment
$ conda create -n new-env </code></pre><button title="Copy to Clipboard" class="inline-flex items-center opacity-0 group-hover:opacity-100 hover:opacity-100 focus:opacity-100 active:opacity-100 cursor-pointer ml-2 transition-color duration-200 ease-in-out text-blue-400 hover:text-blue-500 absolute right-1 myst-code-copy-icon top-1" aria-pressed="false" aria-label="Copy code to clipboard"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="24" height="24"><path stroke-linecap="round" stroke-linejoin="round" d="M15.75 17.25v3.375c0 .621-.504 1.125-1.125 1.125h-9.75a1.125 1.125 0 0 1-1.125-1.125V7.875c0-.621.504-1.125 1.125-1.125H6.75a9.06 9.06 0 0 1 1.5.124m7.5 10.376h3.375c.621 0 1.125-.504 1.125-1.125V11.25c0-4.46-3.243-8.161-7.5-8.876a9.06 9.06 0 0 0-1.5-.124H9.375c-.621 0-1.125.504-1.125 1.125v3.5m7.5 10.375H9.375a1.125 1.125 0 0 1-1.125-1.125v-9.25m12 6.625v-1.875a3.375 3.375 0 0 0-3.375-3.375h-1.5a1.125 1.125 0 0 1-1.125-1.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H9.75"></path></svg></button></div><p>Use the <code>conda create</code> command with the <code>-n</code> flag to designate and save the name of your new environment.</p><h3 id="entering-exiting-an-environment" class="relative group"><span class="heading-text">Entering &amp; exiting an environment</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#entering-exiting-an-environment" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><div class="relative myst-code group not-prose my-5 text-sm shadow hover:shadow-md dark:shadow-2xl dark:shadow-neutral-900 bg-stone-200/10"><pre class="block overflow-auto p-3 myst-code-body hljs" style="background-color:unset"><code class="language-bash" style="white-space:pre"># Enter that environment
$ conda activate new-env
# Exit whatever conda environment we are running
$ conda deactivate</code></pre><button title="Copy to Clipboard" class="inline-flex items-center opacity-0 group-hover:opacity-100 hover:opacity-100 focus:opacity-100 active:opacity-100 cursor-pointer ml-2 transition-color duration-200 ease-in-out text-blue-400 hover:text-blue-500 absolute right-1 myst-code-copy-icon top-1" aria-pressed="false" aria-label="Copy code to clipboard"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="24" height="24"><path stroke-linecap="round" stroke-linejoin="round" d="M15.75 17.25v3.375c0 .621-.504 1.125-1.125 1.125h-9.75a1.125 1.125 0 0 1-1.125-1.125V7.875c0-.621.504-1.125 1.125-1.125H6.75a9.06 9.06 0 0 1 1.5.124m7.5 10.376h3.375c.621 0 1.125-.504 1.125-1.125V11.25c0-4.46-3.243-8.161-7.5-8.876a9.06 9.06 0 0 0-1.5-.124H9.375c-.621 0-1.125.504-1.125 1.125v3.5m7.5 10.375H9.375a1.125 1.125 0 0 1-1.125-1.125v-9.25m12 6.625v-1.875a3.375 3.375 0 0 0-3.375-3.375h-1.5a1.125 1.125 0 0 1-1.125-1.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H9.75"></path></svg></button></div><p>The table below provides example installation codes and Anaconda addresses for the programs used in this SOP.</p><table class=""><tbody><tr class=""><th class="">Program</th><th class="">Installation code</th><th class="">ANACONDA addresses</th></tr><tr class=""><td class="">KNEDDATA</td><td class=""><code>conda create -n kneaddata -c bioconda kneaddata</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/kneaddata" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/kneaddata<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">HUMANN3.0</td><td class=""><code>conda create -n humann -c biobakery humann</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/biobakery/humann" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/biobakery<wbr/>/humann<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">METAPHLAN3.0</td><td class=""><code>conda create –n metaphlan -c bioconda metaphlan</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/metaphlan" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/metaphlan<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">MEGAHIT</td><td class=""><code>conda create –n MEGAHIT -c bioconda MEGAHIT</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/MEGAHIT" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/MEGAHIT<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">QUAST</td><td class=""><code>conda create –n quast -c bioconda quast</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/quast" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/quast<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">KAIJU</td><td class=""><code>conda create –n kaiju -c bioconda kaiju</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/kaiju" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/kaiju<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">MINIMAP2</td><td class=""><code>conda create –n minimap2 -c bioconda minimap2</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/minimap2" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/minimap2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">HIFIASM_META</td><td class=""><code>conda create –n hifiasm_meta -c bioconda hifiasm_meta</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/hifiasm_meta" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/hifiasm<wbr/>_meta<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">KRONA</td><td class=""><code>conda create –n krona -c bioconda krona</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/krona" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/krona<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">PRODIGAL</td><td class=""><code>conda create –n prodigal -c bioconda prodigal</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/prodigal" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/prodigal<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">EGGNOG</td><td class=""><code>conda create –n eggnog -c bioconda eggnog-mapper</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/eggnog-mapper" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/eggnog<wbr/>-mapper<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">METABAT2</td><td class=""><code>conda create –n MetaBAT2 -c bioconda MetaBAT2</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/MetaBAT2" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/MetaBAT2<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">BBMAP</td><td class=""><code>conda create –n bbmap -c bioconda bbmap</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/bbmap" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/bbmap<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">SAMTOOLS</td><td class=""><code>conda create –n samtools -c bioconda samtools</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/samtools" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/samtools<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">CHECKM</td><td class=""><code>conda create -n CheckM python=3.9</code> <br/> <code>conda activate CheckM</code> <br/><code>conda install -c bioconda numpy matplotlib pysam</code><br/><code>conda install -c bioconda hmmer prodigal pplacer</code><br/><code>pip3 install CheckM-genome</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/CheckM-genome" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/CheckM<wbr/>-genome<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">GTDB-TK</td><td class=""><code>conda create -n gtdbtk-2.1.1 -c conda-forge -c bioconda gtdbtk=2.1.1</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/gtdbtk" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/gtdbtk<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr><tr class=""><td class="">PHYLOPHLAN3</td><td class=""><code>conda create –n phylophlan -c bioconda phylophlan</code></td><td class=""><a target="_blank" rel="noreferrer" href="https://anaconda.org/bioconda/phylophlan" class="link">https://<wbr/>anaconda<wbr/>.org<wbr/>/bioconda<wbr/>/phylophlan<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" class="link-icon"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 0 0 3 8.25v10.5A2.25 2.25 0 0 0 5.25 21h10.5A2.25 2.25 0 0 0 18 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25"></path></svg></a></td></tr></tbody></table><h3 id="references" class="relative group"><span class="heading-text">References</span><a class="no-underline text-inherit hover:text-inherit inline-block w-0 px-0 translate-x-[10px] font-normal select-none transition-opacity opacity-0 focus:opacity-100 group-hover:opacity-70" href="#references" title="Link to this Section" aria-label="Link to this Section">¶</a></h3><ol start="1"><li><p>Li D, Liu CM, Luo R, et al. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. <em>Bioinformatics</em>. 2015;31(10):1674-1676. doi:10.1093/bioinformatics/btv033</p></li><li><p>Nurk S, Meleshko D, Korobeynikov A, et al. metaSPAdes: a new versatile metagenomic assembler. <em>Genome Res</em>. 2017;27(5):824-834. doi:10.1101/gr.213959.116</p></li><li><p>Kang DD, Li F, Kirton ES, et al. MetaBAT 2: an improved binning algorithm, assembly-independent binning and recovering metagenome-assembled genomes from heterogeneous sources. <em>PeerJ</em>. 2019;7:e7359. doi:10.7717/peerj.7359</p></li><li><p>Parks DH, Imelfort M, Skennerton CT, et al. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <em>Genome Res</em>. 2015;25(7):1043-1055. doi:10.1101/gr.186072.114</p></li><li><p>Chaumeil PA, Mussig AJ, Hugenholtz P, et al. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. <em>Bioinformatics</em>. 2020;36(6):1925-1927. doi:10.1093/bioinformatics/btz848</p></li><li><p>Hyatt D, Chen GL, Locascio PF, et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. <em>BMC Bioinformatics</em>. 2010;11:119. doi:10.1186/1471-2105-11-119</p></li><li><p>Huerta-Cepas J, Forslund K, Coelho LP, et al. eggNOG 4.5: a hierarchical orthology framework with improved functional annotations for eukaryotic, prokaryotic and viral sequences. <em>Nucleic Acids Res</em>. 2016;44(D1):D286-D293. doi:10.1093/nar/gkv1248</p></li><li><p>Asnicar F, Thomas AM, Beghini F, et al. Serotype and clonal analysis of pathogenic Listeria monocytogenes strains. <em>Nat Commun</em>. 2020;11(1):2500. doi:10.1038/s41467-020-16366-7</p></li><li><p>Franzosa EA, McIver LJ, Rahnavard G, et al. Species-level functional profiling of metagenomes and metatranscriptomes. <em>Nat Methods</em>. 2018;15(11):962-968. doi:10.1038/s41592-018-0176-y</p></li><li><p>Segata N, Waldron L, Ballarini A, et al. Metagenomic microbial community profiling using unique clade-specific marker genes. <em>Nat Methods</em>. 2012;9(8):811-814. doi:10.1038/nmeth.2066</p></li></ol><div class="myst-backmatter-parts"></div><div class="myst-footer-links flex pt-10 mb-10 space-x-4"><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-prev" href="/metainsight-book/sequencing"><div class="flex h-full align-middle"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:-translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5 3 12m0 0 7.5-7.5M3 12h18"></path></svg><div class="flex-grow text-right"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metagenomics</div>Preparation for sequencing</div></div></a><a class="myst-footer-link flex-1 block p-4 font-normal text-gray-600 no-underline border border-gray-200 rounded shadow-sm group hover:border-blue-600 dark:hover:border-blue-400 hover:text-blue-600 dark:hover:text-blue-400 dark:text-gray-100 dark:border-gray-500 hover:shadow-lg dark:shadow-neutral-700 myst-footer-link-next" href="/metainsight-book/preprocessing"><div class="flex h-full align-middle"><div class="flex-grow"><div class="myst-footer-link-group text-xs text-gray-500 dark:text-gray-400">Metagenomics</div>Preprocessing of the sequencing reads</div><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true" data-slot="icon" width="1.5rem" height="1.5rem" class="myst-footer-link-icon self-center transition-transform group-hover:translate-x-1 shrink-0"><path stroke-linecap="round" stroke-linejoin="round" d="M13.5 4.5 21 12m0 0-7.5 7.5M21 12H3"></path></svg></div></a></div></article></main><script>((a,l)=>{if(!window.history.state||!window.history.state.key){let u=Math.random().toString(32).slice(2);window.history.replaceState({key:u},"")}try{let d=JSON.parse(sessionStorage.getItem(a)||"{}")[l||window.history.state.key];typeof d=="number"&&window.scrollTo(0,d)}catch(u){console.error(u),sessionStorage.removeItem(a)}})("positions", null)</script><link rel="modulepreload" href="/metainsight-book/build/entry.client-PCJPW7TK.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-AQ2CODAG.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-JJXTQVMA.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-OZE3FFNP.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-CH4FVTDV.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-C4DFGG5C.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-J7TUH54J.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-FZ2S7OYD.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-JEM6JXYA.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-34XIY2DH.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-KQM5FBHR.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-OCWQY3HK.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-7HNKBP4B.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-CUKUDK3R.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-3EBOCCHJ.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-O4VQNZ62.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-4OEDG4JQ.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-GUCIBHGO.js"/><link rel="modulepreload" href="/metainsight-book/build/root-SIO6LUTY.js"/><link rel="modulepreload" href="/metainsight-book/build/_shared/chunk-FAHZZXAC.js"/><link rel="modulepreload" href="/metainsight-book/build/routes/$-PRP77N34.js"/><script>window.__remixContext = {"url":"/installation","state":{"loaderData":{"root":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/metainsight-book/build/META-Insight_logo-f832acbac4efa00dfb06f54c7d2985ee.svg","logo":"/metainsight-book/build/META-Insight_logo-f832acbac4efa00dfb06f54c7d2985ee.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Translatome"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"References","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Translatome"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Translatome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Translatome Analysis Tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"CONTENT_CDN_PORT":"3100","MODE":"static","BASE_URL":"/metainsight-book"},"routes/$":{"config":{"version":3,"myst":"1.8.0","options":{"favicon":"/metainsight-book/build/META-Insight_logo-f832acbac4efa00dfb06f54c7d2985ee.svg","logo":"/metainsight-book/build/META-Insight_logo-f832acbac4efa00dfb06f54c7d2985ee.svg"},"nav":[],"actions":[],"projects":[{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Translatome"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"References","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Translatome"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Translatome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Translatome Analysis Tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}]},"page":{"version":3,"kind":"Article","sha256":"d831eb8c67e0e11635c91d2ca2cd19172cc826c01acaf3b88dc5adab5062a82d","slug":"installation","location":"/metagenomics/Installation.md","dependencies":[],"frontmatter":{"title":"Metagenomic analysis tools","content_includes_title":false,"authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"numbering":{"title":{"offset":1}},"source_url":"https://github.com/aababc1/metainsight-book/blob/main/metagenomics/Installation.md","edit_url":"https://github.com/aababc1/metainsight-book/edit/main/metagenomics/Installation.md","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","exports":[{"format":"md","filename":"Installation.md","url":"/metainsight-book/build/Installation-13dd5c1cf932095e95602db823bfede9.md"}]},"mdast":{"type":"root","children":[{"type":"block","children":[{"type":"paragraph","position":{"start":{"line":3,"column":1},"end":{"line":3,"column":1}},"children":[{"type":"text","value":"This chapter provides a comprehensive guide to installing the bioinformatics software required for the metagenomic analysis pipeline described in this protocol. Metagenomic analysis relies on a diverse set of specialized tools spanning quality control, assembly, binning, taxonomic and functional annotation, and phylogenetic reconstruction. Managing these tools and their dependencies can be challenging, particularly when different programs require incompatible software versions. We recommend using Conda, a cross-platform package and environment manager, which allows each tool to be installed in an isolated environment, thereby avoiding version conflicts and ensuring reproducibility across different computing systems.","position":{"start":{"line":3,"column":1},"end":{"line":3,"column":1}},"key":"L5kuRlJES7"}],"key":"HyFbh0scRf"},{"type":"list","ordered":false,"spread":false,"position":{"start":{"line":5,"column":1},"end":{"line":12,"column":1}},"children":[{"type":"listItem","spread":true,"position":{"start":{"line":5,"column":1},"end":{"line":5,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Preprocessing of the sequencing reads : Kneaddata, minimap2, samtools","position":{"start":{"line":5,"column":1},"end":{"line":5,"column":1}},"key":"M0rELCrEIb"}],"key":"yUiqmnwwM1"}],"key":"NGLgUrCayw"},{"type":"listItem","spread":true,"position":{"start":{"line":6,"column":1},"end":{"line":6,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Read-based profiling : HUMAnN 3.0, MetaPhlAn","position":{"start":{"line":6,"column":1},"end":{"line":6,"column":1}},"key":"uEt17IlEew"}],"key":"WMtKdd8s58"}],"key":"wHmfSzh7mu"},{"type":"listItem","spread":true,"position":{"start":{"line":7,"column":1},"end":{"line":7,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"De novo assembly : MEGAHIT, Quast, hifiasm_meta, Bandage","position":{"start":{"line":7,"column":1},"end":{"line":7,"column":1}},"key":"GWxJeXse2y"}],"key":"abbfhEXcgg"}],"key":"HwVsBmIDdb"},{"type":"listItem","spread":true,"position":{"start":{"line":8,"column":1},"end":{"line":8,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Taxonomic annotation : Kaiju","position":{"start":{"line":8,"column":1},"end":{"line":8,"column":1}},"key":"AhcKIxwSOf"}],"key":"mdiBncqBy6"}],"key":"NPCLDFMTLb"},{"type":"listItem","spread":true,"position":{"start":{"line":9,"column":1},"end":{"line":9,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Functional annotation : Prodigal, EggNOG","position":{"start":{"line":9,"column":1},"end":{"line":9,"column":1}},"key":"h4A6pimkBS"}],"key":"rckKWS4bX0"}],"key":"SJdVYzfeQR"},{"type":"listItem","spread":true,"position":{"start":{"line":10,"column":1},"end":{"line":10,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Binning : MetaBAT2, BBmap, minimap2, samtools, CheckM","position":{"start":{"line":10,"column":1},"end":{"line":10,"column":1}},"key":"BEiJ7fLUj6"}],"key":"RsHQKKzo4A"}],"key":"OUXLAlgJFl"},{"type":"listItem","spread":true,"position":{"start":{"line":11,"column":1},"end":{"line":12,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Phylogenetic tree construction : GTDB-tk, PhyloPhlAn3","position":{"start":{"line":11,"column":1},"end":{"line":11,"column":1}},"key":"U6czIldb8q"}],"key":"DP7zo1ccyU"}],"key":"kRLgQ1FkM9"}],"key":"tfF0iRtFAH"},{"type":"container","children":[{"type":"image","style":{"width":"90%"},"url":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","alt":"Metagenomics analysis pipeline","data":{"altTextIsAutoGenerated":true},"key":"lUNDXSjxFt","urlSource":"https://github.com/sujin9819/MetaInsight/blob/main/SOP/MetaGenomic/img/G_4_1.png?raw=true"},{"type":"caption","children":[{"type":"paragraph","children":[{"type":"strong","children":[{"type":"text","value":"Metagenomics analysis pipeline","key":"uyijArn6wl"}],"key":"zCsZrm6kmD"}],"key":"l9u67nZ4FM"}],"key":"hb6YUz4HGP"}],"enumerator":"1","key":"wQAgfLt76H"},{"type":"heading","depth":2,"position":{"start":{"line":18,"column":1},"end":{"line":18,"column":1}},"children":[{"type":"text","value":"Installation","position":{"start":{"line":18,"column":1},"end":{"line":18,"column":1}},"key":"uIY7KJE2jY"}],"identifier":"installation","label":"Installation","html_id":"installation","implicit":true,"key":"PlZcuiFpls"},{"type":"heading","depth":3,"position":{"start":{"line":20,"column":1},"end":{"line":20,"column":1}},"children":[{"type":"text","value":"An introduction to Conda","position":{"start":{"line":20,"column":1},"end":{"line":20,"column":1}},"key":"n2fNxkK4Td"}],"identifier":"an-introduction-to-conda","label":"An introduction to Conda","html_id":"an-introduction-to-conda","implicit":true,"key":"cfdJ2KLEnk"},{"type":"paragraph","position":{"start":{"line":21,"column":1},"end":{"line":23,"column":1}},"children":[{"type":"text","value":"Metagenomic analysis necessitates the installation of various analysis programs, a process that can sometimes result in conflicts or version dependencies between these programs.\nFor instance, when program A relies on a specific version of program B, and program C depends on a different version of program B, it can be challenging and time-consuming for beginners to set up program C.\nIn such scenarios, conda, a Python package and environment management tool, offers an effective solution for resolving these version conflicts.","position":{"start":{"line":21,"column":1},"end":{"line":21,"column":1}},"key":"lAKQOfi2u5"}],"key":"zVeVpdPCDV"},{"type":"paragraph","position":{"start":{"line":25,"column":1},"end":{"line":27,"column":1}},"children":[{"type":"text","value":"For this Standard Operating Procedure (SOP), we recommend installing the analysis tools using conda.\nYou can find the official documentation for conda at the following ","position":{"start":{"line":25,"column":1},"end":{"line":25,"column":1}},"key":"RyZNG6kQbO"},{"type":"link","url":"https://docs.conda.io/projects/conda/en/latest/user-guide/index.html","position":{"start":{"line":25,"column":1},"end":{"line":25,"column":1}},"children":[{"type":"text","value":"website","position":{"start":{"line":25,"column":1},"end":{"line":25,"column":1}},"key":"YC9TB5t6jw"}],"urlSource":"https://docs.conda.io/projects/conda/en/latest/user-guide/index.html","key":"J5RSVgZfUu"},{"type":"text","value":".\nThis approach ensures a smoother and more efficient installation process, making metagenomic analysis more accessible and less prone to compatibility issues.","position":{"start":{"line":25,"column":1},"end":{"line":25,"column":1}},"key":"QvxjY6ZVxA"}],"key":"u5nSOM3o9U"},{"type":"heading","depth":3,"position":{"start":{"line":29,"column":1},"end":{"line":29,"column":1}},"children":[{"type":"text","value":"Making a new environment","position":{"start":{"line":29,"column":1},"end":{"line":29,"column":1}},"key":"m4jGQVQIs1"}],"identifier":"making-a-new-environment","label":"Making a new environment","html_id":"making-a-new-environment","implicit":true,"key":"Zbm1KJOsUW"},{"type":"paragraph","position":{"start":{"line":30,"column":1},"end":{"line":31,"column":1}},"children":[{"type":"text","value":"It is highly recommended to create a new conda environment every time you install a program. Doing so helps to avoid conflicts between programs and ensures a clean, isolated environment for each analysis tool.\nAn example of how to create a new conda environment is provided below.","position":{"start":{"line":30,"column":1},"end":{"line":30,"column":1}},"key":"Swywr20Mf5"}],"key":"zZ88ptwp7u"},{"type":"code","lang":"bash","value":"# Create a new conda environment\n$ conda create -n new-env ","position":{"start":{"line":32,"column":1},"end":{"line":35,"column":1}},"key":"P8dh3z0a0E"},{"type":"paragraph","position":{"start":{"line":36,"column":1},"end":{"line":36,"column":1}},"children":[{"type":"text","value":"Use the ","position":{"start":{"line":36,"column":1},"end":{"line":36,"column":1}},"key":"fO1CxTrEdu"},{"type":"inlineCode","value":"conda create","position":{"start":{"line":36,"column":1},"end":{"line":36,"column":1}},"key":"SoNTYGyQ7S"},{"type":"text","value":" command with the ","position":{"start":{"line":36,"column":1},"end":{"line":36,"column":1}},"key":"ShDPXf5qHa"},{"type":"inlineCode","value":"-n","position":{"start":{"line":36,"column":1},"end":{"line":36,"column":1}},"key":"YVQC13lSdC"},{"type":"text","value":" flag to designate and save the name of your new environment.","position":{"start":{"line":36,"column":1},"end":{"line":36,"column":1}},"key":"HFNzbk3lQ2"}],"key":"IhjJDDxATH"},{"type":"heading","depth":3,"position":{"start":{"line":37,"column":1},"end":{"line":37,"column":1}},"children":[{"type":"text","value":"Entering \u0026 exiting an environment","position":{"start":{"line":37,"column":1},"end":{"line":37,"column":1}},"key":"xMRuDd5T7h"}],"identifier":"entering-exiting-an-environment","label":"Entering \u0026 exiting an environment","html_id":"entering-exiting-an-environment","implicit":true,"key":"NpBfYAIsIc"},{"type":"code","lang":"bash","value":"# Enter that environment\n$ conda activate new-env\n# Exit whatever conda environment we are running\n$ conda deactivate","position":{"start":{"line":38,"column":1},"end":{"line":43,"column":1}},"key":"Wp1XfP0Rc6"},{"type":"paragraph","position":{"start":{"line":45,"column":1},"end":{"line":45,"column":1}},"children":[{"type":"text","value":"The table below provides example installation codes and Anaconda addresses for the programs used in this SOP.","position":{"start":{"line":45,"column":1},"end":{"line":45,"column":1}},"key":"b5kPacKXVG"}],"key":"QbB3DVkGbv"},{"type":"table","position":{"start":{"line":47,"column":1},"end":{"line":65,"column":1}},"children":[{"type":"tableRow","position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"children":[{"type":"tableCell","header":true,"position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"children":[{"type":"text","value":"Program","position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"key":"DB4dwzbZRy"}],"key":"gg7TzHlQE8"},{"type":"tableCell","header":true,"position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"children":[{"type":"text","value":"Installation code","position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"key":"EworFClKC0"}],"key":"sIqGIbRuLg"},{"type":"tableCell","header":true,"position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"children":[{"type":"text","value":"ANACONDA addresses","position":{"start":{"line":47,"column":1},"end":{"line":47,"column":1}},"key":"KV4r2jMkes"}],"key":"fteVv6zQ0H"}],"key":"JUPEiqpR3G"},{"type":"tableRow","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"children":[{"type":"text","value":"KNEDDATA","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"key":"llj8aqbghO"}],"key":"bOOLuPCYUP"},{"type":"tableCell","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"children":[{"type":"inlineCode","value":"conda create -n kneaddata -c bioconda kneaddata","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"key":"XLpPkc1ph6"}],"key":"fBbF5WO6Vh"},{"type":"tableCell","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/kneaddata","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/kneaddata","position":{"start":{"line":49,"column":1},"end":{"line":49,"column":1}},"key":"r5liPBXTo0"}],"urlSource":"https://anaconda.org/bioconda/kneaddata","key":"gUcEhF0xHD"}],"key":"TV6TSfAM5C"}],"key":"jU2cT1GAPg"},{"type":"tableRow","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"text","value":"HUMANN3.0","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"key":"nAEYpgNQBs"}],"key":"YMduiJDdZo"},{"type":"tableCell","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"inlineCode","value":"conda create -n humann -c biobakery humann","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"key":"lHGwqxJIjm"}],"key":"fMPLsI32Dr"},{"type":"tableCell","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/biobakery/humann","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/biobakery​/humann","position":{"start":{"line":50,"column":1},"end":{"line":50,"column":1}},"key":"iNutctMhGs"}],"urlSource":"https://anaconda.org/biobakery/humann","key":"pCKtmP1W5W"}],"key":"Gyi57RvQfh"}],"key":"h4ojhfFjx3"},{"type":"tableRow","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"children":[{"type":"text","value":"METAPHLAN3.0","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"key":"g3WmeN6G34"}],"key":"fro2O8upUB"},{"type":"tableCell","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n metaphlan -c bioconda metaphlan","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"key":"ykJ45TenNh"}],"key":"opDYbIBA6Y"},{"type":"tableCell","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/metaphlan","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/metaphlan","position":{"start":{"line":51,"column":1},"end":{"line":51,"column":1}},"key":"GJYm8Yv4bJ"}],"urlSource":"https://anaconda.org/bioconda/metaphlan","key":"VzIJilrgDd"}],"key":"GgbcP44D4p"}],"key":"iBePvLC0bu"},{"type":"tableRow","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"children":[{"type":"text","value":"MEGAHIT","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"key":"EUAdRPDKCo"}],"key":"uWV30f2wMW"},{"type":"tableCell","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n MEGAHIT -c bioconda MEGAHIT","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"key":"YPgIghupCf"}],"key":"xBIsr3tDqL"},{"type":"tableCell","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/MEGAHIT","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/MEGAHIT","position":{"start":{"line":52,"column":1},"end":{"line":52,"column":1}},"key":"SgMX5KLho2"}],"urlSource":"https://anaconda.org/bioconda/MEGAHIT","key":"ocikKq4OqM"}],"key":"lNJMT4Efx5"}],"key":"ufoh9bn3rN"},{"type":"tableRow","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"text","value":"QUAST","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"dE52UJD263"}],"key":"rTS65kMpzi"},{"type":"tableCell","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n quast -c bioconda quast","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"yHZwpwVV6E"}],"key":"uIUZTsMjCW"},{"type":"tableCell","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/quast","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/quast","position":{"start":{"line":53,"column":1},"end":{"line":53,"column":1}},"key":"D7NcdYcoQi"}],"urlSource":"https://anaconda.org/bioconda/quast","key":"NvRgEqNK9q"}],"key":"NJCautaaSS"}],"key":"leBN6VNnHA"},{"type":"tableRow","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"children":[{"type":"text","value":"KAIJU","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"key":"E4NxrF8apa"}],"key":"g84IhzEYjZ"},{"type":"tableCell","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n kaiju -c bioconda kaiju","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"key":"gEPU1kqvcf"}],"key":"p4vbdmMT3G"},{"type":"tableCell","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/kaiju","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/kaiju","position":{"start":{"line":54,"column":1},"end":{"line":54,"column":1}},"key":"zd6cJIzfJf"}],"urlSource":"https://anaconda.org/bioconda/kaiju","key":"YKRYBLHFtI"}],"key":"yV1JH4Jq9Q"}],"key":"uCfGFSLLyt"},{"type":"tableRow","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"children":[{"type":"text","value":"MINIMAP2","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"key":"HP5vFN9QJ4"}],"key":"GBMGjyMQjn"},{"type":"tableCell","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n minimap2 -c bioconda minimap2","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"key":"XELEqTvnHZ"}],"key":"U1JyLdJdQ5"},{"type":"tableCell","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/minimap2","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/minimap2","position":{"start":{"line":55,"column":1},"end":{"line":55,"column":1}},"key":"Uv8kSXeN9p"}],"urlSource":"https://anaconda.org/bioconda/minimap2","key":"aiGWeHoJJ4"}],"key":"f3IxvAwALg"}],"key":"DIS5CzWosA"},{"type":"tableRow","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"children":[{"type":"text","value":"HIFIASM_META","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"key":"hkvEzQo98H"}],"key":"sPhMLlKzR7"},{"type":"tableCell","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n hifiasm_meta -c bioconda hifiasm_meta","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"key":"DwD7QhhTws"}],"key":"KprZriUj8v"},{"type":"tableCell","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/hifiasm_meta","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/hifiasm​_meta","position":{"start":{"line":56,"column":1},"end":{"line":56,"column":1}},"key":"JTkDVHP1Ak"}],"urlSource":"https://anaconda.org/bioconda/hifiasm_meta","key":"sPn3ppx8Tx"}],"key":"ZLDNrJqDxB"}],"key":"SvsytmPioa"},{"type":"tableRow","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"children":[{"type":"text","value":"KRONA","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"key":"lnfpKnNqLf"}],"key":"RKAfAtdnrL"},{"type":"tableCell","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n krona -c bioconda krona","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"key":"ywS2ZPEubm"}],"key":"AVZZY33KZ8"},{"type":"tableCell","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/krona","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/krona","position":{"start":{"line":57,"column":1},"end":{"line":57,"column":1}},"key":"V0iZGlGgjg"}],"urlSource":"https://anaconda.org/bioconda/krona","key":"hbsWJCqiAq"}],"key":"HU2sqz3SZA"}],"key":"tGCLJpzfXU"},{"type":"tableRow","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"children":[{"type":"text","value":"PRODIGAL","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"key":"mJvRTkiXTU"}],"key":"cCmEh9iY05"},{"type":"tableCell","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n prodigal -c bioconda prodigal","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"key":"EcUWTKDZ9f"}],"key":"z16BabofQl"},{"type":"tableCell","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/prodigal","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/prodigal","position":{"start":{"line":58,"column":1},"end":{"line":58,"column":1}},"key":"KIfH5CxwwW"}],"urlSource":"https://anaconda.org/bioconda/prodigal","key":"TlIevbhr09"}],"key":"OIwy3Dx6ph"}],"key":"hzQtNhG8i0"},{"type":"tableRow","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"children":[{"type":"text","value":"EGGNOG","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"key":"kM3HbYRynw"}],"key":"H2nhBEOYQt"},{"type":"tableCell","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n eggnog -c bioconda eggnog-mapper","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"key":"fTc4nsBzRT"}],"key":"spw6DTIan2"},{"type":"tableCell","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/eggnog-mapper","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/eggnog​-mapper","position":{"start":{"line":59,"column":1},"end":{"line":59,"column":1}},"key":"h6GTHTCF9r"}],"urlSource":"https://anaconda.org/bioconda/eggnog-mapper","key":"ScrzrWRU5x"}],"key":"Xro7xriR7C"}],"key":"ADUtVyBD0H"},{"type":"tableRow","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"children":[{"type":"text","value":"METABAT2","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"key":"ww8c1Jz0py"}],"key":"moRYRAeyCT"},{"type":"tableCell","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n MetaBAT2 -c bioconda MetaBAT2","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"key":"Rvk71130Mm"}],"key":"TItEneftnj"},{"type":"tableCell","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/MetaBAT2","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/MetaBAT2","position":{"start":{"line":60,"column":1},"end":{"line":60,"column":1}},"key":"UWlu5mYbfJ"}],"urlSource":"https://anaconda.org/bioconda/MetaBAT2","key":"YE66yaKpjV"}],"key":"hSaq3Ktlcu"}],"key":"gS8E7aT6sg"},{"type":"tableRow","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"children":[{"type":"text","value":"BBMAP","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"key":"Wymhzfid3y"}],"key":"Tptyb3mXgm"},{"type":"tableCell","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n bbmap -c bioconda bbmap","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"key":"oIF8qvXvFt"}],"key":"Z73UFjeJsJ"},{"type":"tableCell","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/bbmap","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/bbmap","position":{"start":{"line":61,"column":1},"end":{"line":61,"column":1}},"key":"TSV9Jo7ZRP"}],"urlSource":"https://anaconda.org/bioconda/bbmap","key":"BQDafNOKvn"}],"key":"xGom4AhEu4"}],"key":"Mz5JJ6Ys8r"},{"type":"tableRow","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"children":[{"type":"text","value":"SAMTOOLS","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"key":"J8dBwPUWpr"}],"key":"e1ceCtf8GX"},{"type":"tableCell","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n samtools -c bioconda samtools","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"key":"rQwncCn9lL"}],"key":"u7QWJqB7KU"},{"type":"tableCell","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/samtools","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/samtools","position":{"start":{"line":62,"column":1},"end":{"line":62,"column":1}},"key":"ynoUEvJa1s"}],"urlSource":"https://anaconda.org/bioconda/samtools","key":"V1UfDRmT5t"}],"key":"opwbtGKaK5"}],"key":"SL0guJkQZ2"},{"type":"tableRow","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"text","value":"CHECKM","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"M4VlgGufEJ"}],"key":"CbYQjvEBKs"},{"type":"tableCell","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"inlineCode","value":"conda create -n CheckM python=3.9","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"gGEiIbzXN8"},{"type":"text","value":" ","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"QJGvYHTKbN"},{"type":"break","key":"w4YKlv610d"},{"type":"text","value":" ","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"ECbuZJriHT"},{"type":"inlineCode","value":"conda activate CheckM","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"GbNWexrcQN"},{"type":"text","value":" ","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"CyPxJCD0Ua"},{"type":"break","key":"o9LpD7DcIT"},{"type":"inlineCode","value":"conda install -c bioconda numpy matplotlib pysam","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"GT3slXqGh2"},{"type":"break","key":"C1JQkyfEFO"},{"type":"inlineCode","value":"conda install -c bioconda hmmer prodigal pplacer","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"XFs7Ud9RMV"},{"type":"break","key":"Bo00nuLGDI"},{"type":"inlineCode","value":"pip3 install CheckM-genome","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"UKoH7OraaF"}],"key":"ViVAPOjppZ"},{"type":"tableCell","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/CheckM-genome","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/CheckM​-genome","position":{"start":{"line":63,"column":1},"end":{"line":63,"column":1}},"key":"tmSgU95cbM"}],"urlSource":"https://anaconda.org/bioconda/CheckM-genome","key":"g6DhE016yr"}],"key":"A4r4CiRHrU"}],"key":"EooePaLJXc"},{"type":"tableRow","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"children":[{"type":"text","value":"GTDB-TK","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"key":"oost3Howx9"}],"key":"cKWJrBqN4C"},{"type":"tableCell","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"children":[{"type":"inlineCode","value":"conda create -n gtdbtk-2.1.1 -c conda-forge -c bioconda gtdbtk=2.1.1","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"key":"uBtvoC01qc"}],"key":"IkZIIVHpHM"},{"type":"tableCell","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/gtdbtk","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/gtdbtk","position":{"start":{"line":64,"column":1},"end":{"line":64,"column":1}},"key":"rV48293Mqq"}],"urlSource":"https://anaconda.org/bioconda/gtdbtk","key":"djHuQYdkqI"}],"key":"JzgHwpWTzm"}],"key":"jaaM6peYH2"},{"type":"tableRow","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"children":[{"type":"tableCell","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"children":[{"type":"text","value":"PHYLOPHLAN3","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"key":"VNOPIc8OiH"}],"key":"z7PoELHo6m"},{"type":"tableCell","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"children":[{"type":"inlineCode","value":"conda create –n phylophlan -c bioconda phylophlan","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"key":"w3p76RlYm2"}],"key":"xwaZeTZqPQ"},{"type":"tableCell","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"children":[{"type":"link","url":"https://anaconda.org/bioconda/phylophlan","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"children":[{"type":"text","value":"https://​anaconda​.org​/bioconda​/phylophlan","position":{"start":{"line":65,"column":1},"end":{"line":65,"column":1}},"key":"w9Ru67j8Ni"}],"urlSource":"https://anaconda.org/bioconda/phylophlan","key":"EhIJ6VBL6D"}],"key":"duzYgGpHQF"}],"key":"jkDTDV1vby"}],"key":"HduzZQiX6z"},{"type":"heading","depth":3,"position":{"start":{"line":67,"column":1},"end":{"line":67,"column":1}},"children":[{"type":"text","value":"References","position":{"start":{"line":67,"column":1},"end":{"line":67,"column":1}},"key":"wmdVavzT1n"}],"identifier":"references","label":"References","html_id":"references","implicit":true,"key":"YbtNqO6Z2s"},{"type":"list","ordered":true,"start":1,"spread":false,"position":{"start":{"line":69,"column":1},"end":{"line":78,"column":1}},"children":[{"type":"listItem","spread":true,"position":{"start":{"line":69,"column":1},"end":{"line":69,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Li D, Liu CM, Luo R, et al. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. ","position":{"start":{"line":69,"column":1},"end":{"line":69,"column":1}},"key":"usZu748lsM"},{"type":"emphasis","position":{"start":{"line":69,"column":1},"end":{"line":69,"column":1}},"children":[{"type":"text","value":"Bioinformatics","position":{"start":{"line":69,"column":1},"end":{"line":69,"column":1}},"key":"zNNfu14v2i"}],"key":"RP7uLbG6EJ"},{"type":"text","value":". 2015;31(10):1674-1676. doi:10.1093/bioinformatics/btv033","position":{"start":{"line":69,"column":1},"end":{"line":69,"column":1}},"key":"gb5iwBDKuH"}],"key":"RSkIyfezd8"}],"key":"fWhdhYTtUA"},{"type":"listItem","spread":true,"position":{"start":{"line":70,"column":1},"end":{"line":70,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Nurk S, Meleshko D, Korobeynikov A, et al. metaSPAdes: a new versatile metagenomic assembler. ","position":{"start":{"line":70,"column":1},"end":{"line":70,"column":1}},"key":"GwMlqmZvuP"},{"type":"emphasis","position":{"start":{"line":70,"column":1},"end":{"line":70,"column":1}},"children":[{"type":"text","value":"Genome Res","position":{"start":{"line":70,"column":1},"end":{"line":70,"column":1}},"key":"n7d73WFid8"}],"key":"zqTMroafmY"},{"type":"text","value":". 2017;27(5):824-834. doi:10.1101/gr.213959.116","position":{"start":{"line":70,"column":1},"end":{"line":70,"column":1}},"key":"F4EXXIH0JE"}],"key":"O80fm8iuML"}],"key":"jBKorRec1O"},{"type":"listItem","spread":true,"position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Kang DD, Li F, Kirton ES, et al. MetaBAT 2: an improved binning algorithm, assembly-independent binning and recovering metagenome-assembled genomes from heterogeneous sources. ","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"key":"z9qmCHhv8S"},{"type":"emphasis","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"children":[{"type":"text","value":"PeerJ","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"key":"DNELOiXv9G"}],"key":"CgLmXoAKOL"},{"type":"text","value":". 2019;7:e7359. doi:10.7717/peerj.7359","position":{"start":{"line":71,"column":1},"end":{"line":71,"column":1}},"key":"bzhIRQcT7x"}],"key":"YsD5k4QHlO"}],"key":"eB4biRV83I"},{"type":"listItem","spread":true,"position":{"start":{"line":72,"column":1},"end":{"line":72,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Parks DH, Imelfort M, Skennerton CT, et al. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. ","position":{"start":{"line":72,"column":1},"end":{"line":72,"column":1}},"key":"O8CL2RZ3oS"},{"type":"emphasis","position":{"start":{"line":72,"column":1},"end":{"line":72,"column":1}},"children":[{"type":"text","value":"Genome Res","position":{"start":{"line":72,"column":1},"end":{"line":72,"column":1}},"key":"bzFkDOX7qI"}],"key":"j4Rp6BSNsN"},{"type":"text","value":". 2015;25(7):1043-1055. doi:10.1101/gr.186072.114","position":{"start":{"line":72,"column":1},"end":{"line":72,"column":1}},"key":"kLi5VkwUnf"}],"key":"pGMGtxQSNX"}],"key":"oWHFPWf7Wb"},{"type":"listItem","spread":true,"position":{"start":{"line":73,"column":1},"end":{"line":73,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Chaumeil PA, Mussig AJ, Hugenholtz P, et al. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. ","position":{"start":{"line":73,"column":1},"end":{"line":73,"column":1}},"key":"Zc2aHfXhSl"},{"type":"emphasis","position":{"start":{"line":73,"column":1},"end":{"line":73,"column":1}},"children":[{"type":"text","value":"Bioinformatics","position":{"start":{"line":73,"column":1},"end":{"line":73,"column":1}},"key":"RIpdtozlzT"}],"key":"AW7GNShluu"},{"type":"text","value":". 2020;36(6):1925-1927. doi:10.1093/bioinformatics/btz848","position":{"start":{"line":73,"column":1},"end":{"line":73,"column":1}},"key":"gilXCxgmWS"}],"key":"QvoI4onghZ"}],"key":"PKKPP9wPoc"},{"type":"listItem","spread":true,"position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Hyatt D, Chen GL, Locascio PF, et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. ","position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"key":"heWBHHkD9S"},{"type":"emphasis","position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"children":[{"type":"text","value":"BMC Bioinformatics","position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"key":"DghELRgDlS"}],"key":"PduOOvkCvw"},{"type":"text","value":". 2010;11:119. doi:10.1186/1471-2105-11-119","position":{"start":{"line":74,"column":1},"end":{"line":74,"column":1}},"key":"aZKmncCeaL"}],"key":"A70iedN2hy"}],"key":"qUxG0hb3DO"},{"type":"listItem","spread":true,"position":{"start":{"line":75,"column":1},"end":{"line":75,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Huerta-Cepas J, Forslund K, Coelho LP, et al. eggNOG 4.5: a hierarchical orthology framework with improved functional annotations for eukaryotic, prokaryotic and viral sequences. ","position":{"start":{"line":75,"column":1},"end":{"line":75,"column":1}},"key":"H6HMHTo9Lu"},{"type":"emphasis","position":{"start":{"line":75,"column":1},"end":{"line":75,"column":1}},"children":[{"type":"text","value":"Nucleic Acids Res","position":{"start":{"line":75,"column":1},"end":{"line":75,"column":1}},"key":"u8WoSWI8U3"}],"key":"EuRi5ZKDeA"},{"type":"text","value":". 2016;44(D1):D286-D293. doi:10.1093/nar/gkv1248","position":{"start":{"line":75,"column":1},"end":{"line":75,"column":1}},"key":"tLCI3Uyz9o"}],"key":"zBFD0woyq3"}],"key":"ddaLFNg8O0"},{"type":"listItem","spread":true,"position":{"start":{"line":76,"column":1},"end":{"line":76,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Asnicar F, Thomas AM, Beghini F, et al. Serotype and clonal analysis of pathogenic Listeria monocytogenes strains. ","position":{"start":{"line":76,"column":1},"end":{"line":76,"column":1}},"key":"RgdRGbNM0P"},{"type":"emphasis","position":{"start":{"line":76,"column":1},"end":{"line":76,"column":1}},"children":[{"type":"text","value":"Nat Commun","position":{"start":{"line":76,"column":1},"end":{"line":76,"column":1}},"key":"oW1boGfvzK"}],"key":"HNe9d2RuIf"},{"type":"text","value":". 2020;11(1):2500. doi:10.1038/s41467-020-16366-7","position":{"start":{"line":76,"column":1},"end":{"line":76,"column":1}},"key":"NJc9t2VrXE"}],"key":"AaPxKGcUWQ"}],"key":"vGb3rXkx3d"},{"type":"listItem","spread":true,"position":{"start":{"line":77,"column":1},"end":{"line":77,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Franzosa EA, McIver LJ, Rahnavard G, et al. Species-level functional profiling of metagenomes and metatranscriptomes. ","position":{"start":{"line":77,"column":1},"end":{"line":77,"column":1}},"key":"TzT8WHobEa"},{"type":"emphasis","position":{"start":{"line":77,"column":1},"end":{"line":77,"column":1}},"children":[{"type":"text","value":"Nat Methods","position":{"start":{"line":77,"column":1},"end":{"line":77,"column":1}},"key":"H7S6kg5MpC"}],"key":"jVqefEdA86"},{"type":"text","value":". 2018;15(11):962-968. doi:10.1038/s41592-018-0176-y","position":{"start":{"line":77,"column":1},"end":{"line":77,"column":1}},"key":"PXRtGb16rU"}],"key":"GXAymIDHXj"}],"key":"FC3OobpEn9"},{"type":"listItem","spread":true,"position":{"start":{"line":78,"column":1},"end":{"line":78,"column":1}},"children":[{"type":"paragraph","children":[{"type":"text","value":"Segata N, Waldron L, Ballarini A, et al. Metagenomic microbial community profiling using unique clade-specific marker genes. ","position":{"start":{"line":78,"column":1},"end":{"line":78,"column":1}},"key":"AhTvzzoFSz"},{"type":"emphasis","position":{"start":{"line":78,"column":1},"end":{"line":78,"column":1}},"children":[{"type":"text","value":"Nat Methods","position":{"start":{"line":78,"column":1},"end":{"line":78,"column":1}},"key":"O7ymBDLq1I"}],"key":"Be6zbpY4y1"},{"type":"text","value":". 2012;9(8):811-814. doi:10.1038/nmeth.2066","position":{"start":{"line":78,"column":1},"end":{"line":78,"column":1}},"key":"V1pGS0rrFv"}],"key":"IJsfb2Pfgg"}],"key":"KKntROF9Ko"}],"key":"H5p1T36qKf"}],"key":"xTYHWQw53h"}],"key":"PXZvvqj3aK"},"references":{"cite":{"order":[],"data":{}}},"footer":{"navigation":{"prev":{"title":"Preparation for sequencing","url":"/sequencing","group":"Metagenomics"},"next":{"title":"Preprocessing of the sequencing reads","url":"/preprocessing","group":"Metagenomics"}}},"domain":"http://localhost:3000"},"project":{"title":"META-Insight","description":"Meta-omics Analysis Protocols for Microbiome Research","authors":[{"nameParsed":{"literal":"META-Insight Team","given":"META-Insight","family":"Team"},"name":"META-Insight Team","id":"contributors-myst-generated-uid-0"}],"github":"https://github.com/aababc1/metainsight-book","keywords":["metagenomics","metatranscriptomics","metaproteomics","microbiome","bioinformatics"],"id":"c2a2c0b2-1da1-45ec-9e5a-48ed4e825549","toc":[{"file":"index.md"},{"children":[{"file":"metagenomics/Introduction.md"},{"file":"metagenomics/Sequencing.md"},{"file":"metagenomics/Installation.md"},{"file":"metagenomics/Preprocessing.md"},{"file":"metagenomics/Denovo_assembly.md"},{"file":"metagenomics/Binning.md"},{"file":"metagenomics/Taxonomic.md"},{"file":"metagenomics/phylogenetic.md"},{"file":"metagenomics/Functional.md"},{"file":"metagenomics/Read-based.md"}],"title":"Metagenomics"},{"children":[{"file":"metatranscriptomics/Introduction.md"},{"file":"metatranscriptomics/Sequencing.md"},{"file":"metatranscriptomics/Installation.md"},{"file":"metatranscriptomics/Preprocessing.md"},{"file":"metatranscriptomics/Alignment.md"},{"file":"metatranscriptomics/Expression_profiling.md"},{"file":"metatranscriptomics/Taxonomy.md"},{"file":"metatranscriptomics/Denovo_assembly.md"},{"file":"metatranscriptomics/Read_based.md"},{"file":"metatranscriptomics/DE_analysis.md"}],"title":"Metatranscriptomics"},{"children":[{"file":"metaproteomics/Introduction.md"},{"file":"metaproteomics/Sequencing.md"},{"file":"metaproteomics/Installation.md"},{"file":"metaproteomics/Preprocessing.md"},{"file":"metaproteomics/Alignment.md"},{"file":"metaproteomics/Ribosome_profiling.md"},{"file":"metaproteomics/Translational_quantification.md"},{"file":"metaproteomics/Read_based.md"},{"file":"metaproteomics/Small_protein_prediction.md"},{"file":"metaproteomics/DE_analysis.md"}],"title":"Translatome"}],"exports":[],"bibliography":[],"index":"index","pages":[{"level":1,"title":"Metagenomics"},{"slug":"introduction","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metagenome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/9c38558592f6006cd3733c07b78d8e06.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing","title":"Preparation for sequencing","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation","title":"Metagenomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/53258f98503c652a48c7b51df2b5d12b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/ae19e36fb558195791299558aff26c80.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly","title":"De novo assembly","description":"","date":"","thumbnail":"/metainsight-book/build/030715b7a3310134807a1e7844273e01.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"binning","title":"Binning","description":"","date":"","thumbnail":"/metainsight-book/build/f226ce7a430a813404ff54147137e120.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomic","title":"Taxonomic annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/201d3bcd8d9e1547a5b3fd4f98408d2a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"phylogenetic","title":"References","description":"","date":"","thumbnail":"/metainsight-book/build/e6b02b1a697b1a8a415b4b0b36d34ff7.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"functional","title":"Functional annotation of contigs","description":"","date":"","thumbnail":"/metainsight-book/build/98064564346bc129cc6c61fb7c96e203.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based","title":"Read-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/b52f136c3368f4f1552e38357183e433.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Metatranscriptomics"},{"slug":"introduction-1","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Metatranscriptome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/3b93ceb54e312f19f9048aa575c5358a.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-1","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/1285dc54e97172cc765f4939c33885fa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-1","title":"Metatranscriptomic analysis tools","description":"","date":"","thumbnail":"/metainsight-book/build/3acc32a5c0f73ae79ce10c10276f0e89.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-1","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/e246ba899591ac8b73803b16a0523732.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/e4b27805981a71b09cc8a79c9a4769e5.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"expression-profiling","title":"Transcriptional expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5f774a0f96ccdbf775e515a8522f3932.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"taxonomy","title":"Transcript taxonomy classification","description":"","date":"","thumbnail":"/metainsight-book/build/bec278d3bf748130ed9de187e488df5b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"denovo-assembly-1","title":"Transcript de novo assembly","description":"","date":"","thumbnail":"","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-1","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/008ecfeb4b8272602dc68d3c98f0bafa.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/c6608b975bf563132df84fc74d23aca6.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"level":1,"title":"Translatome"},{"slug":"introduction-2","title":"Standard Operating Procedure for Next-Generation Sequencing (NGS)-Based Translatome Analysis","description":"","date":"","thumbnail":"/metainsight-book/build/75d390134430cb9d5bf8b8faeb6f3c9c.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"sequencing-2","title":"Sequencing","description":"","date":"","thumbnail":"/metainsight-book/build/8d08af7c044f137f7bb82466bad3e9af.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"installation-2","title":"Translatome Analysis Tools","description":"","date":"","thumbnail":"/metainsight-book/build/0fd52ff68adab0aef247a24f80b8e20b.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"preprocessing-2","title":"Preprocessing of the sequencing reads","description":"","date":"","thumbnail":"/metainsight-book/build/1c767ab6674d590b1ef1852cf702350f.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"alignment-1","title":"Reads alignment","description":"","date":"","thumbnail":"/metainsight-book/build/da72c7b73b93e10407425ac1b75d81ca.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"ribosome-profiling","title":"Ribosome profiling","description":"","date":"","thumbnail":"/metainsight-book/build/0d56656caea2762d4799b84a70e87207.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"translational-quantification","title":"Translational expression profiling","description":"","date":"","thumbnail":"/metainsight-book/build/051814868d0a180c47144de1ab9b65cc.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"read-based-2","title":"Reads-based profiling","description":"","date":"","thumbnail":"/metainsight-book/build/5c985cccb6005cf72eb9485f587acfed.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"small-protein-prediction","title":"Small protein prediction","description":"","date":"","thumbnail":"/metainsight-book/build/7eb605b21cf15a378b22b793218862d2.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2},{"slug":"de-analysis-1","title":"Differential (transcriptional) expression analysis","description":"","date":"","thumbnail":"/metainsight-book/build/ff905171291bc710415f3ba404d2fbdd.png","thumbnailOptimized":"","banner":"","bannerOptimized":"","tags":[],"level":2}]}}},"actionData":null,"errors":null},"future":{"unstable_dev":false,"unstable_postcss":false,"unstable_tailwind":false,"v2_errorBoundary":true,"v2_headers":true,"v2_meta":true,"v2_normalizeFormMethod":true,"v2_routeConvention":true}};</script><script type="module" async="">import "/metainsight-book/build/manifest-3595279D.js";
import * as route0 from "/metainsight-book/build/root-SIO6LUTY.js";
import * as route1 from "/metainsight-book/build/routes/$-PRP77N34.js";
window.__remixRouteModules = {"root":route0,"routes/$":route1};

import("/metainsight-book/build/entry.client-PCJPW7TK.js");</script></body></html>